<script src="https://code.jquery.com/jquery-3.7.0.js"></script>
<script src="https://cdn.datatables.net/1.13.6/js/jquery.dataTables.min.js"></script>
<script src="https://cdn.datatables.net/searchbuilder/1.6.0/js/dataTables.searchBuilder.min.js"></script>
<script src="https://cdn.datatables.net/datetime/1.5.1/js/dataTables.dateTime.min.js"></script>

<script src="https://cdn.datatables.net/responsive/2.5.0/js/dataTables.responsive.min.js"></script>

<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/responsive/2.2.3/css/responsive.dataTables.min.css">

<link rel="stylesheet" href="https://cdn.datatables.net/1.13.6/css/jquery.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/searchbuilder/1.6.0/css/searchBuilder.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/datetime/1.5.1/css/dataTables.dateTime.min.css">

<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" />
<script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>

<style type="text/css">
  .tooltip {
    font-family: Georgia;
    font-size: 18px;
  }
  .tooltip .tooltip-inner {
    background-color: #00FFFF;
    color: #000000;
    min-width: 400px;
  }
  </style>

<script>
    $(document).ready(function() {
    $('#dataframe').DataTable( {
        "sScrollX": "100%",
        "dom": "Qlfrtip",

        responsive: {
            details: {
                type: 'column'
            }
        },
        "bAutoWidth": false,
        columnDefs: [ {
            className: 'control',
            orderable: false,
            targets:   [ 0 ],
            width: 100,
        },
        { type: 'natural', targets: 1 } // allow normal sorting
        ],

    });
});
</script>

<div class="display nowrap" style="width:80%" width="100%">

<table class="dataframe display nowrap" id="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th>Expand</th>
      <th>Topic</th>
      <th>Title</th>
      <th>Link</th>
      <th>EDAM topic</th>
      <th>EDAM operation</th>
      <th>Creation</th>
      <th>Last modification</th>
      <th>Version</th>
      <th>Tutorial</th>
      <th>Slides</th>
      <th>Video</th>
      <th>Workflows</th>
      <th>Tools</th>
      <th>Servers with precise tool versions</th>
      <th>Servers with tool but different versions</th>
      <th>Feedback number</th>
      <th>Feedback mean note</th>
      <th>Visitors</th>
      <th>Page views</th>
      <th>Visit duration</th>
      <th>Video views</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Making sense of a newly assembled genome</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/ecoli_comparison/tutorial.html</td>
      <td>Sequence assembly, Genomics, Microbiology</td>
      <td>Mapping, Sequence visualisation, Genome visualisation, Sequence alignment, Read mapping</td>
      <td>2018-06-14</td>
      <td>2024-06-24</td>
      <td>29</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tp_cat, Grep1, fasta_compute_length, circos_interval_to_tiles, tp_replace_in_column, bedtools_intersectbed, fasta_filter_by_length, tp_sort_header_tool, addValue, bedtools_complementbed, mergeCols1, upload1, tp_sed_tool, Cut1, join1, datamash_ops, collapse_dataset, tp_replace_in_line, tp_grep_tool, jbrowse, circos_aln_to_links, cat1, Filter1, lastz_wrapper_2, circos, bedtools_sortbed, random_lines1</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>6318</td>
      <td>12511</td>
      <td>3.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Assembly of metagenomic sequencing data</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/metagenomics-assembly/tutorial.html</td>
      <td>Metagenomics, Sequence assembly</td>
      <td>Data handling, Variant calling, Sequence contamination filtering, Sequencing quality control, Visualisation, Sequence composition calculation, Local alignment, Sequence assembly validation, Sequence alignment analysis, Genome assembly, Sequence trimming, Primer removal, Formatting, Sequence file editing, Read mapping, Statistical calculation, Sequence assembly visualisation, Read pre-processing</td>
      <td>2025-02-23</td>
      <td>2025-02-23</td>
      <td>0</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>tp_cat, bamtools, bowtie2, quast, megahit_contig2fastg, megahit, coverm_contig, ngsutils_bam_filter, bandage_image, seqtk_subseq, cutadapt, bg_uniq, filter_tabular, metaspades, collection_column_join, fastqc, bandage_info, random_lines1</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td>0</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Genome Assembly of MRSA from Oxford Nanopore MinION data (and optionally Illumina data)</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/mrsa-nanopore/tutorial.html</td>
      <td>Sequence assembly, Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology, Antimicrobial resistance</td>
      <td>Scatter plot plotting, Genome assembly, Box-Whisker plot plotting, Mapping assembly, Sequencing error detection, De-novo assembly, Sequence alignment, Read mapping, Statistical calculation, Sequence assembly visualisation, Sequence contamination filtering, Sequencing quality control, Visualisation, Sequence composition calculation, Cross-assembly, Filtering, Sequence assembly validation</td>
      <td>2021-03-24</td>
      <td>2024-11-29</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>fastp, polypolish, quast, nanoplot, porechop, filtlong, flye, bandage_image, upload1, fastqc, CONVERTER_bz2_to_uncompressed, bwa_mem2</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>8304</td>
      <td>12545</td>
      <td>3.333333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Unicycler Assembly</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/unicycler-assembly/tutorial.html</td>
      <td>Sequence assembly, Genomics, Microbiology</td>
      <td>Genome assembly, Genome annotation, Statistical calculation, Validation, Coding region prediction, Sequence composition calculation, Sequencing quality control, Visualisation, Gene prediction, Sequence assembly validation, Aggregation</td>
      <td>2017-10-11</td>
      <td>2024-03-13</td>
      <td>23</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>quast, unicycler, multiqc, prokka, fastqc</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>GalaxyTrakr, MISSISSIPPI, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2365059</td>
      <td>6472517</td>
      <td>4.616667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Genome Assembly of a bacterial genome (MRSA) sequenced using Illumina MiSeq Data</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/mrsa-illumina/tutorial.html</td>
      <td>Sequence assembly, Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology</td>
      <td>Genome assembly, Statistical calculation, Sequence assembly visualisation, Sequence contamination filtering, Sequencing quality control, Visualisation, Sequence composition calculation, Sequence assembly validation</td>
      <td>2021-03-24</td>
      <td>2024-09-18</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>fastp, quast, shovill, bandage_image, upload1, fastqc, bandage_info</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>Galaxy@AuBi, GalaxyTrakr, UseGalaxy.be, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>7893</td>
      <td>12460</td>
      <td>3.666667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Checking expected species and contamination in bacterial isolate</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/bacterial-isolate-species-contamination-checking/tutorial.html</td>
      <td>Whole genome sequencing, Genomics, Microbiology, Microbial ecology</td>
      <td>Taxonomic classification, Statistical calculation, Expression analysis, Cross-assembly</td>
      <td>2024-03-04</td>
      <td>2024-06-27</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>upload1, kraken2, est_abundance, recentrifuge</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td>Galaxy@AuBi, GalaxyTrakr, UseGalaxy.cz, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>611</td>
      <td>850</td>
      <td>3.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Tree thinking for tuberculosis evolution and epidemiology</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/mtb_phylogeny/tutorial.html</td>
      <td>Genomics, Microbiology, Phylogeny, Infectious disease</td>
      <td>Phylogenetic tree analysis, Sequence analysis</td>
      <td>2022-03-16</td>
      <td>2024-03-12</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>raxml, upload1, interactive_tool_rstudio</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>1600</td>
      <td>2342</td>
      <td>2.150000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Identifying tuberculosis transmission links: from SNPs to transmission clusters</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/mtb_transmission/tutorial.html</td>
      <td>Genomics, Microbiology, Infectious disease, DNA polymorphism</td>
      <td>Phylogenetic tree generation, Antimicrobial resistance prediction, Variant calling, Phylogenetic tree visualisation</td>
      <td>2022-03-16</td>
      <td>2024-07-12</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tp_cat, snp_dists, tb_variant_filter, __MERGE_COLLECTION__, snp_sites, trimmomatic, Grep1, tp_easyjoin_tool, snippy, addName, collapse_dataset, upload1, samtools_view, tp_replace_in_line, tp_sed_tool, tb_profiler_profile, bcftools_consensus, tp_grep_tool</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>1931</td>
      <td>2898</td>
      <td>2.833333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Genome annotation with Prokka</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/annotation-with-prokka/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis</td>
      <td>Genome annotation, Gene prediction, Coding region prediction, Genome visualisation</td>
      <td>2018-03-06</td>
      <td>2024-08-08</td>
      <td>25</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>prokka, jbrowse</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>Galaxy@AuBi, GalaxyTrakr, MISSISSIPPI, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>2370146</td>
      <td>6481031</td>
      <td>4.400000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Refining Genome Annotations with Apollo (prokaryotes)</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/apollo/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis</td>
      <td>Genome visualisation</td>
      <td>2021-06-04</td>
      <td>2024-10-15</td>
      <td>20</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>create_account, iframe, list_organism, create_or_update, jbrowse</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2127598</td>
      <td>5674472</td>
      <td>5.266667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Bacterial Genome Annotation</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/bacterial-genome-annotation/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis, Whole genome sequencing, Functional genomics, Mobile genetic elements</td>
      <td>Genome assembly, Sequence motif recognition, Genome annotation, Multilocus sequence typing, Protein feature detection, Structural variation detection, Scaffolding, Nucleic acid feature detection, Genome visualisation</td>
      <td>2024-02-01</td>
      <td>2024-10-15</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>tbl2gff3, jbrowse, plasmidfinder, bakta, Grouping1, integron_finder, upload1, tp_replace_in_column, isescan, tp_tail_tool</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td>Galaxy@AuBi, UseGalaxy.cz, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>3813</td>
      <td>5190</td>
      <td>2.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Identification of AMR genes in an assembled bacterial genome</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/amr-gene-detection/tutorial.html</td>
      <td>Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology, Sequence analysis, Infectious disease, Antimicrobial resistance</td>
      <td>Genome annotation, Antimicrobial resistance prediction, Genome visualisation, Read mapping</td>
      <td>2024-01-23</td>
      <td>2024-10-15</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>bowtie2, Grep1, tbl2gff3, bakta, staramr_search, upload1, jbrowse</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr</td>
      <td>Galaxy@AuBi, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>2065</td>
      <td>2886</td>
      <td>3.033333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Comparative gene analysis in unannotated genomes</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/gene-centric/tutorial.html</td>
      <td>Genomics, Gene and protein families, Sequence analysis, Phylogeny</td>
      <td>Coding region prediction, Database search, Transcriptome assembly, De-novo assembly, Phylogenetic tree generation, Multiple sequence alignment, Sequence alignment analysis</td>
      <td>2022-09-08</td>
      <td>2024-03-18</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>bg_diamond_makedb, join1, regexColumn1, rbc_mafft, Add_a_column1, cat1, orfipy, Filter1, tp_split_on_column, bg_diamond_view, tab2fasta, rapidnj, gops_intersect_1, collapse_dataset, Cut1, bg_diamond</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>1280</td>
      <td>1879</td>
      <td>3.283333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Essential genes detection with Transposon insertion sequencing</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/tnseq/tutorial.html</td>
      <td>Genomics, Microbiology, Sequence analysis, Mobile genetic elements</td>
      <td>Sequence trimming, Primer removal, Transposon prediction, Read pre-processing</td>
      <td>2019-07-02</td>
      <td>2024-03-18</td>
      <td>12</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>transit_gumbel, bg_find_subsequences, tp_sort_header_tool, Add_a_column1, __EXTRACT_DATASET__, deeptools_bam_coverage, tp_easyjoin_tool, Filter1, cutadapt, bowtie_wrapper, gff_to_prot, Cut1</td>
      <td>UseGalaxy.be, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td>0</td>
      <td></td>
      <td>2360485</td>
      <td>6465410</td>
      <td>4.333333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Microbial Variant Calling</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/microbial-variants/tutorial.html</td>
      <td>Genomics, Sequence assembly, DNA polymorphism, Microbiology, Sequence analysis, Genetic variation</td>
      <td>Phylogenetic tree generation, Genome visualisation, Variant calling, Phylogenetic tree visualisation</td>
      <td>2018-02-26</td>
      <td>2024-03-15</td>
      <td>23</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>snippy, jbrowse</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>GalaxyTrakr, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>13387</td>
      <td>21798</td>
      <td>1.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Analyses of metagenomics data - The global picture</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/general-tutorial/tutorial.html</td>
      <td>Metagenomics, Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td>Phylogenetic analysis, Taxonomic classification, Sequencing quality control, Visualisation, Sequence read processing, Sequence clustering, DNA barcoding</td>
      <td>2017-06-22</td>
      <td>2024-08-09</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>mothur_classify_seqs, mothur_pre_cluster, mothur_summary_seqs, humann2_renorm_table, metaphlan2, taxonomy_krona_chart, mothur_unique_seqs, mothur_count_seqs, mothur_make_shared, mothur_filter_seqs, mothur_make_group, mothur_classify_otu, humann2_regroup_table, mothur_screen_seqs, metaphlan2krona, krona-text, humann2, mothur_align_seqs, mothur_make_biom, mothur_cluster_split, mothur_merge_files</td>
      <td></td>
      <td>UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>5221</td>
      <td>7087</td>
      <td>2.883333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>16S Microbial analysis with Nanopore data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/nanopore-16S-metagenomics/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td>Taxonomic classification, Statistical calculation, Validation, Sequence contamination filtering, Sequencing quality control, Visualisation, Sequence composition calculation</td>
      <td>2020-11-24</td>
      <td>2024-07-31</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>fastp, multiqc, taxonomy_krona_chart, porechop, Remove beginning1, datamash_reverse, kraken2, tp_replace_in_line, fastqc</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>GalaxyTrakr, UseGalaxy.be, UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>4573</td>
      <td>6335</td>
      <td>2.733333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Metatranscriptomics analysis using microbiome RNA-seq data (short)</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metatranscriptomics-short/tutorial.html</td>
      <td>Metatranscriptomics, Microbial ecology, Taxonomy, Function analysis, Sequence analysis</td>
      <td>Sequence comparison, Conversion, Taxonomic classification, Validation, Sequence composition calculation, Sequencing quality control, Visualisation, Phylogenetic tree visualisation, Nucleic acid sequence analysis, Sequence alignment analysis, Sequence trimming, Primer removal, Phylogenetic analysis, Statistical calculation, Species frequency estimation, Phylogenetic tree analysis, Phylogenetic inference, Sequence similarity search, Phylogenetic tree editing, Read pre-processing</td>
      <td>2020-02-13</td>
      <td>2024-06-14</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>Grep1, humann, multiqc, humann_unpack_pathways, cutadapt, fastqc, bg_sortmerna, humann_renorm_table, taxonomy_krona_chart, metaphlan, fastq_paired_end_interlacer, Cut1, tp_find_and_replace, humann_regroup_table, export2graphlan, humann_rename_table, combine_metaphlan2_humann2, graphlan_annotate, graphlan, humann_split_stratified_table</td>
      <td>UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>417</td>
      <td>531</td>
      <td>1.550000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Assembly of metagenomic sequencing data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metagenomics-assembly/tutorial.html</td>
      <td>Metagenomics, Sequence assembly</td>
      <td>Data handling, Variant calling, Sequence contamination filtering, Sequencing quality control, Visualisation, Sequence composition calculation, Local alignment, Sequence assembly validation, Sequence alignment analysis, Genome assembly, Sequence trimming, Primer removal, Formatting, Sequence file editing, Read mapping, Statistical calculation, Sequence assembly visualisation, Read pre-processing</td>
      <td>2022-12-05</td>
      <td>2024-11-26</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>tp_cat, bamtools, bowtie2, quast, megahit_contig2fastg, megahit, coverm_contig, ngsutils_bam_filter, bandage_image, seqtk_subseq, cutadapt, bg_uniq, filter_tabular, metaspades, collection_column_join, fastqc, bandage_info, random_lines1</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td>0</td>
      <td></td>
      <td>2712</td>
      <td>3849</td>
      <td>2.933333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Taxonomic Profiling and Visualization of Metagenomic Data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/taxonomic-profiling/tutorial.html</td>
      <td>Metagenomics, Microbial ecology, Taxonomy, Sequence analysis</td>
      <td>Statistical calculation, Taxonomic classification, Phylogenetic tree analysis, Visualisation, Genome annotation, Nucleic acid sequence analysis, Aggregation</td>
      <td>2023-05-03</td>
      <td>2024-06-14</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>interactive_tool_phinch, taxonomy_krona_chart, __UNZIP_COLLECTION__, est_abundance, kraken_biom, kraken2, metaphlan, interactive_tool_pavian, krakentools_kreport2krona</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.fr, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>6850</td>
      <td>9276</td>
      <td>1.950000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Antibiotic resistance detection</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/plasmid-metagenomics-nanopore/tutorial.html</td>
      <td>Metagenomic sequencing, Public health and epidemiology, Metagenomics, Microbiology, Infectious disease, Sequence analysis, Antimicrobial resistance</td>
      <td>Scatter plot plotting, Genome assembly, Antimicrobial resistance prediction, De-novo assembly, Mapping assembly, Box-Whisker plot plotting, Sequence analysis, Sequence assembly visualisation, Pairwise sequence alignment, Aggregation</td>
      <td>2019-06-25</td>
      <td>2024-06-14</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>PlasFlow, miniasm, racon, nanoplot, unicycler, minimap2, bandage_image, staramr_search, gfa_to_fa</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>2260</td>
      <td>3231</td>
      <td>3.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Identification of the micro-organisms in a beer using Nanopore sequencing</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/beer-data-analysis/tutorial.html</td>
      <td>Metagenomics, Microbial ecology, Taxonomy, Sequence analysis</td>
      <td>Taxonomic classification, Statistical calculation, Sequence contamination filtering, Sequencing quality control, Visualisation, Sequence composition calculation, Aggregation</td>
      <td>2022-09-29</td>
      <td>2024-11-29</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>fastp, taxonomy_krona_chart, porechop, Filter1, kraken2, fastqc, krakentools_kreport2krona</td>
      <td>GalaxyTrakr, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>1633</td>
      <td>2465</td>
      <td>3.050000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Metatranscriptomics analysis using microbiome RNA-seq data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metatranscriptomics/tutorial.html</td>
      <td>Metatranscriptomics, Microbial ecology, Taxonomy, Function analysis, Sequence analysis</td>
      <td>Sequence comparison, Conversion, Taxonomic classification, Validation, Sequence composition calculation, Sequencing quality control, Visualisation, Phylogenetic tree visualisation, Nucleic acid sequence analysis, Sequence alignment analysis, Sequence trimming, Primer removal, Phylogenetic analysis, Statistical calculation, Species frequency estimation, Phylogenetic tree analysis, Phylogenetic inference, Sequence similarity search, Phylogenetic tree editing, Read pre-processing</td>
      <td>2019-11-21</td>
      <td>2024-06-14</td>
      <td>4</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>Grep1, humann, multiqc, humann_unpack_pathways, cutadapt, fastqc, bg_sortmerna, humann_renorm_table, taxonomy_krona_chart, metaphlan, fastq_paired_end_interlacer, Cut1, tp_find_and_replace, humann_regroup_table, export2graphlan, humann_rename_table, combine_metaphlan2_humann2, graphlan_annotate, graphlan, humann_split_stratified_table</td>
      <td>UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2357217</td>
      <td>6456997</td>
      <td>4.416667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Pathogen detection from (direct Nanopore) sequencing data using Galaxy - Foodborne Edition</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.html</td>
      <td>Metagenomics, Public health and epidemiology, Taxonomy, Sequence assembly, Pathology, Sequence analysis</td>
      <td>Scatter plot plotting, Box-Whisker plot plotting, Data handling, Phylogenetic tree generation (from molecular sequences), Multilocus sequence typing, Variant calling, Multiple sequence alignment, Taxonomic classification, Validation, Sequence composition calculation, Sequencing quality control, Visualisation, Sequence contamination filtering, Cross-assembly, Phylogenetic tree generation (maximum likelihood and Bayesian methods), Sequence alignment analysis, Genome assembly, Antimicrobial resistance prediction, De-novo assembly, Mapping assembly, Phylogenetic tree generation, Sequence assembly, Base-calling, Statistical calculation, Mapping, Sequence assembly visualisation, Phylogenetic tree analysis, Phylogenetic tree reconstruction, Pairwise sequence alignment, Aggregation</td>
      <td>2023-01-26</td>
      <td>2024-09-27</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>__FILTER_FAILED_DATASETS__, collection_element_identifiers, CONVERTER_gz_to_uncompressed, regexColumn1, samtools_coverage, param_value_from_file, snpSift_filter, tp_cut_tool, Grep1, multiqc, Count1, fasta2tab, krakentools_extract_kraken_reads, kraken2, tp_replace_in_column, fastqc, krakentools_kreport2krona, clair3, abricate, fasttree, taxonomy_krona_chart, split_file_to_collection, add_line_to_file, bandage_image, Remove beginning1, bcftools_norm, tp_split_on_column, bamtools_split_mapped, collection_column_join, compose_text_param, fasta_merge_files_and_filter_unique_sequences, bcftools_consensus, Cut1, clustalw, bedtools_getfastabed, tp_find_and_replace, tp_multijoin_tool, flye, minimap2, tab2fasta, newick_display, Grouping1, Paste1, medaka_consensus_pipeline, collapse_dataset, tp_sorted_uniq, ggplot2_heatmap, samtools_depth, fastp, tp_head_tool, Add_a_column1, samtools_fastx, barchart_gnuplot, nanoplot, porechop, mlst, table_compute, regex1, __FILTER_EMPTY_DATASETS__, __BUILD_LIST__, snpSift_extractFields</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>3022</td>
      <td>4478</td>
      <td>2.933333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Building an amplicon sequence variant (ASV) table from 16S data using DADA2</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/dada-16S/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td>Analysis, Visualisation, Deposition, Variant calling, DNA barcoding</td>
      <td>2024-06-05</td>
      <td>2024-12-05</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>collection_element_identifiers, dada2_filterAndTrim, dada2_plotQualityProfile, interactive_tool_phyloseq, dada2_dada, tp_replace_in_column, dada2_assignTaxonomyAddspecies, phyloseq_from_dada2, dada2_removeBimeraDenovo, __UNZIP_COLLECTION__, dada2_makeSequenceTable, Grouping1, dada2_seqCounts, tp_replace_in_line, tp_head_tool, dada2_mergePairs, __SORTLIST__, Add_a_column1, cat1, dada2_learnErrors</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2834</td>
      <td>3830</td>
      <td>2.033333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Binning of metagenomic sequencing data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metagenomics-binning/tutorial.html</td>
      <td>Metagenomics, Sequence assembly</td>
      <td>Statistical calculation, Genome assembly, Validation, Sequence composition calculation, Sequencing quality control, Sequence assembly validation</td>
      <td>2023-12-05</td>
      <td>2024-09-12</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>checkm_lineage_wf, megahit</td>
      <td>GalaxyTrakr, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr</td>
      <td>UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>3276</td>
      <td>3997</td>
      <td>1.400000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>QIIME 2 Moving Pictures</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/qiime2-moving-pictures/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td></td>
      <td>2024-03-14</td>
      <td>2024-03-14</td>
      <td>1</td>
      <td>external</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>248</td>
      <td>383</td>
      <td>3.433333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>QIIME 2 Cancer Microbiome Intervention</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/qiime2-cancer-microbiome-intervention/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td></td>
      <td>2024-02-12</td>
      <td>2024-03-14</td>
      <td>3</td>
      <td>external</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>233</td>
      <td>409</td>
      <td>2.966667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>16S Microbial Analysis with mothur (short)</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/mothur-miseq-sop-short/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td>Phylogenetic tree generation, Phylogenetic analysis, Taxonomic classification, Phylogenetic tree analysis, Phylogenetic tree reconstruction, Sequencing quality control, Visualisation, Sequence read processing, Sequence clustering, DNA barcoding</td>
      <td>2019-05-13</td>
      <td>2024-06-14</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>mothur_cluster, mothur_chimera_vsearch, mothur_summary_single, mothur_venn, mothur_classify_seqs, mothur_tree_shared, mothur_pre_cluster, mothur_summary_seqs, mothur_unique_seqs, mothur_rarefaction_single, mothur_count_seqs, mothur_make_shared, mothur_dist_seqs, mothur_remove_groups, mothur_filter_seqs, mothur_classify_otu, mothur_screen_seqs, mothur_count_groups, newick_display, collapse_dataset, mothur_heatmap_sim, mothur_dist_shared, mothur_remove_seqs, mothur_cluster_split, mothur_sub_sample, XY_Plot_1, mothur_remove_lineage</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>3159</td>
      <td>5444</td>
      <td>5.216667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>16S Microbial Analysis with mothur (extended)</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/mothur-miseq-sop/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td>Phylogenetic tree generation, Phylogenetic analysis, Taxonomic classification, Phylogenetic tree analysis, Phylogenetic tree reconstruction, Sequencing quality control, Visualisation, Sequence read processing, Sequence clustering, DNA barcoding</td>
      <td>2017-02-12</td>
      <td>2024-03-14</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>mothur_get_groups, mothur_cluster, mothur_chimera_vsearch, mothur_summary_single, mothur_venn, mothur_classify_seqs, mothur_tree_shared, mothur_pre_cluster, mothur_summary_seqs, taxonomy_krona_chart, mothur_taxonomy_to_krona, mothur_unique_seqs, mothur_rarefaction_single, mothur_count_seqs, mothur_make_shared, mothur_dist_seqs, mothur_filter_seqs, mothur_remove_groups, mothur_classify_otu, mothur_screen_seqs, mothur_count_groups, newick_display, mothur_heatmap_sim, mothur_dist_shared, mothur_seq_error, mothur_remove_seqs, mothur_align_seqs, mothur_make_biom, mothur_cluster_split, mothur_sub_sample, XY_Plot_1, mothur_make_contigs, mothur_remove_lineage</td>
      <td>UseGalaxy.be, UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>7846</td>
      <td>11724</td>
      <td>2.916667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>M. tuberculosis Variant Analysis</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/tb-variant-analysis/tutorial.html</td>
      <td>Genomics, Microbiology, Sequence assembly, Genetic variation, Public health and epidemiology, Infectious disease</td>
      <td>Antimicrobial resistance prediction, Global alignment, Phylogenetic tree generation, Variant calling, Sequence alignment, Sequence analysis, Taxonomic classification, Statistical calculation, Validation, Sequence composition calculation, Sequencing quality control, Phylogenetic tree visualisation, Sequence contamination filtering, Genome visualisation, Local alignment, Sequence alignment analysis</td>
      <td>2020-07-25</td>
      <td>2024-07-26</td>
      <td>30</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>tp_awk_tool, jvarkit_wgscoverageplotter, tb_variant_filter, multiqc, snippy, kraken2, __FLATTEN__, EMBOSS: seqret84, fastqc, tbvcfreport, mosdepth, qualimap_bamqc, upload1, tp_sed_tool, tb_profiler_profile, bcftools_consensus, samtools_stats, jbrowse, fastp, EMBOSS:%20seqret84</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>6285</td>
      <td>9793</td>
      <td>2.866667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>metaQuantome 2: Function</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/metaquantome-function/tutorial.html</td>
      <td>Proteomics, Proteogenomics, Metatranscriptomics, Microbial ecology, Metagenomics</td>
      <td>Statistical inference, Query and retrieval, Functional clustering, Visualisation, Heat map generation, Differential protein expression analysis, Quantification, Principal component visualisation, Filtering, Indexing</td>
      <td>2020-10-29</td>
      <td>2024-03-15</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>metaquantome_filter, metaquantome_stat, metaquantome_expand, metaquantome_db, metaquantome_sample, metaquantome_viz</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>380</td>
      <td>543</td>
      <td>4.383333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>metaQuantome 1: Data creation</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/metaquantome-data-creation/tutorial.html</td>
      <td>Proteomics, Proteogenomics, Biodiversity, Taxonomy</td>
      <td>Label-free quantification, Visualisation, Formatting, Filtering, Prediction and recognition</td>
      <td>2020-10-16</td>
      <td>2024-03-14</td>
      <td>20</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>query_tabular, search_gui, msconvert, Grep1, unipept, tp_replace_in_line, Remove beginning1, Filter1, flashlfq, peptide_shaker, regex1, tp_replace_in_column, Cut1</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>818</td>
      <td>1419</td>
      <td>3.816667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Metaproteomics tutorial</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/metaproteomics/tutorial.html</td>
      <td>Proteomics, Proteogenomics, Biodiversity, Taxonomy</td>
      <td>Visualisation, Prediction and recognition</td>
      <td>2017-06-28</td>
      <td>2024-06-14</td>
      <td>41</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>query_tabular, unipept, sqlite_to_tabular, peptide_shaker, search_gui</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>2529</td>
      <td>4128</td>
      <td>2.750000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>metaQuantome 3: Taxonomy</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/metaquantome-taxonomy/tutorial.html</td>
      <td>Proteomics, Proteogenomics, Metatranscriptomics, Microbial ecology, Metagenomics, Taxonomy</td>
      <td>Statistical inference, Query and retrieval, Functional clustering, Visualisation, Heat map generation, Differential protein expression analysis, Quantification, Principal component visualisation, Filtering, Indexing</td>
      <td>2020-10-29</td>
      <td>2024-03-15</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>metaquantome_filter, metaquantome_stat, metaquantome_expand, metaquantome_db, metaquantome_sample, metaquantome_viz</td>
      <td>UseGalaxy.be, UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>398</td>
      <td>531</td>
      <td>2.616667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Sequence analysis</td>
      <td>Quality and contamination control in bacterial isolate using Illumina MiSeq Data</td>
      <td>https://training.galaxyproject.org//topics/sequence-analysis/tutorials/quality-contamination-control/tutorial.html</td>
      <td>Whole genome sequencing, Genomics, Microbiology, Microbial ecology</td>
      <td>Expression analysis, Taxonomic classification, Statistical calculation, Sequence contamination filtering, Sequencing quality control, Sequence composition calculation, Cross-assembly</td>
      <td>2024-07-15</td>
      <td>2024-10-09</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>fastp, recentrifuge, est_abundance, kraken2, upload1, fastqc</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td>Galaxy@AuBi, GalaxyTrakr, UseGalaxy.cz, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>705</td>
      <td>932</td>
      <td>4.150000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Calling variants in non-diploid systems</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/non-dip/tutorial.html</td>
      <td>Genomics, Sequence assembly, DNA polymorphism, Microbiology, Sequence analysis, Genetic variation</td>
      <td>Data handling, Formatting, Sequence alignment, Variant calling, Genome indexing, Read mapping, Statistical calculation, Generation, Sequence composition calculation, Sequencing quality control, Sequence alignment analysis</td>
      <td>2017-02-16</td>
      <td>2024-03-15</td>
      <td>23</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>bamleftalign, bwa_mem, picard_MergeSamFiles, freebayes, vcffilter2, picard_MarkDuplicates, bamFilter, vcf2tsv, fastqc, Cut1</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>7696</td>
      <td>14362</td>
      <td>2.166667</td>
      <td>0</td>
    </tr>
  </tbody>
</table>

</div>