<script src="https://code.jquery.com/jquery-3.7.0.js"></script>
<script src="https://cdn.datatables.net/1.13.6/js/jquery.dataTables.min.js"></script>
<script src="https://cdn.datatables.net/searchbuilder/1.6.0/js/dataTables.searchBuilder.min.js"></script>
<script src="https://cdn.datatables.net/datetime/1.5.1/js/dataTables.dateTime.min.js"></script>

<script src="https://cdn.datatables.net/responsive/2.5.0/js/dataTables.responsive.min.js"></script>

<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/responsive/2.2.3/css/responsive.dataTables.min.css">

<link rel="stylesheet" href="https://cdn.datatables.net/1.13.6/css/jquery.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/searchbuilder/1.6.0/css/searchBuilder.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/datetime/1.5.1/css/dataTables.dateTime.min.css">

<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" />
<script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>

<style type="text/css">
  .tooltip {
    font-family: Georgia;
    font-size: 18px;
  }
  .tooltip .tooltip-inner {
    background-color: #00FFFF;
    color: #000000;
    min-width: 400px;
  }
  </style>

<script>
    $(document).ready(function() {
    $('#dataframe').DataTable( {
        "sScrollX": "100%",
        "dom": "Qlfrtip",

        responsive: {
            details: {
                type: 'column'
            }
        },
        "bAutoWidth": false,
        columnDefs: [ {
            className: 'control',
            orderable: false,
            targets:   [ 0 ],
            width: 100,
        },
        { type: 'natural', targets: 1 } // allow normal sorting
        ],

    });
});
</script>

<div class="display nowrap" style="width:80%" width="100%">

<table class="dataframe display nowrap" id="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th>Expand</th>
      <th>Topic</th>
      <th>Title</th>
      <th>Link</th>
      <th>EDAM topic</th>
      <th>EDAM operation</th>
      <th>Creation</th>
      <th>Last modification</th>
      <th>Version</th>
      <th>Tutorial</th>
      <th>Slides</th>
      <th>Video</th>
      <th>Workflows</th>
      <th>Tools</th>
      <th>Servers with precise tool versions</th>
      <th>Servers with tool but different versions</th>
      <th>Feedback number</th>
      <th>Feedback mean note</th>
      <th>Visitors</th>
      <th>Page views</th>
      <th>Visit duration</th>
      <th>Video views</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Making sense of a newly assembled genome</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/ecoli_comparison/tutorial.html</td>
      <td>Sequence assembly, Genomics, Microbiology</td>
      <td>Sequence alignment, Sequence visualisation, Genome visualisation, Read mapping, Mapping</td>
      <td>2018-06-14</td>
      <td>2024-06-24</td>
      <td>29</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>jbrowse, tp_sort_header_tool, tp_sed_tool, tp_replace_in_column, tp_grep_tool, bedtools_intersectbed, circos_interval_to_tiles, lastz_wrapper_2, cat1, circos, addValue, circos_aln_to_links, join1, collapse_dataset, Filter1, Grep1, Cut1, datamash_ops, mergeCols1, bedtools_sortbed, fasta_compute_length, tp_replace_in_line, random_lines1, fasta_filter_by_length, tp_cat, upload1, bedtools_complementbed</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>6292</td>
      <td>12478</td>
      <td>3.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Assembly of metagenomic sequencing data</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/metagenomics-assembly/tutorial.html</td>
      <td>Metagenomics, Sequence assembly</td>
      <td>Sequence assembly visualisation, Visualisation, Sequence trimming, Sequence composition calculation, Statistical calculation, Read mapping, Sequence contamination filtering, Sequence assembly validation, Primer removal, Data handling, Sequence file editing, Formatting, Sequencing quality control, Variant calling, Read pre-processing, Genome assembly, Local alignment, Sequence alignment analysis</td>
      <td>2025-02-12</td>
      <td>2025-02-12</td>
      <td>0</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>seqtk_subseq, collection_column_join, coverm_contig, ngsutils_bam_filter, random_lines1, bandage_image, fastqc, bg_uniq, bandage_info, quast, tp_cat, filter_tabular, metaspades, bowtie2, megahit_contig2fastg, megahit, bamtools, cutadapt</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td>0</td>
      <td></td>
      <td>5</td>
      <td>9</td>
      <td>0.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Unicycler Assembly</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/unicycler-assembly/tutorial.html</td>
      <td>Sequence assembly, Genomics, Microbiology</td>
      <td>Visualisation, Genome assembly, Sequencing quality control, Sequence composition calculation, Aggregation, Genome annotation, Statistical calculation, Coding region prediction, Validation, Gene prediction, Sequence assembly validation</td>
      <td>2017-10-11</td>
      <td>2024-03-13</td>
      <td>23</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>quast, prokka, fastqc, multiqc, unicycler</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>GalaxyTrakr, MISSISSIPPI, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2347499</td>
      <td>6427397</td>
      <td>4.600000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Genome Assembly of MRSA from Oxford Nanopore MinION data (and optionally Illumina data)</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/mrsa-nanopore/tutorial.html</td>
      <td>Sequence assembly, Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology, Antimicrobial resistance</td>
      <td>Sequence assembly visualisation, Filtering, De-novo assembly, Cross-assembly, Visualisation, Genome assembly, Sequencing quality control, Scatter plot plotting, Sequence composition calculation, Box-Whisker plot plotting, Sequence alignment, Statistical calculation, Sequencing error detection, Read mapping, Sequence contamination filtering, Mapping assembly, Sequence assembly validation</td>
      <td>2021-03-24</td>
      <td>2024-11-29</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>nanoplot, flye, quast, bandage_image, fastqc, filtlong, porechop, polypolish, CONVERTER_bz2_to_uncompressed, upload1, bwa_mem2, fastp</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>8190</td>
      <td>12391</td>
      <td>3.350000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Genome Assembly of a bacterial genome (MRSA) sequenced using Illumina MiSeq Data</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/mrsa-illumina/tutorial.html</td>
      <td>Sequence assembly, Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology</td>
      <td>Sequence assembly visualisation, Visualisation, Genome assembly, Sequencing quality control, Sequence composition calculation, Statistical calculation, Sequence contamination filtering, Sequence assembly validation</td>
      <td>2021-03-24</td>
      <td>2024-09-18</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>shovill, quast, bandage_image, fastqc, bandage_info, upload1, fastp</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>Galaxy@AuBi, GalaxyTrakr, UseGalaxy.be, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>7826</td>
      <td>12369</td>
      <td>3.666667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Checking expected species and contamination in bacterial isolate</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/bacterial-isolate-species-contamination-checking/tutorial.html</td>
      <td>Whole genome sequencing, Genomics, Microbiology, Microbial ecology</td>
      <td>Cross-assembly, Taxonomic classification, Statistical calculation, Expression analysis</td>
      <td>2024-03-04</td>
      <td>2024-06-27</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>kraken2, upload1, recentrifuge, est_abundance</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td>Galaxy@AuBi, GalaxyTrakr, UseGalaxy.cz, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>604</td>
      <td>841</td>
      <td>3.733333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Tree thinking for tuberculosis evolution and epidemiology</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/mtb_phylogeny/tutorial.html</td>
      <td>Genomics, Microbiology, Phylogeny, Infectious disease</td>
      <td>Sequence analysis, Phylogenetic tree analysis</td>
      <td>2022-03-16</td>
      <td>2024-03-12</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>raxml, upload1, interactive_tool_rstudio</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>1577</td>
      <td>2315</td>
      <td>2.166667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Identifying tuberculosis transmission links: from SNPs to transmission clusters</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/mtb_transmission/tutorial.html</td>
      <td>Genomics, Microbiology, Infectious disease, DNA polymorphism</td>
      <td>Phylogenetic tree generation, Phylogenetic tree visualisation, Antimicrobial resistance prediction, Variant calling</td>
      <td>2022-03-16</td>
      <td>2024-07-12</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>samtools_view, addName, tp_replace_in_line, tp_sed_tool, bcftools_consensus, snp_dists, Grep1, tp_grep_tool, tp_easyjoin_tool, upload1, tp_cat, tb_profiler_profile, trimmomatic, __MERGE_COLLECTION__, tb_variant_filter, snippy, snp_sites, collapse_dataset</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>1914</td>
      <td>2878</td>
      <td>2.833333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Genome annotation with Prokka</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/annotation-with-prokka/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis</td>
      <td>Genome visualisation, Genome annotation, Coding region prediction, Gene prediction</td>
      <td>2018-03-06</td>
      <td>2024-08-08</td>
      <td>25</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>jbrowse, prokka</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>Galaxy@AuBi, GalaxyTrakr, MISSISSIPPI, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>2352560</td>
      <td>6435878</td>
      <td>4.383333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Identification of AMR genes in an assembled bacterial genome</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/amr-gene-detection/tutorial.html</td>
      <td>Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology, Sequence analysis, Infectious disease, Antimicrobial resistance</td>
      <td>Genome visualisation, Read mapping, Genome annotation</td>
      <td>2024-01-23</td>
      <td>2024-10-15</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>jbrowse, Grep1, bakta, bowtie2, staramr_search, upload1, tbl2gff3</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr</td>
      <td>Galaxy@AuBi, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>2000</td>
      <td>2797</td>
      <td>3.116667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Refining Genome Annotations with Apollo (prokaryotes)</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/apollo/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis</td>
      <td>Genome visualisation</td>
      <td>2021-06-04</td>
      <td>2024-10-15</td>
      <td>20</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>jbrowse, list_organism, create_or_update, create_account, iframe</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2110106</td>
      <td>5629497</td>
      <td>5.266667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Bacterial Genome Annotation</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/bacterial-genome-annotation/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis, Whole genome sequencing, Functional genomics, Mobile genetic elements</td>
      <td>Multilocus sequence typing, Nucleic acid feature detection, Sequence motif recognition, Genome visualisation, Protein feature detection, Genome assembly, Genome annotation, Scaffolding, Structural variation detection</td>
      <td>2024-02-01</td>
      <td>2024-10-15</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>jbrowse, tp_tail_tool, tp_replace_in_column, Grouping1, plasmidfinder, integron_finder, bakta, upload1, tbl2gff3, isescan</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td>Galaxy@AuBi, UseGalaxy.cz, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>3738</td>
      <td>5107</td>
      <td>2.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Comparative gene analysis in unannotated genomes</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/gene-centric/tutorial.html</td>
      <td>Genomics, Gene and protein families, Sequence analysis, Phylogeny</td>
      <td>Multiple sequence alignment, Coding region prediction, De-novo assembly, Phylogenetic tree generation, Transcriptome assembly, Database search, Sequence alignment analysis</td>
      <td>2022-09-08</td>
      <td>2024-03-18</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>bg_diamond_view, bg_diamond, tp_split_on_column, regexColumn1, bg_diamond_makedb, Filter1, Add_a_column1, Cut1, orfipy, rbc_mafft, rapidnj, tab2fasta, gops_intersect_1, join1, cat1, collapse_dataset</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>1259</td>
      <td>1857</td>
      <td>3.250000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Essential genes detection with Transposon insertion sequencing</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/tnseq/tutorial.html</td>
      <td>Genomics, Microbiology, Sequence analysis, Mobile genetic elements</td>
      <td>Read pre-processing, Primer removal, Transposon prediction, Sequence trimming</td>
      <td>2019-07-02</td>
      <td>2024-03-18</td>
      <td>12</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>bowtie_wrapper, tp_sort_header_tool, Filter1, Add_a_column1, deeptools_bam_coverage, Cut1, bg_find_subsequences, tp_easyjoin_tool, gff_to_prot, __EXTRACT_DATASET__, transit_gumbel, cutadapt</td>
      <td>UseGalaxy.be, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td>0</td>
      <td></td>
      <td>2342991</td>
      <td>6420405</td>
      <td>4.333333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Microbial Variant Calling</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/microbial-variants/tutorial.html</td>
      <td>Genomics, Sequence assembly, DNA polymorphism, Microbiology, Sequence analysis, Genetic variation</td>
      <td>Genome visualisation, Phylogenetic tree visualisation, Phylogenetic tree generation, Variant calling</td>
      <td>2018-02-26</td>
      <td>2024-03-15</td>
      <td>23</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>jbrowse, snippy</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>GalaxyTrakr, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>13332</td>
      <td>21732</td>
      <td>1.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Analyses of metagenomics data - The global picture</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/general-tutorial/tutorial.html</td>
      <td>Metagenomics, Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td>DNA barcoding, Taxonomic classification, Sequence clustering, Visualisation, Sequencing quality control, Sequence read processing, Phylogenetic analysis</td>
      <td>2017-06-22</td>
      <td>2024-08-09</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>humann2, humann2_regroup_table, mothur_pre_cluster, mothur_summary_seqs, mothur_screen_seqs, mothur_merge_files, krona-text, humann2_renorm_table, mothur_filter_seqs, mothur_unique_seqs, mothur_make_biom, mothur_align_seqs, mothur_classify_seqs, metaphlan2, mothur_classify_otu, metaphlan2krona, mothur_cluster_split, mothur_count_seqs, taxonomy_krona_chart, mothur_make_shared, mothur_make_group</td>
      <td></td>
      <td>UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>5111</td>
      <td>6960</td>
      <td>2.933333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Antibiotic resistance detection</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/plasmid-metagenomics-nanopore/tutorial.html</td>
      <td>Metagenomic sequencing, Public health and epidemiology, Metagenomics, Microbiology, Infectious disease, Sequence analysis, Antimicrobial resistance</td>
      <td>Sequence assembly visualisation, De-novo assembly, Genome assembly, Scatter plot plotting, Box-Whisker plot plotting, Aggregation, Sequence analysis, Pairwise sequence alignment, Mapping assembly</td>
      <td>2019-06-25</td>
      <td>2024-06-14</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>nanoplot, unicycler, racon, bandage_image, gfa_to_fa, PlasFlow, staramr_search, minimap2, miniasm</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>2212</td>
      <td>3165</td>
      <td>3.016667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Metatranscriptomics analysis using microbiome RNA-seq data (short)</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metatranscriptomics-short/tutorial.html</td>
      <td>Metatranscriptomics, Microbial ecology, Taxonomy, Function analysis, Sequence analysis</td>
      <td>Visualisation, Sequence trimming, Sequence composition calculation, Phylogenetic analysis, Phylogenetic tree visualisation, Species frequency estimation, Statistical calculation, Phylogenetic inference, Validation, Phylogenetic tree analysis, Nucleic acid sequence analysis, Taxonomic classification, Primer removal, Sequence comparison, Phylogenetic tree editing, Sequencing quality control, Read pre-processing, Conversion, Sequence similarity search, Sequence alignment analysis</td>
      <td>2020-02-13</td>
      <td>2024-06-14</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>humann_split_stratified_table, humann_renorm_table, fastqc, metaphlan, humann_regroup_table, fastq_paired_end_interlacer, graphlan_annotate, tp_find_and_replace, humann, export2graphlan, Grep1, Cut1, multiqc, graphlan, taxonomy_krona_chart, humann_rename_table, bg_sortmerna, humann_unpack_pathways, combine_metaphlan2_humann2, cutadapt</td>
      <td>UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>410</td>
      <td>523</td>
      <td>1.533333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Assembly of metagenomic sequencing data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metagenomics-assembly/tutorial.html</td>
      <td>Metagenomics, Sequence assembly</td>
      <td>Sequence assembly visualisation, Visualisation, Sequence trimming, Sequence composition calculation, Statistical calculation, Read mapping, Sequence contamination filtering, Sequence assembly validation, Primer removal, Data handling, Sequence file editing, Formatting, Sequencing quality control, Variant calling, Read pre-processing, Genome assembly, Local alignment, Sequence alignment analysis</td>
      <td>2022-12-05</td>
      <td>2024-11-26</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>seqtk_subseq, collection_column_join, coverm_contig, ngsutils_bam_filter, random_lines1, bandage_image, fastqc, bg_uniq, bandage_info, quast, tp_cat, filter_tabular, metaspades, bowtie2, megahit_contig2fastg, megahit, bamtools, cutadapt</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td>0</td>
      <td></td>
      <td>2674</td>
      <td>3803</td>
      <td>2.966667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Metatranscriptomics analysis using microbiome RNA-seq data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metatranscriptomics/tutorial.html</td>
      <td>Metatranscriptomics, Microbial ecology, Taxonomy, Function analysis, Sequence analysis</td>
      <td>Visualisation, Sequence trimming, Sequence composition calculation, Phylogenetic analysis, Phylogenetic tree visualisation, Species frequency estimation, Statistical calculation, Phylogenetic inference, Validation, Phylogenetic tree analysis, Nucleic acid sequence analysis, Taxonomic classification, Primer removal, Sequence comparison, Phylogenetic tree editing, Sequencing quality control, Read pre-processing, Conversion, Sequence similarity search, Sequence alignment analysis</td>
      <td>2019-11-21</td>
      <td>2024-06-14</td>
      <td>4</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>humann_split_stratified_table, humann_renorm_table, fastqc, metaphlan, humann_regroup_table, fastq_paired_end_interlacer, graphlan_annotate, tp_find_and_replace, humann, export2graphlan, Grep1, Cut1, multiqc, graphlan, taxonomy_krona_chart, humann_rename_table, bg_sortmerna, humann_unpack_pathways, combine_metaphlan2_humann2, cutadapt</td>
      <td>UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2339836</td>
      <td>6412298</td>
      <td>4.416667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Pathogen detection from (direct Nanopore) sequencing data using Galaxy - Foodborne Edition</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.html</td>
      <td>Metagenomics, Public health and epidemiology, Taxonomy, Sequence assembly, Pathology, Sequence analysis</td>
      <td>Multilocus sequence typing, Sequence assembly visualisation, De-novo assembly, Phylogenetic tree generation, Cross-assembly, Visualisation, Phylogenetic tree generation (maximum likelihood and Bayesian methods), Scatter plot plotting, Mapping, Box-Whisker plot plotting, Sequence composition calculation, Aggregation, Phylogenetic tree generation (from molecular sequences), Statistical calculation, Pairwise sequence alignment, Phylogenetic tree analysis, Base-calling, Validation, Phylogenetic tree reconstruction, Sequence contamination filtering, Sequence assembly, Taxonomic classification, Data handling, Genome assembly, Sequencing quality control, Variant calling, Multiple sequence alignment, Antimicrobial resistance prediction, Mapping assembly, Sequence alignment analysis</td>
      <td>2023-01-26</td>
      <td>2024-09-27</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>nanoplot, regex1, bedtools_getfastabed, regexColumn1, tp_replace_in_column, fasttree, fastqc, table_compute, tp_cut_tool, __BUILD_LIST__, kraken2, tp_head_tool, minimap2, krakentools_kreport2krona, fasta2tab, tab2fasta, param_value_from_file, krakentools_extract_kraken_reads, Count1, flye, bcftools_norm, mlst, tp_sorted_uniq, split_file_to_collection, ggplot2_heatmap, collapse_dataset, tp_find_and_replace, collection_column_join, __FILTER_FAILED_DATASETS__, collection_element_identifiers, compose_text_param, bcftools_consensus, newick_display, samtools_coverage, Grep1, Cut1, bamtools_split_mapped, fasta_merge_files_and_filter_unique_sequences, medaka_consensus_pipeline, porechop, clair3, snpSift_filter, clustalw, multiqc, taxonomy_krona_chart, fastp, tp_split_on_column, abricate, CONVERTER_gz_to_uncompressed, Remove beginning1, snpSift_extractFields, bandage_image, Add_a_column1, Grouping1, add_line_to_file, tp_multijoin_tool, barchart_gnuplot, __FILTER_EMPTY_DATASETS__, Paste1, samtools_fastx, samtools_depth</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>2958</td>
      <td>4403</td>
      <td>2.950000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>16S Microbial analysis with Nanopore data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/nanopore-16S-metagenomics/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td>Taxonomic classification, Statistical calculation, Validation, Visualisation, Sequence contamination filtering, Sequencing quality control, Sequence composition calculation</td>
      <td>2020-11-24</td>
      <td>2024-07-31</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tp_replace_in_line, datamash_reverse, Remove beginning1, fastqc, porechop, kraken2, multiqc, taxonomy_krona_chart, fastp</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>GalaxyTrakr, UseGalaxy.be, UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>4408</td>
      <td>6110</td>
      <td>2.733333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Taxonomic Profiling and Visualization of Metagenomic Data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/taxonomic-profiling/tutorial.html</td>
      <td>Metagenomics, Microbial ecology, Taxonomy, Sequence analysis</td>
      <td>Nucleic acid sequence analysis, Aggregation, Taxonomic classification, Statistical calculation, Visualisation, Phylogenetic tree analysis</td>
      <td>2023-05-03</td>
      <td>2024-06-14</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>interactive_tool_pavian, __UNZIP_COLLECTION__, kraken_biom, metaphlan, est_abundance, kraken2, krakentools_kreport2krona, taxonomy_krona_chart, interactive_tool_phinch</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.fr, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>6680</td>
      <td>9077</td>
      <td>1.950000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Building an amplicon sequence variant (ASV) table from 16S data using DADA2</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/dada-16S/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td>DNA barcoding, Analysis, Deposition, Visualisation, Variant calling</td>
      <td>2024-06-05</td>
      <td>2024-12-05</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>dada2_plotQualityProfile, tp_replace_in_column, dada2_assignTaxonomyAddspecies, tp_head_tool, dada2_dada, cat1, interactive_tool_phyloseq, dada2_makeSequenceTable, dada2_seqCounts, dada2_mergePairs, dada2_removeBimeraDenovo, collection_element_identifiers, __UNZIP_COLLECTION__, tp_replace_in_line, dada2_learnErrors, Add_a_column1, Grouping1, phyloseq_from_dada2, __SORTLIST__, dada2_filterAndTrim</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2683</td>
      <td>3641</td>
      <td>1.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Identification of the micro-organisms in a beer using Nanopore sequencing</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/beer-data-analysis/tutorial.html</td>
      <td>Metagenomics, Microbial ecology, Taxonomy, Sequence analysis</td>
      <td>Aggregation, Taxonomic classification, Statistical calculation, Visualisation, Sequence contamination filtering, Sequencing quality control, Sequence composition calculation</td>
      <td>2022-09-29</td>
      <td>2024-11-29</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>Filter1, fastqc, porechop, kraken2, krakentools_kreport2krona, taxonomy_krona_chart, fastp</td>
      <td>GalaxyTrakr, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>1565</td>
      <td>2377</td>
      <td>3.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>QIIME 2 Moving Pictures</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/qiime2-moving-pictures/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td></td>
      <td>2024-03-14</td>
      <td>2024-03-14</td>
      <td>1</td>
      <td>external</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>241</td>
      <td>373</td>
      <td>3.483333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>QIIME 2 Cancer Microbiome Intervention</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/qiime2-cancer-microbiome-intervention/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td></td>
      <td>2024-02-12</td>
      <td>2024-03-14</td>
      <td>3</td>
      <td>external</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>225</td>
      <td>395</td>
      <td>2.933333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Binning of metagenomic sequencing data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metagenomics-binning/tutorial.html</td>
      <td>Metagenomics, Sequence assembly</td>
      <td>Statistical calculation, Sequence assembly validation, Validation, Genome assembly, Sequencing quality control, Sequence composition calculation</td>
      <td>2023-12-05</td>
      <td>2024-09-12</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>checkm_lineage_wf, megahit</td>
      <td>GalaxyTrakr, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr</td>
      <td>UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>3188</td>
      <td>3895</td>
      <td>1.400000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>16S Microbial Analysis with mothur (extended)</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/mothur-miseq-sop/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td>DNA barcoding, Taxonomic classification, Phylogenetic tree generation, Sequence clustering, Visualisation, Sequencing quality control, Sequence read processing, Phylogenetic analysis, Phylogenetic tree reconstruction, Phylogenetic tree analysis</td>
      <td>2017-02-12</td>
      <td>2024-03-14</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>mothur_make_contigs, mothur_pre_cluster, mothur_dist_shared, mothur_summary_seqs, mothur_chimera_vsearch, mothur_screen_seqs, mothur_heatmap_sim, mothur_sub_sample, mothur_venn, mothur_dist_seqs, mothur_remove_lineage, mothur_filter_seqs, mothur_taxonomy_to_krona, mothur_tree_shared, mothur_make_biom, mothur_align_seqs, mothur_unique_seqs, mothur_remove_seqs, XY_Plot_1, mothur_count_groups, mothur_remove_groups, mothur_classify_seqs, mothur_seq_error, mothur_summary_single, mothur_classify_otu, mothur_cluster_split, newick_display, mothur_cluster, mothur_count_seqs, mothur_rarefaction_single, taxonomy_krona_chart, mothur_get_groups, mothur_make_shared</td>
      <td>UseGalaxy.be, UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>7649</td>
      <td>11466</td>
      <td>2.933333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>16S Microbial Analysis with mothur (short)</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/mothur-miseq-sop-short/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td>DNA barcoding, Taxonomic classification, Phylogenetic tree generation, Sequence clustering, Visualisation, Sequencing quality control, Sequence read processing, Phylogenetic analysis, Phylogenetic tree reconstruction, Phylogenetic tree analysis</td>
      <td>2019-05-13</td>
      <td>2024-06-14</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>mothur_pre_cluster, mothur_dist_shared, mothur_summary_seqs, mothur_chimera_vsearch, mothur_screen_seqs, mothur_heatmap_sim, mothur_sub_sample, mothur_venn, mothur_dist_seqs, mothur_remove_lineage, mothur_filter_seqs, mothur_tree_shared, mothur_unique_seqs, mothur_remove_seqs, collapse_dataset, XY_Plot_1, mothur_count_groups, mothur_remove_groups, mothur_classify_seqs, newick_display, mothur_summary_single, mothur_classify_otu, mothur_cluster_split, mothur_cluster, mothur_count_seqs, mothur_rarefaction_single, mothur_make_shared</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>3085</td>
      <td>5351</td>
      <td>5.250000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>M. tuberculosis Variant Analysis</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/tb-variant-analysis/tutorial.html</td>
      <td>Genomics, Microbiology, Sequence assembly, Genetic variation, Public health and epidemiology, Infectious disease</td>
      <td>Taxonomic classification, Genome visualisation, Phylogenetic tree generation, Sequencing quality control, Variant calling, Sequence composition calculation, Sequence alignment, Phylogenetic tree visualisation, Local alignment, Sequence analysis, Global alignment, Statistical calculation, Validation, Antimicrobial resistance prediction, Sequence contamination filtering, Sequence alignment analysis</td>
      <td>2020-07-25</td>
      <td>2024-07-26</td>
      <td>30</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>jbrowse, tp_sed_tool, EMBOSS: seqret84, fastqc, kraken2, tbvcfreport, tp_awk_tool, upload1, jvarkit_wgscoverageplotter, tb_variant_filter, mosdepth, EMBOSS:%20seqret84, bcftools_consensus, qualimap_bamqc, snippy, multiqc, fastp, samtools_stats, tb_profiler_profile, __FLATTEN__</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>6235</td>
      <td>9734</td>
      <td>2.883333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>metaQuantome 3: Taxonomy</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/metaquantome-taxonomy/tutorial.html</td>
      <td>Proteomics, Proteogenomics, Metatranscriptomics, Microbial ecology, Metagenomics, Taxonomy</td>
      <td>Visualisation, Functional clustering, Indexing, Quantification, Filtering, Query and retrieval, Differential protein expression analysis, Principal component visualisation, Heat map generation, Statistical inference</td>
      <td>2020-10-29</td>
      <td>2024-03-15</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>metaquantome_stat, metaquantome_db, metaquantome_expand, metaquantome_sample, metaquantome_viz, metaquantome_filter</td>
      <td>UseGalaxy.be, UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>398</td>
      <td>531</td>
      <td>2.616667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Metaproteomics tutorial</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/metaproteomics/tutorial.html</td>
      <td>Proteomics, Proteogenomics, Biodiversity, Taxonomy</td>
      <td>Prediction and recognition, Visualisation</td>
      <td>2017-06-28</td>
      <td>2024-06-14</td>
      <td>41</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>search_gui, unipept, peptide_shaker, sqlite_to_tabular, query_tabular</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>2513</td>
      <td>4109</td>
      <td>2.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>metaQuantome 2: Function</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/metaquantome-function/tutorial.html</td>
      <td>Proteomics, Proteogenomics, Metatranscriptomics, Microbial ecology, Metagenomics</td>
      <td>Visualisation, Functional clustering, Indexing, Quantification, Filtering, Query and retrieval, Differential protein expression analysis, Principal component visualisation, Heat map generation, Statistical inference</td>
      <td>2020-10-29</td>
      <td>2024-03-15</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>metaquantome_stat, metaquantome_db, metaquantome_expand, metaquantome_sample, metaquantome_viz, metaquantome_filter</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>380</td>
      <td>543</td>
      <td>4.383333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>metaQuantome 1: Data creation</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/metaquantome-data-creation/tutorial.html</td>
      <td>Proteomics, Proteogenomics, Biodiversity, Taxonomy</td>
      <td>Prediction and recognition, Filtering, Label-free quantification, Visualisation, Formatting</td>
      <td>2020-10-16</td>
      <td>2024-03-14</td>
      <td>20</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tp_replace_in_line, msconvert, regex1, Remove beginning1, tp_replace_in_column, Filter1, search_gui, unipept, Grep1, Cut1, peptide_shaker, query_tabular, flashlfq</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>816</td>
      <td>1416</td>
      <td>3.833333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Sequence analysis</td>
      <td>Quality and contamination control in bacterial isolate using Illumina MiSeq Data</td>
      <td>https://training.galaxyproject.org//topics/sequence-analysis/tutorials/quality-contamination-control/tutorial.html</td>
      <td>Whole genome sequencing, Genomics, Microbiology, Microbial ecology</td>
      <td>Taxonomic classification, Statistical calculation, Cross-assembly, Sequence contamination filtering, Sequencing quality control, Expression analysis, Sequence composition calculation</td>
      <td>2024-07-15</td>
      <td>2024-10-09</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>fastqc, recentrifuge, est_abundance, kraken2, upload1, fastp</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td>Galaxy@AuBi, GalaxyTrakr, UseGalaxy.cz, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>664</td>
      <td>871</td>
      <td>4.050000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Calling variants in non-diploid systems</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/non-dip/tutorial.html</td>
      <td>Genomics, Sequence assembly, DNA polymorphism, Microbiology, Sequence analysis, Genetic variation</td>
      <td>Generation, Data handling, Genome indexing, Formatting, Sequencing quality control, Variant calling, Sequence composition calculation, Sequence alignment, Statistical calculation, Read mapping, Sequence alignment analysis</td>
      <td>2017-02-16</td>
      <td>2024-03-15</td>
      <td>23</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>vcf2tsv, bamleftalign, fastqc, bamFilter, Cut1, picard_MarkDuplicates, freebayes, picard_MergeSamFiles, vcffilter2, bwa_mem</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>7661</td>
      <td>14311</td>
      <td>2.166667</td>
      <td>0</td>
    </tr>
  </tbody>
</table>

</div>