<script src="https://code.jquery.com/jquery-3.7.0.js"></script>
<script src="https://cdn.datatables.net/1.13.6/js/jquery.dataTables.min.js"></script>
<script src="https://cdn.datatables.net/searchbuilder/1.6.0/js/dataTables.searchBuilder.min.js"></script>
<script src="https://cdn.datatables.net/datetime/1.5.1/js/dataTables.dateTime.min.js"></script>

<script src="https://cdn.datatables.net/responsive/2.5.0/js/dataTables.responsive.min.js"></script>

<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/responsive/2.2.3/css/responsive.dataTables.min.css">

<link rel="stylesheet" href="https://cdn.datatables.net/1.13.6/css/jquery.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/searchbuilder/1.6.0/css/searchBuilder.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/datetime/1.5.1/css/dataTables.dateTime.min.css">

<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" />
<script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>

<style type="text/css">
  .tooltip {
    font-family: Georgia;
    font-size: 18px;
  }
  .tooltip .tooltip-inner {
    background-color: #00FFFF;
    color: #000000;
    min-width: 400px;
  }
  </style>

<script>
    $(document).ready(function() {
    $('#dataframe').DataTable( {
        "sScrollX": "100%",
        "dom": "Qlfrtip",

        responsive: {
            details: {
                type: 'column'
            }
        },
        "bAutoWidth": false,
        columnDefs: [ {
            className: 'control',
            orderable: false,
            targets:   [ 0 ],
            width: 100,
        },
        { type: 'natural', targets: 1 } // allow normal sorting
        ],

    });
});
</script>

<div class="display nowrap" style="width:80%" width="100%">

<table class="dataframe display nowrap" id="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th>Expand</th>
      <th>Topic</th>
      <th>Title</th>
      <th>Link</th>
      <th>EDAM topic</th>
      <th>EDAM operation</th>
      <th>Creation</th>
      <th>Last modification</th>
      <th>Version</th>
      <th>Tutorial</th>
      <th>Slides</th>
      <th>Video</th>
      <th>Workflows</th>
      <th>Tools</th>
      <th>Servers with precise tool versions</th>
      <th>Servers with tool but different versions</th>
      <th>Feedback number</th>
      <th>Feedback mean note</th>
      <th>Visitors</th>
      <th>Page views</th>
      <th>Visit duration</th>
      <th>Video views</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Unicycler Assembly</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/unicycler-assembly/tutorial.html</td>
      <td>Sequence assembly, Genomics, Microbiology</td>
      <td>Validation, Coding region prediction, Visualisation, Aggregation, Sequence composition calculation, Sequencing quality control, Sequence assembly validation, Genome assembly, Statistical calculation, Genome annotation, Gene prediction</td>
      <td>2017-10-11</td>
      <td>2025-05-15</td>
      <td>24</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>prokka, fastqc, multiqc, quast, unicycler</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>GalaxyTrakr, MISSISSIPPI, UseGalaxy.be, UseGalaxy.fr, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2547668</td>
      <td>6937019</td>
      <td>4.650000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Genome Assembly of a bacterial genome (MRSA) sequenced using Illumina MiSeq Data</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/mrsa-illumina/tutorial.html</td>
      <td>Sequence assembly, Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology</td>
      <td>Visualisation, Sequence composition calculation, Sequence assembly visualisation, Sequencing quality control, Read mapping, Sequence assembly validation, Genome assembly, Statistical calculation, Sequence contamination filtering</td>
      <td>2021-03-24</td>
      <td>2025-03-20</td>
      <td>16</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>shovill, fastqc, quast, fastp, bandage_image, falco, upload1, bandage_info</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>8844</td>
      <td>13817</td>
      <td>3.716667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Assembly of metagenomic sequencing data</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/metagenomics-assembly/tutorial.html</td>
      <td>Metagenomics, Sequence assembly</td>
      <td>Visualisation, Data handling, Variant calling, Sequencing quality control, Sequence assembly validation, Primer removal, Local alignment, Genome assembly, Formatting, Sequence file editing, Sequence alignment analysis, Sequence trimming, Sequence composition calculation, Sequence assembly visualisation, Read mapping, Read pre-processing, Statistical calculation, Sequence contamination filtering</td>
      <td>2025-06-02</td>
      <td>2025-06-02</td>
      <td>0</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>fastqc, bg_uniq, bamtools, megahit_contig2fastg, random_lines1, megahit, quast, metaspades, bandage_image, bowtie2, filter_tabular, seqtk_subseq, tp_cat, coverm_contig, collection_column_join, cutadapt, ngsutils_bam_filter, bandage_info</td>
      <td></td>
      <td>UseGalaxy.eu</td>
      <td>0</td>
      <td></td>
      <td>4</td>
      <td>5</td>
      <td>0.016667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Making sense of a newly assembled genome</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/ecoli_comparison/tutorial.html</td>
      <td>Sequence assembly, Genomics, Microbiology</td>
      <td>Sequence visualisation, Genome visualisation, Read mapping, Sequence alignment, Mapping</td>
      <td>2018-06-14</td>
      <td>2024-06-24</td>
      <td>29</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>collapse_dataset, join1, random_lines1, tp_grep_tool, datamash_ops, lastz_wrapper_2, Cut1, Grep1, addValue, tp_replace_in_line, fasta_filter_by_length, jbrowse, mergeCols1, upload1, Filter1, tp_replace_in_column, tp_sed_tool, bedtools_complementbed, tp_sort_header_tool, cat1, circos, bedtools_sortbed, tp_cat, bedtools_intersectbed, circos_interval_to_tiles, circos_aln_to_links, fasta_compute_length</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>6511</td>
      <td>12741</td>
      <td>2.966667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Genome Assembly of MRSA from Oxford Nanopore MinION data (and optionally Illumina data)</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/mrsa-nanopore/tutorial.html</td>
      <td>Sequence assembly, Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology, Antimicrobial resistance</td>
      <td>Visualisation, Sequencing error detection, De-novo assembly, Sequence composition calculation, Sequence assembly visualisation, Filtering, Sequencing quality control, Mapping assembly, Sequence assembly validation, Read mapping, Cross-assembly, Sequence alignment, Scatter plot plotting, Genome assembly, Statistical calculation, Sequence contamination filtering, Box-Whisker plot plotting</td>
      <td>2021-03-24</td>
      <td>2024-11-29</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>CONVERTER_bz2_to_uncompressed, filtlong, fastqc, porechop, quast, fastp, bandage_image, flye, polypolish, upload1, bwa_mem2, nanoplot</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>9207</td>
      <td>13838</td>
      <td>3.416667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Checking expected species and contamination in bacterial isolate</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/bacterial-isolate-species-contamination-checking/tutorial.html</td>
      <td>Whole genome sequencing, Genomics, Microbiology, Microbial ecology</td>
      <td>Taxonomic classification, Expression analysis, Statistical calculation, Cross-assembly</td>
      <td>2024-03-04</td>
      <td>2024-06-27</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>est_abundance, upload1, kraken2, recentrifuge</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td>Galaxy@AuBi, GalaxyTrakr, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>685</td>
      <td>936</td>
      <td>3.483333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Identifying tuberculosis transmission links: from SNPs to transmission clusters</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/mtb_transmission/tutorial.html</td>
      <td>Genomics, Microbiology, Infectious disease, DNA polymorphism</td>
      <td>Antimicrobial resistance prediction, Phylogenetic tree generation, Phylogenetic tree visualisation, Variant calling</td>
      <td>2022-03-16</td>
      <td>2025-05-26</td>
      <td>13</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>snippy, collapse_dataset, bcftools_consensus, addName, tp_grep_tool, tb_variant_filter, Grep1, tp_replace_in_line, clustering_from_distmat, samtools_view, upload1, tp_sed_tool, trimmomatic, snp_dists, snp_sites, __MERGE_COLLECTION__, tp_easyjoin_tool, tb_profiler_profile, tp_cat</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2221</td>
      <td>3263</td>
      <td>2.833333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Tree thinking for tuberculosis evolution and epidemiology</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/mtb_phylogeny/tutorial.html</td>
      <td>Genomics, Microbiology, Phylogeny, Infectious disease</td>
      <td>Phylogenetic tree analysis, Sequence analysis</td>
      <td>2022-03-16</td>
      <td>2024-03-12</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>upload1, raxml, interactive_tool_rstudio</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>1904</td>
      <td>2780</td>
      <td>2.233333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Phylogenetic analysis for bacterial comparative genomics</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/bacterial-comparative-genomics/tutorial.html</td>
      <td>Genomics, Microbiology, Phylogeny</td>
      <td>Phylogenetic tree generation (maximum likelihood and Bayesian methods), Phylogenetic tree generation (from molecular sequences)</td>
      <td>2025-03-14</td>
      <td>2025-03-14</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>upload1, fasttree</td>
      <td>Galaxy@AuBi, GalaxyTrakr, MISSISSIPPI, UseGalaxy.be, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>37</td>
      <td>41</td>
      <td>4.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Genome annotation with Prokka</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/annotation-with-prokka/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis</td>
      <td>Coding region prediction, Genome visualisation, Genome annotation, Gene prediction</td>
      <td>2018-03-06</td>
      <td>2025-03-19</td>
      <td>26</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>prokka, jbrowse</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>Galaxy@AuBi, GalaxyTrakr, MISSISSIPPI, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>2552938</td>
      <td>6945825</td>
      <td>4.483333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Bacterial Genome Annotation</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/bacterial-genome-annotation/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis, Whole genome sequencing, Functional genomics, Mobile genetic elements</td>
      <td>Multilocus sequence typing, Genome visualisation, Protein feature detection, Nucleic acid feature detection, Genome assembly, Sequence motif recognition, Structural variation detection, Genome annotation, Scaffolding</td>
      <td>2024-02-01</td>
      <td>2024-10-15</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>plasmidfinder, tp_tail_tool, isescan, jbrowse, Grouping1, bakta, tp_replace_in_column, tbl2gff3, upload1, integron_finder</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td>Galaxy@AuBi, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>4872</td>
      <td>6595</td>
      <td>2.700000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Refining Genome Annotations with Apollo (prokaryotes)</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/apollo/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis</td>
      <td>Genome visualisation</td>
      <td>2021-06-04</td>
      <td>2025-05-15</td>
      <td>22</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>create_or_update, jbrowse, list_organism, iframe, create_account</td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2309956</td>
      <td>6138571</td>
      <td>5.300000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Identification of AMR genes in an assembled bacterial genome</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/amr-gene-detection/tutorial.html</td>
      <td>Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology, Sequence analysis, Infectious disease, Antimicrobial resistance</td>
      <td>Antimicrobial resistance prediction, Read mapping, Genome visualisation, Genome annotation</td>
      <td>2024-01-23</td>
      <td>2025-05-15</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>Grep1, jbrowse, bowtie2, bakta, staramr_search, tbl2gff3, upload1</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td>Galaxy@AuBi, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>2638</td>
      <td>3599</td>
      <td>2.883333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Dataset construction for bacterial comparative genomics</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/bacterial-comparative-genomics-dataset-construction/tutorial.html</td>
      <td>Genomics, Microbiology</td>
      <td>Data retrieval, Sequence database search, Data handling</td>
      <td>2025-03-14</td>
      <td>2025-03-14</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>upload1, Remove beginning1, sort1, bg_uniq, cat1, datasets_download_genome, Cut1</td>
      <td>UseGalaxy.fr</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>36</td>
      <td>47</td>
      <td>5.616667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Bacterial genome quality control</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/bacterial-genome-quality-control/tutorial.html</td>
      <td>Genomics, Microbiology</td>
      <td>Validation, Genome comparison, Sequence composition calculation, Statistical calculation, Sequencing quality control, Sequence assembly validation</td>
      <td>2025-03-14</td>
      <td>2025-03-14</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>drep_dereplicate, upload1, checkm2</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>33</td>
      <td>36</td>
      <td>2.300000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Comparative gene analysis in unannotated genomes</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/gene-centric/tutorial.html</td>
      <td>Genomics, Gene and protein families, Sequence analysis, Phylogeny</td>
      <td>Sequence alignment analysis, De-novo assembly, Database search, Phylogenetic tree generation, Multiple sequence alignment, Transcriptome assembly, Coding region prediction</td>
      <td>2022-09-08</td>
      <td>2025-05-15</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>bg_diamond_makedb, collapse_dataset, bg_diamond_view, Add_a_column1, gops_intersect_1, join1, cat1, regexColumn1, rbc_mafft, orfipy, bg_diamond, rapidnj, tp_split_on_column, tab2fasta, Cut1, Filter1</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1626</td>
      <td>2343</td>
      <td>3.133333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Essential genes detection with Transposon insertion sequencing</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/tnseq/tutorial.html</td>
      <td>Genomics, Microbiology, Sequence analysis, Mobile genetic elements</td>
      <td>Transposon prediction, Sequence trimming, Read pre-processing, Primer removal</td>
      <td>2019-07-02</td>
      <td>2024-03-18</td>
      <td>12</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>transit_gumbel, cutadapt, bowtie_wrapper, Filter1, Add_a_column1, gff_to_prot, tp_sort_header_tool, deeptools_bam_coverage, tp_easyjoin_tool, __EXTRACT_DATASET__, Cut1, bg_find_subsequences</td>
      <td>UseGalaxy.no, UseGalaxy.org.au</td>
      <td>UseGalaxy.eu</td>
      <td>0</td>
      <td></td>
      <td>2542827</td>
      <td>6929554</td>
      <td>4.350000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Microbial Variant Calling</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/microbial-variants/tutorial.html</td>
      <td>Genomics, Sequence assembly, DNA polymorphism, Microbiology, Sequence analysis, Genetic variation</td>
      <td>Phylogenetic tree generation, Phylogenetic tree visualisation, Variant calling, Genome visualisation</td>
      <td>2018-02-26</td>
      <td>2025-03-19</td>
      <td>24</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>snippy, jbrowse</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>GalaxyTrakr, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>14106</td>
      <td>22743</td>
      <td>2.016667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Analyses of metagenomics data - The global picture</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/general-tutorial/tutorial.html</td>
      <td>Metagenomics, Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td>Visualisation, Phylogenetic analysis, Sequencing quality control, Sequence read processing, Sequence clustering, Taxonomic classification, DNA barcoding</td>
      <td>2017-06-22</td>
      <td>2024-08-09</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>mothur_filter_seqs, mothur_make_biom, mothur_summary_seqs, krona-text, mothur_pre_cluster, mothur_screen_seqs, mothur_align_seqs, mothur_count_seqs, metaphlan2krona, humann2, mothur_classify_otu, mothur_merge_files, mothur_classify_seqs, humann2_regroup_table, mothur_unique_seqs, mothur_make_group, metaphlan2, humann2_renorm_table, mothur_make_shared, taxonomy_krona_chart, mothur_cluster_split</td>
      <td></td>
      <td>UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>6376</td>
      <td>8660</td>
      <td>2.883333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Assembly of metagenomic sequencing data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metagenomics-assembly/tutorial.html</td>
      <td>Metagenomics, Sequence assembly</td>
      <td>Visualisation, Data handling, Variant calling, Sequencing quality control, Sequence assembly validation, Primer removal, Local alignment, Genome assembly, Formatting, Sequence file editing, Sequence alignment analysis, Sequence trimming, Sequence composition calculation, Sequence assembly visualisation, Read mapping, Read pre-processing, Statistical calculation, Sequence contamination filtering</td>
      <td>2022-12-05</td>
      <td>2024-11-26</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>fastqc, bg_uniq, bamtools, megahit_contig2fastg, random_lines1, megahit, quast, metaspades, bandage_image, bowtie2, filter_tabular, seqtk_subseq, tp_cat, coverm_contig, collection_column_join, cutadapt, ngsutils_bam_filter, bandage_info</td>
      <td></td>
      <td>UseGalaxy.eu</td>
      <td>0</td>
      <td></td>
      <td>3388</td>
      <td>4741</td>
      <td>2.966667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Building an amplicon sequence variant (ASV) table from 16S data using DADA2</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/dada-16S/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td>Visualisation, Variant calling, DNA barcoding, Analysis, Deposition</td>
      <td>2024-06-05</td>
      <td>2025-04-22</td>
      <td>10</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>Grouping1, tp_replace_in_line, dada2_filterAndTrim, phyloseq_from_dada2, __SORTLIST__, dada2_seqCounts, collection_element_identifiers, dada2_learnErrors, dada2_plotQualityProfile, dada2_makeSequenceTable, tp_replace_in_column, dada2_removeBimeraDenovo, interactive_tool_phyloseq, dada2_mergePairs, Add_a_column1, dada2_dada, cat1, dada2_assignTaxonomyAddspecies, __UNZIP_COLLECTION__, tp_head_tool</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>4182</td>
      <td>5613</td>
      <td>2.133333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Identification of the micro-organisms in a beer using Nanopore sequencing</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/beer-data-analysis/tutorial.html</td>
      <td>Metagenomics, Microbial ecology, Taxonomy, Sequence analysis</td>
      <td>Visualisation, Aggregation, Sequence composition calculation, Taxonomic classification, Sequencing quality control, Statistical calculation, Sequence contamination filtering</td>
      <td>2022-09-29</td>
      <td>2024-11-29</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>fastqc, porechop, krakentools_kreport2krona, fastp, kraken2, taxonomy_krona_chart, Filter1</td>
      <td>GalaxyTrakr, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>2204</td>
      <td>3264</td>
      <td>2.850000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Metatranscriptomics analysis using microbiome RNA-seq data (short)</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metatranscriptomics-short/tutorial.html</td>
      <td>Metatranscriptomics, Microbial ecology, Taxonomy, Function analysis, Sequence analysis</td>
      <td>Visualisation, Phylogenetic analysis, Sequencing quality control, Conversion, Species frequency estimation, Primer removal, Sequence comparison, Taxonomic classification, Validation, Sequence alignment analysis, Sequence trimming, Sequence composition calculation, Read pre-processing, Phylogenetic tree visualisation, Sequence similarity search, Phylogenetic inference, Statistical calculation, Phylogenetic tree editing</td>
      <td>2020-02-13</td>
      <td>2025-04-07</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>humann_unpack_pathways, multiqc, bg_sortmerna, cutadapt, Cut1, Grep1, humann_rename_table, fastq_paired_end_interlacer, humann, export2graphlan, humann_split_stratified_table, humann_regroup_table, graphlan_annotate, humann_renorm_table, tp_find_and_replace, fastqc, metaphlan, graphlan, combine_metaphlan2_humann2, taxonomy_krona_chart</td>
      <td>UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>554</td>
      <td>721</td>
      <td>2.383333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>16S Microbial analysis with Nanopore data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/nanopore-16S-metagenomics/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td>Validation, Visualisation, Sequence composition calculation, Taxonomic classification, Sequencing quality control, Statistical calculation, Sequence contamination filtering</td>
      <td>2020-11-24</td>
      <td>2024-07-31</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Remove beginning1, tp_replace_in_line, fastqc, multiqc, porechop, fastp, kraken2, taxonomy_krona_chart, datamash_reverse</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>GalaxyTrakr, UseGalaxy.be, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>6132</td>
      <td>8507</td>
      <td>2.650000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Antibiotic resistance detection</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/plasmid-metagenomics-nanopore/tutorial.html</td>
      <td>Metagenomic sequencing, Public health and epidemiology, Metagenomics, Microbiology, Infectious disease, Sequence analysis, Antimicrobial resistance</td>
      <td>Aggregation, De-novo assembly, Sequence assembly visualisation, Antimicrobial resistance prediction, Mapping assembly, Scatter plot plotting, Sequence analysis, Genome assembly, Pairwise sequence alignment, Box-Whisker plot plotting</td>
      <td>2019-06-25</td>
      <td>2024-06-14</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>PlasFlow, miniasm, minimap2, bandage_image, unicycler, staramr_search, racon, nanoplot, gfa_to_fa</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.be, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>2668</td>
      <td>3785</td>
      <td>2.933333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Metatranscriptomics analysis using microbiome RNA-seq data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metatranscriptomics/tutorial.html</td>
      <td>Metatranscriptomics, Microbial ecology, Taxonomy, Function analysis, Sequence analysis</td>
      <td>Visualisation, Phylogenetic analysis, Sequencing quality control, Conversion, Species frequency estimation, Primer removal, Sequence comparison, Taxonomic classification, Validation, Sequence alignment analysis, Sequence trimming, Sequence composition calculation, Read pre-processing, Phylogenetic tree visualisation, Sequence similarity search, Phylogenetic inference, Statistical calculation, Phylogenetic tree editing</td>
      <td>2019-11-21</td>
      <td>2025-04-07</td>
      <td>5</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>humann_unpack_pathways, multiqc, bg_sortmerna, cutadapt, Cut1, Grep1, humann_rename_table, fastq_paired_end_interlacer, humann, export2graphlan, humann_split_stratified_table, humann_regroup_table, graphlan_annotate, humann_renorm_table, tp_find_and_replace, fastqc, metaphlan, graphlan, combine_metaphlan2_humann2, taxonomy_krona_chart</td>
      <td>UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2540561</td>
      <td>6922306</td>
      <td>4.516667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Taxonomic Profiling and Visualization of Metagenomic Data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/taxonomic-profiling/tutorial.html</td>
      <td>Metagenomics, Microbial ecology, Taxonomy, Sequence analysis</td>
      <td>Visualisation, Aggregation, Taxonomic classification, Statistical calculation, Genome annotation</td>
      <td>2023-05-03</td>
      <td>2025-04-24</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>kraken2, krakentools_kreport2krona, metaphlan, kraken_biom, __UNZIP_COLLECTION__, est_abundance, interactive_tool_pavian, interactive_tool_phinch, taxonomy_krona_chart</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>9082</td>
      <td>12266</td>
      <td>2.166667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Pathogen detection from (direct Nanopore) sequencing data using Galaxy - Foodborne Edition</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.html</td>
      <td>Metagenomics, Public health and epidemiology, Taxonomy, Sequence assembly, Pathology, Sequence analysis</td>
      <td>Visualisation, Aggregation, Data handling, Variant calling, Phylogenetic tree analysis, Sequencing quality control, Mapping, Scatter plot plotting, Antimicrobial resistance prediction, Cross-assembly, Base-calling, Genome assembly, Taxonomic classification, Multiple sequence alignment, Phylogenetic tree generation (maximum likelihood and Bayesian methods), Box-Whisker plot plotting, Validation, Sequence alignment analysis, Multilocus sequence typing, De-novo assembly, Sequence composition calculation, Sequence assembly visualisation, Phylogenetic tree reconstruction, Mapping assembly, Phylogenetic tree generation (from molecular sequences), Sequence assembly, Pairwise sequence alignment, Phylogenetic tree generation, Statistical calculation, Sequence contamination filtering</td>
      <td>2023-01-26</td>
      <td>2024-09-27</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>clustalw, Remove beginning1, collapse_dataset, multiqc, Count1, bcftools_consensus, CONVERTER_gz_to_uncompressed, bamtools_split_mapped, minimap2, snpSift_filter, Grouping1, flye, fasttree, bcftools_norm, Cut1, fasta_merge_files_and_filter_unique_sequences, snpSift_extractFields, Grep1, __BUILD_LIST__, table_compute, samtools_depth, fastp, Paste1, collection_element_identifiers, mlst, tp_sorted_uniq, newick_display, ggplot2_heatmap, tp_cut_tool, add_line_to_file, nanoplot, regex1, clair3, porechop, tp_multijoin_tool, bandage_image, samtools_fastx, tp_replace_in_column, tp_split_on_column, fasta2tab, tab2fasta, collection_column_join, kraken2, split_file_to_collection, tp_find_and_replace, __FILTER_FAILED_DATASETS__, krakentools_extract_kraken_reads, Add_a_column1, fastqc, krakentools_kreport2krona, samtools_coverage, regexColumn1, bedtools_getfastabed, compose_text_param, tp_head_tool, abricate, barchart_gnuplot, __FILTER_EMPTY_DATASETS__, taxonomy_krona_chart, param_value_from_file, medaka_consensus_pipeline</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>3799</td>
      <td>5764</td>
      <td>3.466667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>QIIME 2 Cancer Microbiome Intervention</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/qiime2-cancer-microbiome-intervention/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td></td>
      <td>2024-02-12</td>
      <td>2024-03-14</td>
      <td>3</td>
      <td>external</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>301</td>
      <td>508</td>
      <td>3.283333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>QIIME 2 Moving Pictures</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/qiime2-moving-pictures/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td></td>
      <td>2024-03-14</td>
      <td>2024-03-14</td>
      <td>1</td>
      <td>external</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>353</td>
      <td>556</td>
      <td>3.483333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Binning of metagenomic sequencing data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metagenomics-binning/tutorial.html</td>
      <td>Metagenomics, Sequence assembly</td>
      <td>Validation, Sequence composition calculation, Genome assembly, Statistical calculation, Sequencing quality control, Sequence assembly validation</td>
      <td>2023-12-05</td>
      <td>2025-05-15</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>checkm_lineage_wf, megahit</td>
      <td>GalaxyTrakr, UseGalaxy.eu, UseGalaxy.fr</td>
      <td>UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>4226</td>
      <td>5169</td>
      <td>1.466667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>16S Microbial Analysis with mothur (extended)</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/mothur-miseq-sop/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td>Visualisation, Phylogenetic analysis, Sequencing quality control, Phylogenetic tree analysis, Sequence read processing, Phylogenetic tree reconstruction, Sequence clustering, Taxonomic classification, DNA barcoding, Phylogenetic tree generation</td>
      <td>2017-02-12</td>
      <td>2024-03-14</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>mothur_filter_seqs, mothur_dist_seqs, mothur_seq_error, mothur_make_biom, mothur_summary_seqs, mothur_venn, mothur_pre_cluster, mothur_screen_seqs, mothur_align_seqs, mothur_count_seqs, mothur_summary_single, mothur_chimera_vsearch, mothur_tree_shared, mothur_get_groups, mothur_classify_otu, mothur_remove_seqs, newick_display, mothur_remove_groups, mothur_taxonomy_to_krona, mothur_make_contigs, mothur_cluster, mothur_count_groups, mothur_dist_shared, mothur_classify_seqs, mothur_remove_lineage, mothur_sub_sample, XY_Plot_1, mothur_unique_seqs, mothur_heatmap_sim, mothur_make_shared, mothur_rarefaction_single, taxonomy_krona_chart, mothur_cluster_split</td>
      <td>UseGalaxy.be, UseGalaxy.eu</td>
      <td>UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>9504</td>
      <td>14059</td>
      <td>2.816667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>16S Microbial Analysis with mothur (short)</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/mothur-miseq-sop-short/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td>Visualisation, Phylogenetic analysis, Sequencing quality control, Phylogenetic tree analysis, Sequence read processing, Phylogenetic tree reconstruction, Sequence clustering, Taxonomic classification, DNA barcoding, Phylogenetic tree generation</td>
      <td>2019-05-13</td>
      <td>2024-06-14</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>mothur_filter_seqs, collapse_dataset, mothur_dist_seqs, mothur_summary_seqs, mothur_venn, mothur_pre_cluster, mothur_screen_seqs, mothur_count_seqs, mothur_summary_single, mothur_chimera_vsearch, mothur_tree_shared, mothur_classify_otu, mothur_remove_seqs, newick_display, mothur_remove_groups, mothur_cluster, mothur_count_groups, mothur_dist_shared, mothur_classify_seqs, mothur_remove_lineage, mothur_sub_sample, XY_Plot_1, mothur_unique_seqs, mothur_heatmap_sim, mothur_make_shared, mothur_rarefaction_single, mothur_cluster_split</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>4332</td>
      <td>7318</td>
      <td>4.700000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>M. tuberculosis Variant Analysis</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/tb-variant-analysis/tutorial.html</td>
      <td>Genomics, Microbiology, Sequence assembly, Genetic variation, Public health and epidemiology, Infectious disease</td>
      <td>Validation, Sequence alignment analysis, Variant calling, Genome visualisation, Sequence composition calculation, Antimicrobial resistance prediction, Sequencing quality control, Global alignment, Phylogenetic tree visualisation, Sequence alignment, Sequence analysis, Local alignment, Taxonomic classification, Phylogenetic tree generation, Statistical calculation, Sequence contamination filtering</td>
      <td>2020-07-25</td>
      <td>2024-07-26</td>
      <td>30</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>snippy, multiqc, bcftools_consensus, tp_awk_tool, tb_variant_filter, tbvcfreport, __FLATTEN__, fastp, jbrowse, upload1, mosdepth, tp_sed_tool, qualimap_bamqc, kraken2, EMBOSS: seqret84, EMBOSS:%20seqret84, fastqc, jvarkit_wgscoverageplotter, samtools_stats, tb_profiler_profile</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>6702</td>
      <td>10388</td>
      <td>2.900000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>metaQuantome 3: Taxonomy</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/metaquantome-taxonomy/tutorial.html</td>
      <td>Proteomics, Proteogenomics, Metatranscriptomics, Microbial ecology, Metagenomics, Taxonomy</td>
      <td>Visualisation, Differential protein expression analysis, Heat map generation, Query and retrieval, Functional clustering, Filtering, Quantification, Principal component visualisation, Statistical inference, Indexing</td>
      <td>2020-10-29</td>
      <td>2024-03-15</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>metaquantome_sample, metaquantome_filter, metaquantome_stat, metaquantome_viz, metaquantome_expand, metaquantome_db</td>
      <td>UseGalaxy.be, UseGalaxy.eu</td>
      <td>UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>420</td>
      <td>558</td>
      <td>2.500000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Metaproteomics tutorial</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/metaproteomics/tutorial.html</td>
      <td>Proteomics, Proteogenomics, Biodiversity, Taxonomy</td>
      <td>Visualisation, Prediction and recognition</td>
      <td>2017-06-28</td>
      <td>2024-06-14</td>
      <td>41</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>sqlite_to_tabular, query_tabular, peptide_shaker, unipept, search_gui</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.be</td>
      <td>0</td>
      <td></td>
      <td>2688</td>
      <td>4338</td>
      <td>2.716667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>metaQuantome 1: Data creation</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/metaquantome-data-creation/tutorial.html</td>
      <td>Proteomics, Proteogenomics, Biodiversity, Taxonomy</td>
      <td>Visualisation, Label-free quantification, Formatting, Filtering, Prediction and recognition</td>
      <td>2020-10-16</td>
      <td>2024-03-14</td>
      <td>20</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>Grep1, Remove beginning1, tp_replace_in_line, regex1, query_tabular, flashlfq, msconvert, peptide_shaker, tp_replace_in_column, unipept, Cut1, Filter1, search_gui</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>848</td>
      <td>1457</td>
      <td>3.800000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>metaQuantome 2: Function</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/metaquantome-function/tutorial.html</td>
      <td>Proteomics, Proteogenomics, Metatranscriptomics, Microbial ecology, Metagenomics</td>
      <td>Visualisation, Differential protein expression analysis, Heat map generation, Query and retrieval, Functional clustering, Filtering, Quantification, Principal component visualisation, Statistical inference, Indexing</td>
      <td>2020-10-29</td>
      <td>2024-03-15</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>metaquantome_sample, metaquantome_filter, metaquantome_stat, metaquantome_viz, metaquantome_expand, metaquantome_db</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.be, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>394</td>
      <td>562</td>
      <td>4.333333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Sequence analysis</td>
      <td>Quality and contamination control in bacterial isolate using Illumina MiSeq Data</td>
      <td>https://training.galaxyproject.org//topics/sequence-analysis/tutorials/quality-contamination-control/tutorial.html</td>
      <td>Whole genome sequencing, Genomics, Microbiology, Microbial ecology</td>
      <td>Visualisation, Read mapping, Sequencing quality control, Cross-assembly, Taxonomic classification, Expression analysis, Statistical calculation, Sequence contamination filtering</td>
      <td>2024-07-15</td>
      <td>2025-03-20</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>upload1, fastp, est_abundance, falco, kraken2, recentrifuge</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td>UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>1166</td>
      <td>1511</td>
      <td>3.233333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Calling variants in non-diploid systems</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/non-dip/tutorial.html</td>
      <td>Genomics, Sequence assembly, DNA polymorphism, Microbiology, Sequence analysis, Genetic variation</td>
      <td>Sequence alignment analysis, Generation, Variant calling, Sequence composition calculation, Data handling, Sequencing quality control, Read mapping, Sequence alignment, Genome indexing, Formatting, Statistical calculation</td>
      <td>2017-02-16</td>
      <td>2024-03-15</td>
      <td>23</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>vcf2tsv, bamleftalign, fastqc, picard_MergeSamFiles, freebayes, bamFilter, picard_MarkDuplicates, vcffilter2, Cut1, bwa_mem</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.be, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>8023</td>
      <td>14767</td>
      <td>2.166667</td>
      <td>0</td>
    </tr>
  </tbody>
</table>

</div>