<script src="https://code.jquery.com/jquery-3.7.0.js"></script>
<script src="https://cdn.datatables.net/1.13.6/js/jquery.dataTables.min.js"></script>
<script src="https://cdn.datatables.net/searchbuilder/1.6.0/js/dataTables.searchBuilder.min.js"></script>
<script src="https://cdn.datatables.net/datetime/1.5.1/js/dataTables.dateTime.min.js"></script>

<script src="https://cdn.datatables.net/responsive/2.5.0/js/dataTables.responsive.min.js"></script>

<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/responsive/2.2.3/css/responsive.dataTables.min.css">

<link rel="stylesheet" href="https://cdn.datatables.net/1.13.6/css/jquery.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/searchbuilder/1.6.0/css/searchBuilder.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/datetime/1.5.1/css/dataTables.dateTime.min.css">

<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" />
<script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>

<style type="text/css">
  .tooltip {
    font-family: Georgia;
    font-size: 18px;
  }
  .tooltip .tooltip-inner {
    background-color: #00FFFF;
    color: #000000;
    min-width: 400px;
  }
  </style>

<script>
    $(document).ready(function() {
    $('#dataframe').DataTable( {
        "sScrollX": "100%",
        "dom": "Qlfrtip",

        responsive: {
            details: {
                type: 'column'
            }
        },
        "bAutoWidth": false,
        columnDefs: [ {
            className: 'control',
            orderable: false,
            targets:   [ 0 ],
            width: 100,
        },
        { type: 'natural', targets: 1 } // allow normal sorting
        ],

    });
});
</script>

<div class="display nowrap" style="width:80%" width="100%">

<table class="dataframe display nowrap" id="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th>Expand</th>
      <th>Topic</th>
      <th>Title</th>
      <th>Link</th>
      <th>EDAM topic</th>
      <th>EDAM operation</th>
      <th>Creation</th>
      <th>Last modification</th>
      <th>Version</th>
      <th>Tutorial</th>
      <th>Slides</th>
      <th>Video</th>
      <th>Workflows</th>
      <th>Tools</th>
      <th>Servers with precise tool versions</th>
      <th>Servers with tool but different versions</th>
      <th>Feedback number</th>
      <th>Feedback mean note</th>
      <th>Visitors</th>
      <th>Page views</th>
      <th>Visit duration</th>
      <th>Video views</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Making sense of a newly assembled genome</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/ecoli_comparison/tutorial.html</td>
      <td>Sequence assembly, Genomics, Microbiology</td>
      <td>Genome visualisation, Sequence alignment, Mapping, Read mapping, Sequence visualisation</td>
      <td>2018-06-14</td>
      <td>2024-06-24</td>
      <td>29</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>bedtools_sortbed, circos, random_lines1, Cut1, Filter1, collapse_dataset, tp_sed_tool, tp_sort_header_tool, tp_replace_in_line, datamash_ops, addValue, bedtools_intersectbed, tp_grep_tool, upload1, circos_aln_to_links, Grep1, fasta_compute_length, tp_replace_in_column, cat1, bedtools_complementbed, circos_interval_to_tiles, join1, mergeCols1, fasta_filter_by_length, tp_cat, jbrowse, lastz_wrapper_2</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>6301</td>
      <td>12487</td>
      <td>3.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Assembly of metagenomic sequencing data</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/metagenomics-assembly/tutorial.html</td>
      <td>Metagenomics, Sequence assembly</td>
      <td>Sequence alignment analysis, Primer removal, Sequence assembly visualisation, Sequence contamination filtering, Read pre-processing, Statistical calculation, Sequence trimming, Local alignment, Sequence file editing, Formatting, Genome assembly, Sequencing quality control, Data handling, Visualisation, Variant calling, Sequence assembly validation, Read mapping, Sequence composition calculation</td>
      <td>2025-02-16</td>
      <td>2025-02-16</td>
      <td>0</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>megahit_contig2fastg, bowtie2, metaspades, random_lines1, ngsutils_bam_filter, fastqc, megahit, bg_uniq, collection_column_join, bamtools, filter_tabular, seqtk_subseq, cutadapt, bandage_image, coverm_contig, bandage_info, tp_cat, quast</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td>0</td>
      <td></td>
      <td>0</td>
      <td>0</td>
      <td>0.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Genome Assembly of MRSA from Oxford Nanopore MinION data (and optionally Illumina data)</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/mrsa-nanopore/tutorial.html</td>
      <td>Sequence assembly, Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology, Antimicrobial resistance</td>
      <td>Scatter plot plotting, Sequence assembly validation, Read mapping, Sequence contamination filtering, Sequence assembly visualisation, Cross-assembly, Sequence alignment, Genome assembly, Sequencing quality control, Box-Whisker plot plotting, Visualisation, Sequencing error detection, De-novo assembly, Statistical calculation, Mapping assembly, Filtering, Sequence composition calculation</td>
      <td>2021-03-24</td>
      <td>2024-11-29</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>upload1, fastqc, bwa_mem2, polypolish, filtlong, porechop, nanoplot, flye, fastp, bandage_image, quast, CONVERTER_bz2_to_uncompressed</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>8247</td>
      <td>12468</td>
      <td>3.333333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Unicycler Assembly</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/unicycler-assembly/tutorial.html</td>
      <td>Sequence assembly, Genomics, Microbiology</td>
      <td>Validation, Coding region prediction, Statistical calculation, Genome annotation, Aggregation, Gene prediction, Sequencing quality control, Visualisation, Sequence assembly validation, Genome assembly, Sequence composition calculation</td>
      <td>2017-10-11</td>
      <td>2024-03-13</td>
      <td>23</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>fastqc, unicycler, multiqc, prokka, quast</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>GalaxyTrakr, MISSISSIPPI, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2353800</td>
      <td>6442860</td>
      <td>4.600000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Genome Assembly of a bacterial genome (MRSA) sequenced using Illumina MiSeq Data</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/mrsa-illumina/tutorial.html</td>
      <td>Sequence assembly, Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology</td>
      <td>Visualisation, Sequencing quality control, Sequence assembly visualisation, Sequence contamination filtering, Sequence assembly validation, Genome assembly, Statistical calculation, Sequence composition calculation</td>
      <td>2021-03-24</td>
      <td>2024-09-18</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>shovill, fastqc, bandage_image, fastp, bandage_info, quast, upload1</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>Galaxy@AuBi, GalaxyTrakr, UseGalaxy.be, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>7850</td>
      <td>12400</td>
      <td>3.666667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Checking expected species and contamination in bacterial isolate</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/bacterial-isolate-species-contamination-checking/tutorial.html</td>
      <td>Whole genome sequencing, Genomics, Microbiology, Microbial ecology</td>
      <td>Expression analysis, Cross-assembly, Statistical calculation, Taxonomic classification</td>
      <td>2024-03-04</td>
      <td>2024-06-27</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>est_abundance, recentrifuge, kraken2, upload1</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td>Galaxy@AuBi, GalaxyTrakr, UseGalaxy.cz, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>605</td>
      <td>842</td>
      <td>3.733333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Tree thinking for tuberculosis evolution and epidemiology</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/mtb_phylogeny/tutorial.html</td>
      <td>Genomics, Microbiology, Phylogeny, Infectious disease</td>
      <td>Phylogenetic tree analysis, Sequence analysis</td>
      <td>2022-03-16</td>
      <td>2024-03-12</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>raxml, interactive_tool_rstudio, upload1</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>1592</td>
      <td>2333</td>
      <td>2.166667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Evolution</td>
      <td>Identifying tuberculosis transmission links: from SNPs to transmission clusters</td>
      <td>https://training.galaxyproject.org//topics/evolution/tutorials/mtb_transmission/tutorial.html</td>
      <td>Genomics, Microbiology, Infectious disease, DNA polymorphism</td>
      <td>Variant calling, Phylogenetic tree generation, Antimicrobial resistance prediction, Phylogenetic tree visualisation</td>
      <td>2022-03-16</td>
      <td>2024-07-12</td>
      <td>12</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>tp_easyjoin_tool, bcftools_consensus, snippy, __MERGE_COLLECTION__, samtools_view, Grep1, snp_dists, tp_sed_tool, tb_variant_filter, tb_profiler_profile, tp_replace_in_line, addName, collapse_dataset, snp_sites, trimmomatic, tp_cat, tp_grep_tool, upload1</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>1922</td>
      <td>2886</td>
      <td>2.833333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Genome annotation with Prokka</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/annotation-with-prokka/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis</td>
      <td>Genome visualisation, Coding region prediction, Genome annotation, Gene prediction</td>
      <td>2018-03-06</td>
      <td>2024-08-08</td>
      <td>25</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>prokka, jbrowse</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>Galaxy@AuBi, GalaxyTrakr, MISSISSIPPI, UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>2358890</td>
      <td>6451364</td>
      <td>4.383333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Refining Genome Annotations with Apollo (prokaryotes)</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/apollo/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis</td>
      <td>Genome visualisation</td>
      <td>2021-06-04</td>
      <td>2024-10-15</td>
      <td>20</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>iframe, list_organism, create_or_update, jbrowse, create_account</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2116448</td>
      <td>5644931</td>
      <td>5.266667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Bacterial Genome Annotation</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/bacterial-genome-annotation/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis, Whole genome sequencing, Functional genomics, Mobile genetic elements</td>
      <td>Sequence motif recognition, Nucleic acid feature detection, Multilocus sequence typing, Scaffolding, Genome annotation, Genome visualisation, Protein feature detection, Genome assembly, Structural variation detection</td>
      <td>2024-02-01</td>
      <td>2024-10-15</td>
      <td>6</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>integron_finder, tp_replace_in_column, isescan, jbrowse, bakta, tbl2gff3, tp_tail_tool, plasmidfinder, Grouping1, upload1</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td>Galaxy@AuBi, UseGalaxy.cz, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>3765</td>
      <td>5135</td>
      <td>2.800000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Identification of AMR genes in an assembled bacterial genome</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/amr-gene-detection/tutorial.html</td>
      <td>Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology, Sequence analysis, Infectious disease, Antimicrobial resistance</td>
      <td>Genome visualisation, Read mapping, Antimicrobial resistance prediction, Genome annotation</td>
      <td>2024-01-23</td>
      <td>2024-10-15</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>bowtie2, Grep1, bakta, tbl2gff3, staramr_search, jbrowse, upload1</td>
      <td>UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr</td>
      <td>Galaxy@AuBi, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>2026</td>
      <td>2828</td>
      <td>3.066667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Comparative gene analysis in unannotated genomes</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/gene-centric/tutorial.html</td>
      <td>Genomics, Gene and protein families, Sequence analysis, Phylogeny</td>
      <td>Sequence alignment analysis, Coding region prediction, Multiple sequence alignment, Transcriptome assembly, Database search, De-novo assembly, Phylogenetic tree generation</td>
      <td>2022-09-08</td>
      <td>2024-03-18</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>rapidnj, cat1, Cut1, Filter1, bg_diamond, bg_diamond_makedb, tp_split_on_column, join1, tab2fasta, collapse_dataset, gops_intersect_1, regexColumn1, Add_a_column1, bg_diamond_view, rbc_mafft, orfipy</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>1264</td>
      <td>1862</td>
      <td>3.250000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Essential genes detection with Transposon insertion sequencing</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/tnseq/tutorial.html</td>
      <td>Genomics, Microbiology, Sequence analysis, Mobile genetic elements</td>
      <td>Primer removal, Read pre-processing, Transposon prediction, Sequence trimming</td>
      <td>2019-07-02</td>
      <td>2024-03-18</td>
      <td>12</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>tp_easyjoin_tool, deeptools_bam_coverage, bowtie_wrapper, tp_sort_header_tool, Cut1, Filter1, cutadapt, __EXTRACT_DATASET__, gff_to_prot, Add_a_column1, transit_gumbel, bg_find_subsequences</td>
      <td>UseGalaxy.be, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td>0</td>
      <td></td>
      <td>2349289</td>
      <td>6435853</td>
      <td>4.333333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Microbial Variant Calling</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/microbial-variants/tutorial.html</td>
      <td>Genomics, Sequence assembly, DNA polymorphism, Microbiology, Sequence analysis, Genetic variation</td>
      <td>Variant calling, Genome visualisation, Phylogenetic tree generation, Phylogenetic tree visualisation</td>
      <td>2018-02-26</td>
      <td>2024-03-15</td>
      <td>23</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>snippy, jbrowse</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td>GalaxyTrakr, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>13348</td>
      <td>21751</td>
      <td>1.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Analyses of metagenomics data - The global picture</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/general-tutorial/tutorial.html</td>
      <td>Metagenomics, Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td>Phylogenetic analysis, Taxonomic classification, DNA barcoding, Sequencing quality control, Sequence clustering, Sequence read processing, Visualisation</td>
      <td>2017-06-22</td>
      <td>2024-08-09</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>mothur_cluster_split, mothur_screen_seqs, mothur_make_group, metaphlan2krona, humann2_renorm_table, mothur_align_seqs, humann2, metaphlan2, taxonomy_krona_chart, mothur_classify_seqs, mothur_merge_files, mothur_make_shared, mothur_unique_seqs, mothur_make_biom, mothur_pre_cluster, mothur_count_seqs, humann2_regroup_table, mothur_filter_seqs, mothur_summary_seqs, krona-text, mothur_classify_otu</td>
      <td></td>
      <td>UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>5156</td>
      <td>7015</td>
      <td>2.916667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>16S Microbial analysis with Nanopore data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/nanopore-16S-metagenomics/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td>Validation, Sequencing quality control, Visualisation, Sequence contamination filtering, Statistical calculation, Taxonomic classification, Sequence composition calculation</td>
      <td>2020-11-24</td>
      <td>2024-07-31</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>kraken2, fastqc, taxonomy_krona_chart, tp_replace_in_line, multiqc, porechop, fastp, Remove beginning1, datamash_reverse</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>GalaxyTrakr, UseGalaxy.be, UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>4461</td>
      <td>6186</td>
      <td>2.750000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Metatranscriptomics analysis using microbiome RNA-seq data (short)</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metatranscriptomics-short/tutorial.html</td>
      <td>Metatranscriptomics, Microbial ecology, Taxonomy, Function analysis, Sequence analysis</td>
      <td>Nucleic acid sequence analysis, Sequence comparison, Sequence similarity search, Phylogenetic analysis, Taxonomic classification, Sequence trimming, Species frequency estimation, Sequencing quality control, Primer removal, Statistical calculation, Sequence composition calculation, Sequence alignment analysis, Validation, Phylogenetic tree editing, Read pre-processing, Conversion, Phylogenetic tree visualisation, Phylogenetic tree analysis, Visualisation, Phylogenetic inference</td>
      <td>2020-02-13</td>
      <td>2024-06-14</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>export2graphlan, graphlan, humann, humann_unpack_pathways, multiqc, humann_regroup_table, metaphlan, humann_rename_table, humann_split_stratified_table, taxonomy_krona_chart, Cut1, fastq_paired_end_interlacer, humann_renorm_table, cutadapt, combine_metaphlan2_humann2, tp_find_and_replace, graphlan_annotate, fastqc, Grep1, bg_sortmerna</td>
      <td>UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>411</td>
      <td>524</td>
      <td>1.533333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Assembly of metagenomic sequencing data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metagenomics-assembly/tutorial.html</td>
      <td>Metagenomics, Sequence assembly</td>
      <td>Sequence alignment analysis, Primer removal, Sequence assembly visualisation, Sequence contamination filtering, Read pre-processing, Statistical calculation, Sequence trimming, Local alignment, Sequence file editing, Formatting, Genome assembly, Sequencing quality control, Data handling, Visualisation, Variant calling, Sequence assembly validation, Read mapping, Sequence composition calculation</td>
      <td>2022-12-05</td>
      <td>2024-11-26</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>megahit_contig2fastg, bowtie2, metaspades, random_lines1, ngsutils_bam_filter, fastqc, megahit, bg_uniq, collection_column_join, bamtools, filter_tabular, seqtk_subseq, cutadapt, bandage_image, coverm_contig, bandage_info, tp_cat, quast</td>
      <td></td>
      <td>UseGalaxy.cz, UseGalaxy.eu</td>
      <td>0</td>
      <td></td>
      <td>2684</td>
      <td>3817</td>
      <td>2.950000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Taxonomic Profiling and Visualization of Metagenomic Data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/taxonomic-profiling/tutorial.html</td>
      <td>Metagenomics, Microbial ecology, Taxonomy, Sequence analysis</td>
      <td>Aggregation, Nucleic acid sequence analysis, Phylogenetic tree analysis, Visualisation, Taxonomic classification, Statistical calculation, Genome annotation</td>
      <td>2023-05-03</td>
      <td>2024-06-14</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>interactive_tool_phinch, kraken2, __UNZIP_COLLECTION__, taxonomy_krona_chart, est_abundance, krakentools_kreport2krona, interactive_tool_pavian, kraken_biom, metaphlan</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.fr, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>6739</td>
      <td>9152</td>
      <td>1.950000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Antibiotic resistance detection</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/plasmid-metagenomics-nanopore/tutorial.html</td>
      <td>Metagenomic sequencing, Public health and epidemiology, Metagenomics, Microbiology, Infectious disease, Sequence analysis, Antimicrobial resistance</td>
      <td>Scatter plot plotting, Sequence assembly visualisation, Sequence analysis, Pairwise sequence alignment, Aggregation, Box-Whisker plot plotting, De-novo assembly, Genome assembly, Mapping assembly, Antimicrobial resistance prediction</td>
      <td>2019-06-25</td>
      <td>2024-06-14</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>minimap2, racon, miniasm, gfa_to_fa, unicycler, PlasFlow, staramr_search, nanoplot, bandage_image</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>2225</td>
      <td>3187</td>
      <td>3.016667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Identification of the micro-organisms in a beer using Nanopore sequencing</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/beer-data-analysis/tutorial.html</td>
      <td>Metagenomics, Microbial ecology, Taxonomy, Sequence analysis</td>
      <td>Aggregation, Sequencing quality control, Visualisation, Sequence contamination filtering, Statistical calculation, Taxonomic classification, Sequence composition calculation</td>
      <td>2022-09-29</td>
      <td>2024-11-29</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>kraken2, fastqc, taxonomy_krona_chart, Filter1, krakentools_kreport2krona, porechop, fastp</td>
      <td>GalaxyTrakr, UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>1568</td>
      <td>2380</td>
      <td>3.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Metatranscriptomics analysis using microbiome RNA-seq data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metatranscriptomics/tutorial.html</td>
      <td>Metatranscriptomics, Microbial ecology, Taxonomy, Function analysis, Sequence analysis</td>
      <td>Nucleic acid sequence analysis, Sequence comparison, Sequence similarity search, Phylogenetic analysis, Taxonomic classification, Sequence trimming, Species frequency estimation, Sequencing quality control, Primer removal, Statistical calculation, Sequence composition calculation, Sequence alignment analysis, Validation, Phylogenetic tree editing, Read pre-processing, Conversion, Phylogenetic tree visualisation, Phylogenetic tree analysis, Visualisation, Phylogenetic inference</td>
      <td>2019-11-21</td>
      <td>2024-06-14</td>
      <td>4</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>export2graphlan, graphlan, humann, humann_unpack_pathways, multiqc, humann_regroup_table, metaphlan, humann_rename_table, humann_split_stratified_table, taxonomy_krona_chart, Cut1, fastq_paired_end_interlacer, humann_renorm_table, cutadapt, combine_metaphlan2_humann2, tp_find_and_replace, graphlan_annotate, fastqc, Grep1, bg_sortmerna</td>
      <td>UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.cz, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2345972</td>
      <td>6427371</td>
      <td>4.416667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Pathogen detection from (direct Nanopore) sequencing data using Galaxy - Foodborne Edition</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.html</td>
      <td>Metagenomics, Public health and epidemiology, Taxonomy, Sequence assembly, Pathology, Sequence analysis</td>
      <td>Sequence assembly, Sequence contamination filtering, Taxonomic classification, Pairwise sequence alignment, Phylogenetic tree reconstruction, Box-Whisker plot plotting, Sequencing quality control, Data handling, Phylogenetic tree generation (from molecular sequences), Variant calling, Base-calling, Mapping, Statistical calculation, Phylogenetic tree generation, Genome assembly, Antimicrobial resistance prediction, Sequence composition calculation, Sequence alignment analysis, De-novo assembly, Mapping assembly, Scatter plot plotting, Validation, Multiple sequence alignment, Multilocus sequence typing, Sequence assembly visualisation, Cross-assembly, Aggregation, Phylogenetic tree generation (maximum likelihood and Bayesian methods), Phylogenetic tree analysis, Visualisation</td>
      <td>2023-01-26</td>
      <td>2024-09-27</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>tp_cut_tool, Paste1, __FILTER_FAILED_DATASETS__, multiqc, newick_display, Count1, bamtools_split_mapped, regexColumn1, snpSift_extractFields, split_file_to_collection, bcftools_consensus, __FILTER_EMPTY_DATASETS__, Cut1, taxonomy_krona_chart, tab2fasta, samtools_coverage, collapse_dataset, collection_column_join, samtools_fastx, porechop, nanoplot, Add_a_column1, __BUILD_LIST__, Remove beginning1, abricate, tp_find_and_replace, tp_sorted_uniq, regex1, minimap2, clustalw, compose_text_param, snpSift_filter, fastqc, tp_split_on_column, ggplot2_heatmap, mlst, tp_multijoin_tool, krakentools_extract_kraken_reads, fasttree, flye, fasta_merge_files_and_filter_unique_sequences, CONVERTER_gz_to_uncompressed, bandage_image, fastp, Grep1, kraken2, medaka_consensus_pipeline, tp_replace_in_column, fasta2tab, bedtools_getfastabed, krakentools_kreport2krona, samtools_depth, table_compute, barchart_gnuplot, param_value_from_file, tp_head_tool, add_line_to_file, collection_element_identifiers, clair3, bcftools_norm, Grouping1</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>2979</td>
      <td>4428</td>
      <td>2.950000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Building an amplicon sequence variant (ASV) table from 16S data using DADA2</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/dada-16S/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td>DNA barcoding, Visualisation, Variant calling, Deposition, Analysis</td>
      <td>2024-06-05</td>
      <td>2024-12-05</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>__SORTLIST__, __UNZIP_COLLECTION__, dada2_mergePairs, dada2_seqCounts, dada2_dada, dada2_filterAndTrim, dada2_makeSequenceTable, dada2_removeBimeraDenovo, dada2_assignTaxonomyAddspecies, phyloseq_from_dada2, Add_a_column1, dada2_plotQualityProfile, tp_replace_in_line, dada2_learnErrors, tp_replace_in_column, cat1, tp_head_tool, collection_element_identifiers, interactive_tool_phyloseq, Grouping1</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2737</td>
      <td>3705</td>
      <td>2.050000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>Binning of metagenomic sequencing data</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/metagenomics-binning/tutorial.html</td>
      <td>Metagenomics, Sequence assembly</td>
      <td>Validation, Sequencing quality control, Genome assembly, Sequence assembly validation, Statistical calculation, Sequence composition calculation</td>
      <td>2023-12-05</td>
      <td>2024-09-12</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>checkm_lineage_wf, megahit</td>
      <td>GalaxyTrakr, UseGalaxy.cz, UseGalaxy.eu, UseGalaxy.fr</td>
      <td>UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>3226</td>
      <td>3938</td>
      <td>1.400000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>QIIME 2 Moving Pictures</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/qiime2-moving-pictures/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td></td>
      <td>2024-03-14</td>
      <td>2024-03-14</td>
      <td>1</td>
      <td>external</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>247</td>
      <td>381</td>
      <td>3.433333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>QIIME 2 Cancer Microbiome Intervention</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/qiime2-cancer-microbiome-intervention/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td></td>
      <td>2024-02-12</td>
      <td>2024-03-14</td>
      <td>3</td>
      <td>external</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>228</td>
      <td>402</td>
      <td>3.083333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>16S Microbial Analysis with mothur (short)</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/mothur-miseq-sop-short/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td>Phylogenetic analysis, Taxonomic classification, Phylogenetic tree reconstruction, DNA barcoding, Sequencing quality control, Sequence clustering, Sequence read processing, Phylogenetic tree generation, Phylogenetic tree analysis, Visualisation</td>
      <td>2019-05-13</td>
      <td>2024-06-14</td>
      <td>5</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>mothur_remove_groups, mothur_cluster_split, mothur_screen_seqs, newick_display, mothur_chimera_vsearch, mothur_remove_seqs, mothur_dist_seqs, mothur_cluster, mothur_classify_seqs, mothur_heatmap_sim, collapse_dataset, mothur_make_shared, mothur_rarefaction_single, mothur_unique_seqs, mothur_pre_cluster, mothur_count_seqs, mothur_venn, mothur_filter_seqs, XY_Plot_1, mothur_sub_sample, mothur_summary_seqs, mothur_count_groups, mothur_dist_shared, mothur_classify_otu, mothur_tree_shared, mothur_summary_single, mothur_remove_lineage</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>3111</td>
      <td>5386</td>
      <td>5.250000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Microbiome</td>
      <td>16S Microbial Analysis with mothur (extended)</td>
      <td>https://training.galaxyproject.org//topics/microbiome/tutorials/mothur-miseq-sop/tutorial.html</td>
      <td>Microbial ecology, Taxonomy, Sequence analysis, Metabarcoding</td>
      <td>Phylogenetic analysis, Taxonomic classification, Phylogenetic tree reconstruction, DNA barcoding, Sequencing quality control, Sequence clustering, Sequence read processing, Phylogenetic tree generation, Phylogenetic tree analysis, Visualisation</td>
      <td>2017-02-12</td>
      <td>2024-03-14</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>mothur_remove_groups, mothur_taxonomy_to_krona, mothur_cluster_split, mothur_screen_seqs, mothur_seq_error, newick_display, mothur_get_groups, mothur_chimera_vsearch, mothur_remove_seqs, mothur_dist_seqs, mothur_align_seqs, mothur_cluster, taxonomy_krona_chart, mothur_classify_seqs, mothur_heatmap_sim, mothur_make_shared, mothur_rarefaction_single, mothur_unique_seqs, mothur_make_biom, mothur_pre_cluster, mothur_count_seqs, mothur_venn, mothur_filter_seqs, XY_Plot_1, mothur_sub_sample, mothur_summary_seqs, mothur_count_groups, mothur_dist_shared, mothur_classify_otu, mothur_make_contigs, mothur_tree_shared, mothur_summary_single, mothur_remove_lineage</td>
      <td>UseGalaxy.be, UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>7705</td>
      <td>11544</td>
      <td>2.933333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>M. tuberculosis Variant Analysis</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/tb-variant-analysis/tutorial.html</td>
      <td>Genomics, Microbiology, Sequence assembly, Genetic variation, Public health and epidemiology, Infectious disease</td>
      <td>Sequence alignment analysis, Antimicrobial resistance prediction, Validation, Sequence contamination filtering, Taxonomic classification, Sequence analysis, Local alignment, Genome visualisation, Sequence alignment, Sequencing quality control, Phylogenetic tree generation, Phylogenetic tree visualisation, Variant calling, Statistical calculation, Global alignment, Sequence composition calculation</td>
      <td>2020-07-25</td>
      <td>2024-07-26</td>
      <td>30</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>__FLATTEN__, EMBOSS: seqret84, tb_variant_filter, tb_profiler_profile, multiqc, jvarkit_wgscoverageplotter, qualimap_bamqc, bcftools_consensus, samtools_stats, tp_sed_tool, fastqc, tp_awk_tool, fastp, mosdepth, upload1, snippy, kraken2, EMBOSS:%20seqret84, tbvcfreport, jbrowse</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>6257</td>
      <td>9758</td>
      <td>2.883333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>metaQuantome 2: Function</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/metaquantome-function/tutorial.html</td>
      <td>Proteomics, Proteogenomics, Metatranscriptomics, Microbial ecology, Metagenomics</td>
      <td>Statistical inference, Heat map generation, Query and retrieval, Principal component visualisation, Filtering, Functional clustering, Indexing, Differential protein expression analysis, Quantification, Visualisation</td>
      <td>2020-10-29</td>
      <td>2024-03-15</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>metaquantome_db, metaquantome_stat, metaquantome_sample, metaquantome_filter, metaquantome_expand, metaquantome_viz</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>380</td>
      <td>543</td>
      <td>4.383333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>metaQuantome 1: Data creation</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/metaquantome-data-creation/tutorial.html</td>
      <td>Proteomics, Proteogenomics, Biodiversity, Taxonomy</td>
      <td>Formatting, Label-free quantification, Prediction and recognition, Visualisation, Filtering</td>
      <td>2020-10-16</td>
      <td>2024-03-14</td>
      <td>20</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>query_tabular, regex1, Grep1, msconvert, tp_replace_in_column, Cut1, Filter1, tp_replace_in_line, peptide_shaker, unipept, search_gui, Remove beginning1, flashlfq</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.be, UseGalaxy.cz, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>816</td>
      <td>1416</td>
      <td>3.833333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>Metaproteomics tutorial</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/metaproteomics/tutorial.html</td>
      <td>Proteomics, Proteogenomics, Biodiversity, Taxonomy</td>
      <td>Prediction and recognition, Visualisation</td>
      <td>2017-06-28</td>
      <td>2024-06-14</td>
      <td>41</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>query_tabular, peptide_shaker, unipept, search_gui, sqlite_to_tabular</td>
      <td>UseGalaxy.eu, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.be, UseGalaxy.cz</td>
      <td>0</td>
      <td></td>
      <td>2519</td>
      <td>4115</td>
      <td>2.766667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Proteomics</td>
      <td>metaQuantome 3: Taxonomy</td>
      <td>https://training.galaxyproject.org//topics/proteomics/tutorials/metaquantome-taxonomy/tutorial.html</td>
      <td>Proteomics, Proteogenomics, Metatranscriptomics, Microbial ecology, Metagenomics, Taxonomy</td>
      <td>Statistical inference, Heat map generation, Query and retrieval, Principal component visualisation, Filtering, Functional clustering, Indexing, Differential protein expression analysis, Quantification, Visualisation</td>
      <td>2020-10-29</td>
      <td>2024-03-15</td>
      <td>14</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>metaquantome_db, metaquantome_stat, metaquantome_sample, metaquantome_filter, metaquantome_expand, metaquantome_viz</td>
      <td>UseGalaxy.be, UseGalaxy.eu</td>
      <td>UseGalaxy.cz, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>398</td>
      <td>531</td>
      <td>2.616667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Sequence analysis</td>
      <td>Quality and contamination control in bacterial isolate using Illumina MiSeq Data</td>
      <td>https://training.galaxyproject.org//topics/sequence-analysis/tutorials/quality-contamination-control/tutorial.html</td>
      <td>Whole genome sequencing, Genomics, Microbiology, Microbial ecology</td>
      <td>Expression analysis, Sequencing quality control, Sequence contamination filtering, Cross-assembly, Statistical calculation, Taxonomic classification, Sequence composition calculation</td>
      <td>2024-07-15</td>
      <td>2024-10-09</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>kraken2, fastqc, est_abundance, fastp, recentrifuge, upload1</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td>Galaxy@AuBi, GalaxyTrakr, UseGalaxy.cz, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>676</td>
      <td>888</td>
      <td>4.050000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Variant Analysis</td>
      <td>Calling variants in non-diploid systems</td>
      <td>https://training.galaxyproject.org//topics/variant-analysis/tutorials/non-dip/tutorial.html</td>
      <td>Genomics, Sequence assembly, DNA polymorphism, Microbiology, Sequence analysis, Genetic variation</td>
      <td>Sequence alignment analysis, Generation, Read mapping, Genome indexing, Sequence alignment, Formatting, Sequencing quality control, Data handling, Variant calling, Statistical calculation, Sequence composition calculation</td>
      <td>2017-02-16</td>
      <td>2024-03-15</td>
      <td>23</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>bamFilter, bamleftalign, vcf2tsv, fastqc, Cut1, bwa_mem, vcffilter2, freebayes, picard_MarkDuplicates, picard_MergeSamFiles</td>
      <td>UseGalaxy.be, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.cz, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>7672</td>
      <td>14325</td>
      <td>2.166667</td>
      <td>0</td>
    </tr>
  </tbody>
</table>

</div>