<script src="https://code.jquery.com/jquery-3.7.0.js"></script>
<script src="https://cdn.datatables.net/1.13.6/js/jquery.dataTables.min.js"></script>
<script src="https://cdn.datatables.net/searchbuilder/1.6.0/js/dataTables.searchBuilder.min.js"></script>
<script src="https://cdn.datatables.net/datetime/1.5.1/js/dataTables.dateTime.min.js"></script>

<script src="https://cdn.datatables.net/responsive/2.5.0/js/dataTables.responsive.min.js"></script>

<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/responsive/2.2.3/css/responsive.dataTables.min.css">

<link rel="stylesheet" href="https://cdn.datatables.net/1.13.6/css/jquery.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/searchbuilder/1.6.0/css/searchBuilder.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/datetime/1.5.1/css/dataTables.dateTime.min.css">

<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" />
<script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>

<style type="text/css">
  .tooltip {
    font-family: Georgia;
    font-size: 18px;
  }
  .tooltip .tooltip-inner {
    background-color: #00FFFF;
    color: #000000;
    min-width: 400px;
  }
  </style>

<script>
    $(document).ready(function() {
    $('#dataframe').DataTable( {
        "sScrollX": "100%",
        "dom": "Qlfrtip",

        responsive: {
            details: {
                type: 'column'
            }
        },
        "bAutoWidth": false,
        columnDefs: [ {
            className: 'control',
            orderable: false,
            targets:   [ 0 ],
            width: 100,
        },
        { type: 'natural', targets: 1 } // allow normal sorting
        ],

    });
});
</script>

<div class="display nowrap" style="width:80%" width="100%">

<table class="dataframe display nowrap" id="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th>Expand</th>
      <th>Name</th>
      <th>Source</th>
      <th>ID</th>
      <th>Link</th>
      <th>Creators</th>
      <th>Tags</th>
      <th>Creation time</th>
      <th>Update time</th>
      <th>Latest version</th>
      <th>Versions</th>
      <th>Number of steps</th>
      <th>Tools</th>
      <th>EDAM operations</th>
      <th>EDAM topics</th>
      <th>License</th>
      <th>DOI</th>
      <th>Projects</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td></td>
      <td>Taxonomy classification using Kraken2 and Bracken</td>
      <td>WorkflowHub</td>
      <td>1199</td>
      <td>https://workflowhub.eu/workflows/1199?version=1</td>
      <td>Valentine Murigneux, Mike Thang</td>
      <td>gucfg2galaxy, metagenomics, name:collection, shotgun</td>
      <td>2024-11-12</td>
      <td>2024-12-08</td>
      <td>1</td>
      <td>1</td>
      <td>29.0</td>
      <td>cat_multiple, regex1, Prepare alpha diversity summary file (paste Simpson and Fisher results)\nPaste1, Prepare alpha diversity summary file (paste sample identifiers and Simpson results)\nPaste1, Fisher results contains a header line we want to exclude "Fisher's alpha...loading"\nShow tail1, taxonomy_krona_chart, krakentools_alpha_diversity, collection_column_join, Prepare alpha diversity summary file\nPaste1, est_abundance, collection_element_identifiers, Extract column name and fraction_total_reads from Bracken report\nCut1, krakentools_combine_kreports, krakentools_kreport2krona, krakentools_beta_diversity, kraken2, __RELABEL_FROM_FILE__, add_line_to_file</td>
      <td>Statistical calculation, Aggregation, Taxonomic classification, Visualisation</td>
      <td>Metagenomics, Taxonomy</td>
      <td>CC-BY-4.0</td>
      <td>10.48546/workflowhub.workflow.1199.1</td>
      <td>QCIF Bioinformatics</td>
    </tr>
    <tr>
      <td></td>
      <td>AMR-Pathfinder</td>
      <td>WorkflowHub</td>
      <td>1189</td>
      <td>https://workflowhub.eu/workflows/1189?version=2</td>
      <td>Helena Rasche, Dennis Dollée, Birgit Rijvers</td>
      <td>amr, amr-detection, benchamrking</td>
      <td>2024-11-14</td>
      <td>2024-11-14</td>
      <td>2</td>
      <td>2</td>
      <td>36.0</td>
      <td>, cast, tp_text_file_with_recurring_lines, hamronize_tool, __APPLY_RULES__, tp_split_on_column, hamronize_summarize, tp_find_and_replace, Grep1, __MERGE_COLLECTION__, staramr_search, abricate, collapse_dataset, shovill, datamash_ops, Cut1, cat1, cat_multi_datasets, addValue</td>
      <td>Genome assembly, Parsing, Antimicrobial resistance prediction, Data handling</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td>Seq4AMR, ErasmusMC Clinical Bioinformatics</td>
    </tr>
    <tr>
      <td></td>
      <td>bacterial-genome-assembly/main</td>
      <td>WorkflowHub</td>
      <td>1043</td>
      <td>https://workflowhub.eu/workflows/1043?version=7</td>
      <td>Abromics None, Pierre Marin, Clea Siguret</td>
      <td>abromics, assembly, fastq, genomics, bacterial-genomics, paired-end, quality</td>
      <td>2024-11-26</td>
      <td>2024-11-26</td>
      <td>7</td>
      <td>7</td>
      <td>7.0</td>
      <td>refseq_masher_matches, bandage_info, tooldistillator, quast, bandage_image, tooldistillator_summarize, shovill</td>
      <td>Visualisation, Parsing, Genome assembly, Sequence assembly validation, Data handling, Sequence assembly visualisation</td>
      <td></td>
      <td>GPL-3.0-or-later</td>
      <td></td>
      <td>Intergalactic Workflow Commission (IWC)</td>
    </tr>
    <tr>
      <td></td>
      <td>quality-and-contamination-control/main</td>
      <td>WorkflowHub</td>
      <td>1052</td>
      <td>https://workflowhub.eu/workflows/1052?version=8</td>
      <td>ABRomics None, Pierre Marin, Clea Siguret</td>
      <td>abromics, fastq, genomics, bacterial-genomics, paired-end, quality, taxonomy-assignment, trimming</td>
      <td>2024-11-23</td>
      <td>2024-11-23</td>
      <td>8</td>
      <td>8</td>
      <td>6.0</td>
      <td>tooldistillator, est_abundance, recentrifuge, tooldistillator_summarize, kraken2, fastp</td>
      <td>Statistical calculation, Sequencing quality control, Cross-assembly, Taxonomic classification, Sequence contamination filtering, Parsing, Data handling, Expression analysis</td>
      <td></td>
      <td>GPL-3.0-or-later</td>
      <td></td>
      <td>Intergalactic Workflow Commission (IWC)</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow 4: Staramr</td>
      <td>WorkflowHub</td>
      <td>470</td>
      <td>https://workflowhub.eu/workflows/470?version=1</td>
      <td></td>
      <td>10.3390/microorganisms10020292, amr, amr-detection, bioinformatics, antimicrobial resistance</td>
      <td>2023-05-11</td>
      <td>2024-09-09</td>
      <td>1</td>
      <td>1</td>
      <td>10.0</td>
      <td>shovill, hamronize_tool, hamronize_summarize, tp_find_and_replace, staramr_search, abricate, collapse_dataset</td>
      <td>Parsing, Antimicrobial resistance prediction, Genome assembly, Data handling</td>
      <td></td>
      <td>GPL-3.0</td>
      <td></td>
      <td>Seq4AMR</td>
    </tr>
    <tr>
      <td></td>
      <td>bacterial_genome_annotation/main</td>
      <td>WorkflowHub</td>
      <td>1050</td>
      <td>https://workflowhub.eu/workflows/1050?version=9</td>
      <td>ABRomics None, Pierre Marin, Clea Siguret</td>
      <td>abromics, annotation, genomics, bacterial-genomics, fasta, genome-annotation</td>
      <td>2024-10-26</td>
      <td>2024-10-26</td>
      <td>9</td>
      <td>9</td>
      <td>6.0</td>
      <td>integron_finder, plasmidfinder, tooldistillator, tooldistillator_summarize, bakta, isescan</td>
      <td>Parsing, Scaffolding, Genome assembly, Genome annotation, Multilocus sequence typing, Nucleic acid feature detection, Sequence motif recognition, Structural variation detection, Protein feature detection, Data handling</td>
      <td></td>
      <td>GPL-3.0-or-later</td>
      <td></td>
      <td>Intergalactic Workflow Commission (IWC)</td>
    </tr>
    <tr>
      <td></td>
      <td>amr_gene_detection/main</td>
      <td>WorkflowHub</td>
      <td>1049</td>
      <td>https://workflowhub.eu/workflows/1049?version=7</td>
      <td>ABRomics None, Pierre Marin, Clea Siguret</td>
      <td>abromics, amr, amr-detection, genomics, antibiotic-resistance, antimicrobial resistance, antimicrobial-resistance-genes, bacterial-genomics, fasta</td>
      <td>2024-10-26</td>
      <td>2024-10-27</td>
      <td>7</td>
      <td>7</td>
      <td>5.0</td>
      <td>tooldistillator, tooldistillator_summarize, amrfinderplus, staramr_search, abricate</td>
      <td>Parsing, Antimicrobial resistance prediction, Data handling</td>
      <td></td>
      <td>GPL-3.0-or-later</td>
      <td></td>
      <td>Intergalactic Workflow Commission (IWC)</td>
    </tr>
    <tr>
      <td></td>
      <td>Analyses of shotgun metagenomics data with MetaPhlAn2</td>
      <td>WorkflowHub</td>
      <td>624</td>
      <td>https://workflowhub.eu/workflows/624?version=1</td>
      <td>Valentine Murigneux, Mike Thang, Saskia Hiltemann, Bérénice Batut</td>
      <td>gucfg2galaxy, metagenomics, shotgun</td>
      <td>2023-10-26</td>
      <td>2024-04-05</td>
      <td>1</td>
      <td>1</td>
      <td>17.0</td>
      <td>, humann2_regroup_table, metaphlan2, humann2_renorm_table, taxonomy_krona_chart, metaphlan2krona, merge_metaphlan_tables, humann2, Cut1</td>
      <td>Nucleic acid sequence analysis, Phylogenetic tree analysis, Visualisation</td>
      <td>Metagenomic sequencing, Metagenomics</td>
      <td>CC-BY-4.0</td>
      <td>10.48546/workflowhub.workflow.624.1</td>
      <td>QCIF Bioinformatics</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow 7 : Beta Diversity [16S Microbial Analysis With Mothur]</td>
      <td>WorkflowHub</td>
      <td>653</td>
      <td>https://workflowhub.eu/workflows/653?version=1</td>
      <td>Saskia Hiltemann, Bérénice Batut, Dave Clements, Ahmed Mehdi</td>
      <td>metagenomics</td>
      <td>2023-11-09</td>
      <td>2023-11-09</td>
      <td>1</td>
      <td>1</td>
      <td>4.0</td>
      <td>mothur_tree_shared, mothur_heatmap_sim, mothur_dist_shared, newick_display</td>
      <td>Sequence clustering, Phylogenetic tree generation, Sequencing quality control, Sequence read processing, DNA barcoding, Taxonomic classification, Visualisation, Phylogenetic tree analysis, Phylogenetic analysis, Phylogenetic tree reconstruction</td>
      <td></td>
      <td>CC-BY-4.0</td>
      <td></td>
      <td>QCIF Bioinformatics</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow 6: Alpha Diversity [16S Microbial Analysis With Mothur]</td>
      <td>WorkflowHub</td>
      <td>652</td>
      <td>https://workflowhub.eu/workflows/652?version=1</td>
      <td>Saskia Hiltemann, Bérénice Batut, Dave Clements, Ahmed Mehdi</td>
      <td>metagenomics</td>
      <td>2023-11-09</td>
      <td>2023-11-09</td>
      <td>1</td>
      <td>1</td>
      <td>3.0</td>
      <td>mothur_rarefaction_single, mothur_summary_single, XY_Plot_1</td>
      <td>Sequence clustering, Sequencing quality control, Sequence read processing, DNA barcoding, Taxonomic classification, Visualisation, Phylogenetic analysis</td>
      <td></td>
      <td>CC-BY-4.0</td>
      <td></td>
      <td>QCIF Bioinformatics</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow 5: OTU Clustering [16S Microbial Analysis With Mothur]</td>
      <td>WorkflowHub</td>
      <td>651</td>
      <td>https://workflowhub.eu/workflows/651?version=1</td>
      <td>Saskia Hiltemann, Bérénice Batut, Dave Clements, Ahmed Mehdi</td>
      <td>metagenomics</td>
      <td>2023-11-09</td>
      <td>2023-11-09</td>
      <td>1</td>
      <td>1</td>
      <td>5.0</td>
      <td>mothur_sub_sample, mothur_cluster_split, mothur_make_shared, mothur_count_groups, mothur_classify_otu</td>
      <td>Sequence clustering, Sequencing quality control, Sequence read processing, DNA barcoding, Taxonomic classification, Visualisation, Phylogenetic analysis</td>
      <td></td>
      <td>CC-BY-4.0</td>
      <td></td>
      <td>QCIF Bioinformatics</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow 3: Classification [Galaxy Training: 16S Microbial Analysis With Mothur]</td>
      <td>WorkflowHub</td>
      <td>650</td>
      <td>https://workflowhub.eu/workflows/650?version=1</td>
      <td>Saskia Hiltemann, Bérénice Batut, Dave Clements, Ahmed Mehdi</td>
      <td>metagenomics</td>
      <td>2023-11-09</td>
      <td>2023-11-09</td>
      <td>1</td>
      <td>1</td>
      <td>2.0</td>
      <td>mothur_remove_lineage, mothur_classify_seqs</td>
      <td>Sequence clustering, Sequencing quality control, Sequence read processing, DNA barcoding, Taxonomic classification, Visualisation, Phylogenetic analysis</td>
      <td></td>
      <td>CC-BY-4.0</td>
      <td></td>
      <td>QCIF Bioinformatics</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow 1: Further Quality Control [16S Microbial Analysis With Mothur]</td>
      <td>WorkflowHub</td>
      <td>648</td>
      <td>https://workflowhub.eu/workflows/648?version=1</td>
      <td>Saskia  Hiltemann, Bérénice Batut, Dave Clements, Ahmed Mehdi</td>
      <td>metagenomics</td>
      <td>2023-11-09</td>
      <td>2023-11-09</td>
      <td>1</td>
      <td>1</td>
      <td>5.0</td>
      <td>mothur_unique_seqs, mothur_screen_seqs, mothur_count_seqs, mothur_summary_seqs</td>
      <td>Sequence clustering, Sequencing quality control, Sequence read processing, DNA barcoding, Taxonomic classification, Visualisation, Phylogenetic analysis</td>
      <td></td>
      <td>CC-BY-4.0</td>
      <td></td>
      <td>QCIF Bioinformatics</td>
    </tr>
    <tr>
      <td></td>
      <td>Workflow 2: Data Cleaning And Chimera Removal [16S Microbial Analysis With Mothur]</td>
      <td>WorkflowHub</td>
      <td>649</td>
      <td>https://workflowhub.eu/workflows/649?version=1</td>
      <td>Saskia Hiltemann, Bérénice Batut, Dave Clements, Ahmed Mehdi</td>
      <td>metagenomics</td>
      <td>2023-11-09</td>
      <td>2023-11-09</td>
      <td>1</td>
      <td>1</td>
      <td>9.0</td>
      <td>mothur_chimera_vsearch, mothur_summary_seqs, mothur_filter_seqs, mothur_pre_cluster, mothur_remove_seqs, mothur_unique_seqs, mothur_screen_seqs</td>
      <td>Sequence clustering, Sequencing quality control, Sequence read processing, DNA barcoding, Taxonomic classification, Visualisation, Phylogenetic analysis</td>
      <td></td>
      <td>CC-BY-4.0</td>
      <td></td>
      <td>QCIF Bioinformatics</td>
    </tr>
    <tr>
      <td></td>
      <td>pox-virus-amplicon/main</td>
      <td>WorkflowHub</td>
      <td>439</td>
      <td>https://workflowhub.eu/workflows/439?version=2</td>
      <td>Viktoria Isabel Schwarz, Wolfgang Maier</td>
      <td>virology, pox</td>
      <td>2024-06-22</td>
      <td>2024-06-22</td>
      <td>2</td>
      <td>2</td>
      <td>40.0</td>
      <td>compose_text_param, \n param_value_from_file, samtools_view, ivar_consensus, \n __FILTER_FAILED_DATASETS__, tp_sed_tool, fasta_compute_length, tp_cat, ivar_trim, \n __FLATTEN__, \n __SORTLIST__, datamash_ops, samtools_stats, \n Grep1, split_file_to_collection, samtools_merge, EMBOSS: maskseq51, \n Cut1, collection_element_identifiers, multiqc, bwa_mem, \n __ZIP_COLLECTION__, \n __APPLY_RULES__, fastp, qualimap_bamqc</td>
      <td>Sequence analysis, Sequencing quality control, Global alignment, Genome indexing, Read mapping, Sequence contamination filtering, Sequence alignment analysis, Sequence alignment, Validation, Local alignment, Generation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td>Intergalactic Workflow Commission (IWC)</td>
    </tr>
    <tr>
      <td></td>
      <td>3: Plant virus exploration</td>
      <td>WorkflowHub</td>
      <td>103</td>
      <td>https://workflowhub.eu/workflows/103?version=1</td>
      <td></td>
      <td>de_novo, virology, exploration</td>
      <td>2021-02-04</td>
      <td>2023-02-13</td>
      <td>1</td>
      <td>1</td>
      <td>2.0</td>
      <td>fastp, shovill</td>
      <td>Sequencing quality control, Genome assembly, Sequence contamination filtering</td>
      <td></td>
      <td>Apache-2.0</td>
      <td></td>
      <td>Integrated and Urban Plant Pathology Laboratory</td>
    </tr>
    <tr>
      <td></td>
      <td>2: Plant virus confirmation</td>
      <td>WorkflowHub</td>
      <td>102</td>
      <td>https://workflowhub.eu/workflows/102?version=1</td>
      <td></td>
      <td>assembly, virology, blast, mapping, reads_selection</td>
      <td>2021-02-04</td>
      <td>2023-02-13</td>
      <td>1</td>
      <td>1</td>
      <td>7.0</td>
      <td>ncbi_blastn_wrapper, picard_SamToFastq, fasta_merge_files_and_filter_unique_sequences, samtools_stats, minimap2, ngsutils_bam_filter, shovill</td>
      <td>Sequencing quality control, Formatting, Read pre-processing, Sequence contamination filtering, Variant calling, Genome assembly, Pairwise sequence alignment</td>
      <td></td>
      <td>Apache-2.0</td>
      <td></td>
      <td>Integrated and Urban Plant Pathology Laboratory</td>
    </tr>
    <tr>
      <td></td>
      <td>GTN Training - Antibiotic Resistance Detection</td>
      <td>WorkflowHub</td>
      <td>406</td>
      <td>https://workflowhub.eu/workflows/406?version=1</td>
      <td>Saskia Hiltemann, Willem de Koning</td>
      <td>metagenomics</td>
      <td>2022-11-24</td>
      <td>2023-02-13</td>
      <td>1</td>
      <td>1</td>
      <td>12.0</td>
      <td>unicycler, gfa_to_fa, racon, nanoplot, bandage_image, miniasm, staramr_search, minimap2, PlasFlow</td>
      <td>Scatter plot plotting, Box-Whisker plot plotting, Sequence analysis, Aggregation, De-novo assembly, Genome assembly, Pairwise sequence alignment, Mapping assembly, Sequence assembly visualisation</td>
      <td>Microbiology</td>
      <td>CC-BY-4.0</td>
      <td></td>
      <td>Galaxy Training Network</td>
    </tr>
    <tr>
      <td></td>
      <td>1: Plant virus detection with kraken2 (SE)</td>
      <td>WorkflowHub</td>
      <td>124</td>
      <td>https://workflowhub.eu/workflows/124?version=1</td>
      <td></td>
      <td>virology, kraken</td>
      <td>2021-06-17</td>
      <td>2023-02-13</td>
      <td>1</td>
      <td>1</td>
      <td>3.0</td>
      <td>Kraken2Tax, taxonomy_krona_chart, kraken2</td>
      <td>Taxonomic classification, Visualisation</td>
      <td></td>
      <td>Apache-2.0</td>
      <td></td>
      <td>Integrated and Urban Plant Pathology Laboratory</td>
    </tr>
    <tr>
      <td></td>
      <td>1: Plant virus detection with kraken2 (PE)</td>
      <td>WorkflowHub</td>
      <td>101</td>
      <td>https://workflowhub.eu/workflows/101?version=1</td>
      <td></td>
      <td>virology, kraken</td>
      <td>2021-02-04</td>
      <td>2023-02-13</td>
      <td>1</td>
      <td>1</td>
      <td>3.0</td>
      <td>Kraken2Tax, taxonomy_krona_chart, kraken2</td>
      <td>Taxonomic classification, Visualisation</td>
      <td></td>
      <td>Apache-2.0</td>
      <td></td>
      <td>Integrated and Urban Plant Pathology Laboratory</td>
    </tr>
    <tr>
      <td></td>
      <td>Halophiles workup of Comparative gene analysis</td>
      <td>https://usegalaxy.eu</td>
      <td>a2c46deea34d9d80</td>
      <td>https://usegalaxy.eu/published/workflow?id=a2c46deea34d9d80</td>
      <td>Anton Nekrutenko</td>
      <td>genome-annotation, name:microgalaxy</td>
      <td>2024-12-03</td>
      <td>2024-12-05</td>
      <td>2</td>
      <td>2</td>
      <td></td>
      <td>tab2fasta, rapidnj, regexColumn1, Add_a_column1, cat1, Filter1, collapse_dataset, tp_split_on_column, bg_diamond, bg_diamond_view, gops_intersect_1, Cut1, join1, rbc_mafft</td>
      <td>Sequence alignment analysis, Phylogenetic tree generation, Multiple sequence alignment</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>pAllori Blood RNA</td>
      <td>https://usegalaxy.eu</td>
      <td>ca9d3233b0912765</td>
      <td>https://usegalaxy.eu/published/workflow?id=ca9d3233b0912765</td>
      <td>Engy Nasr, Paul Zierep</td>
      <td>name:microgalaxy, name:pallori, name:bloodrna, name:umi, name:barcode, name:transcriptomics</td>
      <td>2023-07-02</td>
      <td>2024-11-24</td>
      <td>43</td>
      <td>43</td>
      <td></td>
      <td>__FILTER_FAILED_DATASETS__, umi_tools_extract, fastqc, featurecounts, rna_star, collection_column_join, multiqc</td>
      <td>Validation, Statistical calculation, Sequencing quality control, RNA-Seq quantification, Sequence alignment, Read summarisation, Sequence composition calculation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Metaproteomics workflow</td>
      <td>https://usegalaxy.eu</td>
      <td>cefc49c13ff73231</td>
      <td>https://usegalaxy.eu/published/workflow?id=cefc49c13ff73231</td>
      <td>Timothy Griffin, Pratik Jagtap, James Johnson, Clemens Blank, Subina Mehta</td>
      <td>name:metaproteomics, name:galaxyp, name:massspectrometry, name:microgalaxy</td>
      <td>2024-11-21</td>
      <td>2024-11-21</td>
      <td>40</td>
      <td>40</td>
      <td></td>
      <td>sqlite_to_tabular, query_tabular, search_gui, unipept, peptide_shaker</td>
      <td>Visualisation, Prediction and recognition</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Feature-Count Table Normalization</td>
      <td>https://usegalaxy.eu</td>
      <td>6239178d7cc4ac68</td>
      <td>https://usegalaxy.eu/published/workflow?id=6239178d7cc4ac68</td>
      <td>Johannes Effenberger</td>
      <td>name:microgalaxy, normalization, softmax, relative-abundance, sigmoid, css, limma-tmm, limma-rle</td>
      <td>2024-11-21</td>
      <td>2024-11-21</td>
      <td>5</td>
      <td>5</td>
      <td></td>
      <td>__EXTRACT_DATASET__, __BUILD_LIST__, __FILTER_EMPTY_DATASETS__, map_param_value, Grep1</td>
      <td></td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>metaQuantome_datacreation_workflow</td>
      <td>https://usegalaxy.eu</td>
      <td>cd675ab32d3e7833</td>
      <td>https://usegalaxy.eu/published/workflow?id=cd675ab32d3e7833</td>
      <td>Subina Metha, Timothy J. Griffin, Pratik Jagtap, Emma Leith, Marie Crane</td>
      <td>proteomics, name:microgalaxy</td>
      <td>2024-11-21</td>
      <td>2024-11-21</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>tp_replace_in_column, regex1, tp_replace_in_line, Remove beginning1, flashlfq, Filter1, query_tabular, msconvert, search_gui, unipept, Cut1, peptide_shaker</td>
      <td>Label-free quantification, Formatting, Visualisation, Filtering, Prediction and recognition</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>metaquantome-function-worklow</td>
      <td>https://usegalaxy.eu</td>
      <td>e5a89ef7b5f1c1d9</td>
      <td>https://usegalaxy.eu/published/workflow?id=e5a89ef7b5f1c1d9</td>
      <td>Subina Mehta, Timothy J. Griffin, Pratik Jagtap, Emma Leith, Marie Crane, Praveen Kumar</td>
      <td>proteomics, name:microgalaxy</td>
      <td>2024-11-21</td>
      <td>2024-11-21</td>
      <td>3</td>
      <td>3</td>
      <td></td>
      <td>metaquantome_stat, metaquantome_viz, metaquantome_db, metaquantome_sample, metaquantome_filter, metaquantome_expand</td>
      <td>Functional clustering, Principal component visualisation, Statistical inference, Differential protein expression analysis, Query and retrieval, Heat map generation, Quantification, Visualisation, Filtering, Indexing</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>pAllori Amplicon 16S Pre-Processing and Taxonomy Classification</td>
      <td>https://usegalaxy.eu</td>
      <td>466bdd8ba7b67264</td>
      <td>https://usegalaxy.eu/published/workflow?id=466bdd8ba7b67264</td>
      <td>Engy Nasr, Paul Zierep</td>
      <td>name:microgalaxy, name:16s, name:metagenomics, name:pallori</td>
      <td>2023-06-30</td>
      <td>2024-11-21</td>
      <td>70</td>
      <td>70</td>
      <td></td>
      <td>tp_cut_tool, ampvis2_ordinate, fastqc, trimmomatic, ampvis2_load, __FLATTEN__, taxonomy_krona_chart, collection_column_join, kraken2, krakentools_kreport2krona, multiqc, lotus2, ampvis2_heatmap, cutadapt, unzip</td>
      <td>Sequencing quality control, Aggregation, Analysis, Sequence feature detection, Read pre-processing, Sequence trimming, Primer removal, Validation, Statistical calculation, DNA barcoding, Taxonomic classification, Visualisation, Sequence composition calculation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Cloud Aerosol MT-MG Contamination Filtering</td>
      <td>https://usegalaxy.eu</td>
      <td>c09159d7aad0f264</td>
      <td>https://usegalaxy.eu/published/workflow?id=c09159d7aad0f264</td>
      <td>Engy Nasr , Bérénice Batut, Raphaëlle Péguilhan, Pierre Amato</td>
      <td>metagenomics, metatranscriptomics, name:microgalaxy</td>
      <td>2024-11-21</td>
      <td>2024-11-21</td>
      <td>2</td>
      <td>2</td>
      <td></td>
      <td>seq_filter_by_id, bowtie2, filter_tabular, recentrifuge, kraken2, fastq_to_tabular</td>
      <td>Cross-assembly, Read mapping, Taxonomic classification, Expression analysis</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Metatranscriptomics data analysis</td>
      <td>https://usegalaxy.eu</td>
      <td>fd90652d475ed739</td>
      <td>https://usegalaxy.eu/published/workflow?id=fd90652d475ed739</td>
      <td>Valerie C. Schiml, Magnus Ø. Arntzen, Francesco Delogu, Praveen Kumar, Benoit Kunath, Bérénice Batut,  Subina Mehta, James E. Johnson,  Björn Grüning, Phillip B. Pope, Pratik D. Jagtap, Timothy J. Griffin</td>
      <td>name:microgalaxy, metatranscriptomics, microbiome</td>
      <td>2024-11-21</td>
      <td>2024-11-21</td>
      <td>5</td>
      <td>5</td>
      <td></td>
      <td>kallisto_quant, fastqc, trim_galore, bg_sortmerna, __UNZIP_COLLECTION__, collection_column_join</td>
      <td>Sequencing quality control, Read pre-processing, Sequence trimming, Sequence alignment analysis, Sequence similarity search, Gene expression profiling, Primer removal, Statistical calculation, Sequence composition calculation, Sequence comparison</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Cloud-Aerosole MT-MG Functional Profiling</td>
      <td>https://usegalaxy.eu</td>
      <td>63478edcea3f449a</td>
      <td>https://usegalaxy.eu/published/workflow?id=63478edcea3f449a</td>
      <td>Engy Nasr, Bérénice Batut, Raphaëlle Péguilhan, Pierre Amato</td>
      <td>metagenomics, metatranscriptomics, name:microgalaxy</td>
      <td>2024-11-21</td>
      <td>2024-11-21</td>
      <td>4</td>
      <td>4</td>
      <td></td>
      <td>bowtie2, multiqc, featurecounts</td>
      <td>Validation, Sequencing quality control, Read mapping, RNA-Seq quantification, Read summarisation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Cloud-Aerosole MT-MG Pre-Processing</td>
      <td>https://usegalaxy.eu</td>
      <td>1ef76b7b86e15792</td>
      <td>https://usegalaxy.eu/published/workflow?id=1ef76b7b86e15792</td>
      <td>Engy Nasr, Bérénice Batut, Raphaëlle Péguilhan, Pierre Amato</td>
      <td>metagenomics, metatranscriptomics, name:microgalaxy</td>
      <td>2024-11-21</td>
      <td>2024-11-21</td>
      <td>6</td>
      <td>6</td>
      <td></td>
      <td>fastqc, trimmomatic, multiqc</td>
      <td>Validation, Statistical calculation, Sequencing quality control, Sequence composition calculation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Allele-based Pathogen Identification (release v0.1.2)</td>
      <td>https://usegalaxy.eu</td>
      <td>64abcddeb516a712</td>
      <td>https://usegalaxy.eu/published/workflow?id=64abcddeb516a712</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-10-31</td>
      <td>2024-10-31</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>table_compute, tp_cut_tool, regexColumn1, bcftools_consensus, samtools_coverage, Paste1, clair3, CONVERTER_gz_to_uncompressed, Remove beginning1, Count1, collapse_dataset, snpSift_extractFields, samtools_depth, snpSift_filter, minimap2, Cut1, tp_head_tool, bcftools_norm</td>
      <td>Variant calling, Pairwise sequence alignment</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Taxonomy Profiling and Visualization with Kron (release v0.1)</td>
      <td>https://usegalaxy.eu</td>
      <td>ad8b6d73c9654305</td>
      <td>https://usegalaxy.eu/published/workflow?id=ad8b6d73c9654305</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-09-19</td>
      <td>2024-09-19</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>krakentools_kreport2krona, taxonomy_krona_chart, kraken2</td>
      <td>Aggregation, Taxonomic classification, Visualisation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Pathogen Detection PathoGFAIR Samples Aggregation and Visualisation (release v0.1)</td>
      <td>https://usegalaxy.eu</td>
      <td>f6a763951d815944</td>
      <td>https://usegalaxy.eu/published/workflow?id=f6a763951d815944</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-09-19</td>
      <td>2024-09-19</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>tab2fasta, newick_display, __FILTER_EMPTY_DATASETS__, fasta_merge_files_and_filter_unique_sequences, tp_split_on_column, bedtools_getfastabed, Count1, __FILTER_FAILED_DATASETS__, regexColumn1, fasta2tab, collapse_dataset, tp_replace_in_column, clustalw, regex1, Remove beginning1, Cut1, ggplot2_heatmap, tp_multijoin_tool, fasttree, Grouping1, collection_column_join, tp_sorted_uniq</td>
      <td>Phylogenetic tree generation (maximum likelihood and Bayesian methods), Phylogenetic tree generation, Multiple sequence alignment, Phylogenetic tree analysis, Visualisation, Phylogenetic tree generation (from molecular sequences), Mapping, Phylogenetic tree reconstruction</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Nanopore Preprocessing (release v0.1)</td>
      <td>https://usegalaxy.eu</td>
      <td>344cd3a3c2ce5302</td>
      <td>https://usegalaxy.eu/published/workflow?id=344cd3a3c2ce5302</td>
      <td>Bérénice Batut, Engy Nasr, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:nanopore, name:iwc</td>
      <td>2024-09-19</td>
      <td>2024-09-19</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>samtools_fastx, __FILTER_FAILED_DATASETS__, bamtools_split_mapped, krakentools_extract_kraken_reads, nanoplot, fastqc, Add_a_column1, fastp, regexColumn1, collection_column_join, Grep1, multiqc, kraken2, minimap2, porechop, Cut1, collapse_dataset</td>
      <td>Scatter plot plotting, Box-Whisker plot plotting, Sequencing quality control, Aggregation, Sequence contamination filtering, Sequence alignment analysis, Pairwise sequence alignment, Validation, Statistical calculation, Taxonomic classification, Visualisation, Sequence composition calculation, Data handling</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Gene-based Pathogen Identification (release v0.1)</td>
      <td>https://usegalaxy.eu</td>
      <td>77e5bbd317750915</td>
      <td>https://usegalaxy.eu/published/workflow?id=77e5bbd317750915</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:pathogfair, name:iwc, name:microgalaxy</td>
      <td>2024-09-19</td>
      <td>2024-09-19</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>compose_text_param, tab2fasta, medaka_consensus_pipeline, __BUILD_LIST__, collection_element_identifiers, bandage_image, tp_find_and_replace, flye, fasta2tab, split_file_to_collection, abricate, param_value_from_file</td>
      <td>Cross-assembly, Base-calling, De-novo assembly, Genome assembly, Antimicrobial resistance prediction, Mapping assembly, Sequence assembly, Variant calling, Sequence assembly visualisation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Allele-based Pathogen Identification (release v0.1)</td>
      <td>https://usegalaxy.eu</td>
      <td>d215a2e554fd83bc</td>
      <td>https://usegalaxy.eu/published/workflow?id=d215a2e554fd83bc</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-09-19</td>
      <td>2024-09-19</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>table_compute, tp_cut_tool, regexColumn1, bcftools_consensus, samtools_coverage, Paste1, clair3, CONVERTER_gz_to_uncompressed, Remove beginning1, Count1, collapse_dataset, snpSift_extractFields, samtools_depth, snpSift_filter, minimap2, Cut1, tp_head_tool, bcftools_norm</td>
      <td>Variant calling, Pairwise sequence alignment</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Allele-based Pathogen Identification (release v0.1.1)</td>
      <td>https://usegalaxy.eu</td>
      <td>cdf2f02428f5f83f</td>
      <td>https://usegalaxy.eu/published/workflow?id=cdf2f02428f5f83f</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-09-19</td>
      <td>2024-09-19</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>table_compute, tp_cut_tool, regexColumn1, bcftools_consensus, samtools_coverage, Paste1, clair3, CONVERTER_gz_to_uncompressed, Remove beginning1, Count1, collapse_dataset, snpSift_extractFields, samtools_depth, snpSift_filter, minimap2, Cut1, tp_head_tool, bcftools_norm</td>
      <td>Variant calling, Pairwise sequence alignment</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>pAllori WGS</td>
      <td>https://usegalaxy.eu</td>
      <td>4f245ac304ab76d9</td>
      <td>https://usegalaxy.eu/published/workflow?id=4f245ac304ab76d9</td>
      <td>Engy Nasr, Paul Zierep</td>
      <td>name:pallori, name:microgalaxy, name:wgs</td>
      <td>2024-07-16</td>
      <td>2024-07-28</td>
      <td>16</td>
      <td>16</td>
      <td></td>
      <td>Grouping1, regexColumn1, fastqc, bowtie2, trimmomatic, __FLATTEN__, Remove beginning1, bcftools_view, collection_column_join, vcf2tsv, freebayes, multiqc, snpSift_filter, ggplot2_heatmap, tp_awk_tool, abricate, shovill, bcftools_norm</td>
      <td>Sequencing quality control, Read mapping, Antimicrobial resistance prediction, Genome assembly, Validation, Statistical calculation, Visualisation, Variant calling, Sequence composition calculation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>MAGs workflow</td>
      <td>https://usegalaxy.eu</td>
      <td>97312d273b6e8bd9</td>
      <td>https://usegalaxy.eu/published/workflow?id=97312d273b6e8bd9</td>
      <td></td>
      <td>name:microgalaxy</td>
      <td>2023-11-15</td>
      <td>2024-07-11</td>
      <td>14</td>
      <td>14</td>
      <td></td>
      <td>das_tool, coverm_genome, bowtie2, quast, concoct, checkm_lineage_wf, concoct_extract_fasta_bins, concoct_coverage_table, Fasta_to_Contig2Bin, __UNZIP_COLLECTION__, fastq_paired_end_interlacer, semibin, samtools_sort, metabat2_jgi_summarize_bam_contig_depths, megahit, drep_dereplicate, concoct_cut_up_fasta, metabat2, concoct_merge_cut_up_clustering, maxbin2</td>
      <td>Sequence clustering, Sequencing quality control, Read mapping, Genome comparison, Genome annotation, Genome assembly, Read binning, Validation, Local alignment, Statistical calculation, Sequence assembly, Visualisation, Sequence composition calculation, Sequence assembly validation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>PathoGFAIR</td>
      <td>https://usegalaxy.eu</td>
      <td>0dce37adb369492c</td>
      <td>https://usegalaxy.eu/published/workflow?id=0dce37adb369492c</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:microgalaxy, name:iwc, name:pathogfair, name:nanopore, name:collection</td>
      <td>2024-05-31</td>
      <td>2024-07-03</td>
      <td>93</td>
      <td>93</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Pathogen Detection PathoGFAIR Samples Aggregation and Visualisation</td>
      <td>https://usegalaxy.eu</td>
      <td>376119528377a3ae</td>
      <td>https://usegalaxy.eu/published/workflow?id=376119528377a3ae</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-04-03</td>
      <td>2024-06-24</td>
      <td>58</td>
      <td>58</td>
      <td></td>
      <td>tab2fasta, newick_display, __FILTER_EMPTY_DATASETS__, fasta_merge_files_and_filter_unique_sequences, tp_split_on_column, bedtools_getfastabed, Count1, __FILTER_FAILED_DATASETS__, regexColumn1, fasta2tab, collapse_dataset, tp_replace_in_column, clustalw, regex1, Remove beginning1, Cut1, ggplot2_heatmap, tp_multijoin_tool, fasttree, Grouping1, collection_column_join, tp_sorted_uniq</td>
      <td>Phylogenetic tree generation (maximum likelihood and Bayesian methods), Phylogenetic tree generation, Multiple sequence alignment, Phylogenetic tree analysis, Visualisation, Phylogenetic tree generation (from molecular sequences), Mapping, Phylogenetic tree reconstruction</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Allele-based Pathogen Identification</td>
      <td>https://usegalaxy.eu</td>
      <td>09c7069ae409c362</td>
      <td>https://usegalaxy.eu/published/workflow?id=09c7069ae409c362</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2023-04-12</td>
      <td>2024-06-24</td>
      <td>72</td>
      <td>72</td>
      <td></td>
      <td>table_compute, tp_cut_tool, regexColumn1, bcftools_consensus, samtools_coverage, Paste1, clair3, CONVERTER_gz_to_uncompressed, Remove beginning1, Count1, collapse_dataset, snpSift_extractFields, samtools_depth, snpSift_filter, minimap2, Cut1, tp_head_tool, bcftools_norm</td>
      <td>Variant calling, Pairwise sequence alignment</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Nanopore Preprocessing</td>
      <td>https://usegalaxy.eu</td>
      <td>a705370bc2c13d5c</td>
      <td>https://usegalaxy.eu/published/workflow?id=a705370bc2c13d5c</td>
      <td>Bérénice Batut, Engy Nasr, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:nanopore, name:iwc</td>
      <td>2023-04-12</td>
      <td>2024-06-24</td>
      <td>190</td>
      <td>190</td>
      <td></td>
      <td>samtools_fastx, __FILTER_FAILED_DATASETS__, bamtools_split_mapped, krakentools_extract_kraken_reads, nanoplot, fastqc, Add_a_column1, fastp, regexColumn1, collection_column_join, Grep1, multiqc, kraken2, minimap2, porechop, Cut1, collapse_dataset</td>
      <td>Scatter plot plotting, Box-Whisker plot plotting, Sequencing quality control, Aggregation, Sequence contamination filtering, Sequence alignment analysis, Pairwise sequence alignment, Validation, Statistical calculation, Taxonomic classification, Visualisation, Sequence composition calculation, Data handling</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Taxonomy Profiling and Visualization with Phinch</td>
      <td>https://usegalaxy.eu</td>
      <td>b67b5ecd3305f830</td>
      <td>https://usegalaxy.eu/published/workflow?id=b67b5ecd3305f830</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:gtn</td>
      <td>2023-04-12</td>
      <td>2024-06-19</td>
      <td>8</td>
      <td>8</td>
      <td></td>
      <td>interactive_tool_phinch, kraken_biom, kraken2</td>
      <td>Taxonomic classification</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>metaQuantome_datacreation_workflow</td>
      <td>https://usegalaxy.eu</td>
      <td>fc42cc9467c6ee8d</td>
      <td>https://usegalaxy.eu/published/workflow?id=fc42cc9467c6ee8d</td>
      <td></td>
      <td>proteomics, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2024-06-19</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>tp_replace_in_column, regex1, tp_replace_in_line, Remove beginning1, flashlfq, Filter1, query_tabular, msconvert, search_gui, unipept, Cut1, peptide_shaker</td>
      <td>Label-free quantification, Formatting, Visualisation, Filtering, Prediction and recognition</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Gene-based Pathogen Identification</td>
      <td>https://usegalaxy.eu</td>
      <td>585c21b7b1d864fc</td>
      <td>https://usegalaxy.eu/published/workflow?id=585c21b7b1d864fc</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:pathogfair, name:iwc, name:microgalaxy</td>
      <td>2023-04-13</td>
      <td>2024-06-05</td>
      <td>142</td>
      <td>142</td>
      <td></td>
      <td>compose_text_param, tab2fasta, medaka_consensus_pipeline, __BUILD_LIST__, collection_element_identifiers, bandage_image, tp_find_and_replace, flye, fasta2tab, split_file_to_collection, abricate, param_value_from_file</td>
      <td>Cross-assembly, Base-calling, De-novo assembly, Genome assembly, Antimicrobial resistance prediction, Mapping assembly, Sequence assembly, Variant calling, Sequence assembly visualisation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Taxonomy Profiling and Visualization with Krona</td>
      <td>https://usegalaxy.eu</td>
      <td>10101558b211a782</td>
      <td>https://usegalaxy.eu/published/workflow?id=10101558b211a782</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2023-05-11</td>
      <td>2024-06-05</td>
      <td>58</td>
      <td>58</td>
      <td></td>
      <td>krakentools_kreport2krona, taxonomy_krona_chart, kraken2</td>
      <td>Aggregation, Taxonomic classification, Visualisation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Pathogen-Detection-Nanopore-All-Samples-Analysis-Juyptool</td>
      <td>https://usegalaxy.eu</td>
      <td>c8bc4af6dd3b5d02</td>
      <td>https://usegalaxy.eu/published/workflow?id=c8bc4af6dd3b5d02</td>
      <td>Engy Nasr, Bérénice Batut</td>
      <td>microbiome, nanopore, iwc, name:collection, name:microgalaxy</td>
      <td>2022-11-29</td>
      <td>2024-04-04</td>
      <td>205</td>
      <td>205</td>
      <td></td>
      <td>tab2fasta, newick_display, __FILTER_EMPTY_DATASETS__, fasta_merge_files_and_filter_unique_sequences, tp_split_on_column, bedtools_getfastabed, interactive_tool_jupyter_notebook, Count1, __FILTER_FAILED_DATASETS__, regexColumn1, fasta2tab, collapse_dataset, tp_replace_in_column, clustalw, regex1, Remove beginning1, Cut1, ggplot2_heatmap, tp_multijoin_tool, fasttree, Grouping1, collection_column_join, tp_sorted_uniq</td>
      <td>Phylogenetic tree generation (maximum likelihood and Bayesian methods), Phylogenetic tree generation, Multiple sequence alignment, Phylogenetic tree analysis, Visualisation, Phylogenetic tree generation (from molecular sequences), Mapping, Phylogenetic tree reconstruction</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Mgnify ML</td>
      <td>https://usegalaxy.eu</td>
      <td>011f701c5a55bfdf</td>
      <td>https://usegalaxy.eu/published/workflow?id=011f701c5a55bfdf</td>
      <td></td>
      <td>name:microgalaxy</td>
      <td>2023-10-10</td>
      <td>2024-03-15</td>
      <td>128</td>
      <td>128</td>
      <td></td>
      <td>interactive_tool_jupyter_notebook, __BUILD_LIST__</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Normalization</td>
      <td>https://usegalaxy.eu</td>
      <td>0603f0898211f806</td>
      <td>https://usegalaxy.eu/published/workflow?id=0603f0898211f806</td>
      <td></td>
      <td>name:microgalaxy</td>
      <td>2023-09-25</td>
      <td>2024-03-13</td>
      <td>80</td>
      <td>80</td>
      <td></td>
      <td>__EXTRACT_DATASET__, __BUILD_LIST__, __FILTER_EMPTY_DATASETS__, map_param_value, Grep1</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>WF1_Database_Generation_Workflow</td>
      <td>https://usegalaxy.eu</td>
      <td>7cfbf4b7bd3f30d3</td>
      <td>https://usegalaxy.eu/published/workflow?id=7cfbf4b7bd3f30d3</td>
      <td></td>
      <td>name:clinicalmp, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>metanovo, fasta_merge_files_and_filter_unique_sequences</td>
      <td>Target-Decoy, Tag-based peptide identification, de Novo sequencing, Protein identification, Expression analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>WF2_Discovery-Workflow</td>
      <td>https://usegalaxy.eu</td>
      <td>8af41219411062ad</td>
      <td>https://usegalaxy.eu/published/workflow?id=8af41219411062ad</td>
      <td></td>
      <td>name:microgalaxy, name:clinicalmp</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>Grouping1, peptide_shaker, filter_tabular, Remove beginning1, Filter1, query_tabular, fasta_merge_files_and_filter_unique_sequences, msconvert, search_gui, dbbuilder, Grep1, tp_cat, maxquant, fasta2tab, fasta_cli, Cut1, ident_params</td>
      <td>Formatting, Clustering, Heat map generation, Principal component plotting, Statistical calculation, Visualisation, Filtering, Protein quantification, Imputation, Standardisation and normalisation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>WF3_VERIFICATION_WORKFLOW</td>
      <td>https://usegalaxy.eu</td>
      <td>d87fc6685e47d27d</td>
      <td>https://usegalaxy.eu/published/workflow?id=d87fc6685e47d27d</td>
      <td></td>
      <td>name:microgalaxy, name:clinicalmp</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>Grouping1, uniprotxml_downloader, Remove beginning1, Filter1, query_tabular, fasta_merge_files_and_filter_unique_sequences, dbbuilder, tp_cat, pepquery2, Cut1, collapse_dataset</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>WF4_Quantitation_Workflow</td>
      <td>https://usegalaxy.eu</td>
      <td>a49275eb8b2b2ecb</td>
      <td>https://usegalaxy.eu/published/workflow?id=a49275eb8b2b2ecb</td>
      <td></td>
      <td>name:clinicalmp, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>maxquant, Grep1, Cut1, Grouping1</td>
      <td>Clustering, Heat map generation, Principal component plotting, Statistical calculation, Visualisation, Protein quantification, Imputation, Standardisation and normalisation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>WF5_Data_Interpretation_Worklow</td>
      <td>https://usegalaxy.eu</td>
      <td>755ede37d94dc456</td>
      <td>https://usegalaxy.eu/published/workflow?id=755ede37d94dc456</td>
      <td></td>
      <td>name:microgalaxy, name:clinicalmp</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>unipept, Grep1, msstatstmt</td>
      <td>Visualisation, Prediction and recognition</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>MetaG_extended</td>
      <td>https://usegalaxy.eu</td>
      <td>d57d41e306241396</td>
      <td>https://usegalaxy.eu/published/workflow?id=d57d41e306241396</td>
      <td></td>
      <td>name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>coverm_contig, fraggenescan, coverm_genome, fastqc, trim_galore, tp_grep_tool, quast, sort1, kofamscan, checkm_lineage_wf, tp_awk_tool, tp_cat, hmmer_hmmscan, maxbin2, interproscan, cat_bins, fasplit</td>
      <td>Sequencing quality control, Sequence generation, Sequence database search, Primer removal, Conversion, Sequence profile generation, Sequence motif recognition, Gene functional annotation, Protein feature detection, Sequence assembly validation, Sequence analysis, Multiple sequence alignment, Read pre-processing, Sequence trimming, Formatting, Probabilistic sequence generation, Data retrieval, Format validation, Validation, Local alignment, Statistical calculation, Sequence assembly, Taxonomic classification, Visualisation, Database search, Gene prediction, Sequence composition calculation, Coding region prediction</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>MetaP</td>
      <td>https://usegalaxy.eu</td>
      <td>01ac8bf9832789e2</td>
      <td>https://usegalaxy.eu/published/workflow?id=01ac8bf9832789e2</td>
      <td></td>
      <td>name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>maxquant</td>
      <td>Clustering, Heat map generation, Principal component plotting, Statistical calculation, Visualisation, Protein quantification, Imputation, Standardisation and normalisation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>MetaT</td>
      <td>https://usegalaxy.eu</td>
      <td>3733bcdf5b458be7</td>
      <td>https://usegalaxy.eu/published/workflow?id=3733bcdf5b458be7</td>
      <td></td>
      <td>name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>kallisto_quant, fastqc, trim_galore, bg_sortmerna, __UNZIP_COLLECTION__, collection_column_join</td>
      <td>Sequencing quality control, Read pre-processing, Sequence trimming, Sequence alignment analysis, Sequence similarity search, Gene expression profiling, Primer removal, Statistical calculation, Sequence composition calculation, Sequence comparison</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>metaquantome-taxonomy-workflow</td>
      <td>https://usegalaxy.eu</td>
      <td>695f7defb516d80d</td>
      <td>https://usegalaxy.eu/published/workflow?id=695f7defb516d80d</td>
      <td></td>
      <td>proteomics, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>metaquantome_stat, metaquantome_viz, metaquantome_db, metaquantome_sample, metaquantome_filter, metaquantome_expand</td>
      <td>Functional clustering, Principal component visualisation, Statistical inference, Differential protein expression analysis, Query and retrieval, Heat map generation, Quantification, Visualisation, Filtering, Indexing</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>metaquantome-function-worklow</td>
      <td>https://usegalaxy.eu</td>
      <td>4c364c6be27981ba</td>
      <td>https://usegalaxy.eu/published/workflow?id=4c364c6be27981ba</td>
      <td></td>
      <td>proteomics, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>metaquantome_stat, metaquantome_viz, metaquantome_db, metaquantome_sample, metaquantome_filter, metaquantome_expand</td>
      <td>Functional clustering, Principal component visualisation, Statistical inference, Differential protein expression analysis, Query and retrieval, Heat map generation, Quantification, Visualisation, Filtering, Indexing</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Metaproteomics_GTN</td>
      <td>https://usegalaxy.eu</td>
      <td>68967c922de149ae</td>
      <td>https://usegalaxy.eu/published/workflow?id=68967c922de149ae</td>
      <td></td>
      <td>proteomics, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>sqlite_to_tabular, query_tabular, search_gui, unipept, peptide_shaker</td>
      <td>Visualisation, Prediction and recognition</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Ecoli Comparison</td>
      <td>https://usegalaxy.eu</td>
      <td>0e492839192d26e0</td>
      <td>https://usegalaxy.eu/published/workflow?id=0e492839192d26e0</td>
      <td></td>
      <td>assembly, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>tp_sed_tool, bedtools_sortbed, tp_grep_tool, fasta_compute_length, Grep1, tp_cat, collapse_dataset, join1, bedtools_intersectbed, datamash_ops, mergeCols1, Cut1, cat1, bedtools_complementbed, fasta_filter_by_length, Filter1, addValue, lastz_wrapper_2, random_lines1, tp_sort_header_tool</td>
      <td>Mapping, Sequence alignment, Read mapping</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Training: 16S rRNA Analysis with Nanopore Sequencing Reads</td>
      <td>https://usegalaxy.eu</td>
      <td>08eb5e55109c4d9c</td>
      <td>https://usegalaxy.eu/published/workflow?id=08eb5e55109c4d9c</td>
      <td></td>
      <td>metagenomics, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>fastqc, tp_replace_in_line, datamash_reverse, fastp, taxonomy_krona_chart, Remove beginning1, multiqc, kraken2, porechop</td>
      <td>Validation, Statistical calculation, Sequencing quality control, Taxonomic classification, Visualisation, Sequence contamination filtering, Sequence composition calculation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Main Metatranscriptomics Analysis</td>
      <td>https://usegalaxy.eu</td>
      <td>7a08ecdf23e0c7a3</td>
      <td>https://usegalaxy.eu/published/workflow?id=7a08ecdf23e0c7a3</td>
      <td>Bérénice Batut, Pratik Jagtap, Subina Mehta, Ray Sajulga, Emma Leith, Praveen Kumar, Saskia Hiltemann, Paul Zierep</td>
      <td>name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Copy Of GTN Training - Antibiotic Resistance Detection</td>
      <td>https://usegalaxy.eu</td>
      <td>44e53ea319e07daa</td>
      <td>https://usegalaxy.eu/published/workflow?id=44e53ea319e07daa</td>
      <td></td>
      <td>metagenomics, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>unicycler, gfa_to_fa, racon, nanoplot, bandage_image, miniasm, staramr_search, minimap2, PlasFlow</td>
      <td>Scatter plot plotting, Box-Whisker plot plotting, Sequence analysis, Aggregation, De-novo assembly, Genome assembly, Pairwise sequence alignment, Mapping assembly, Sequence assembly visualisation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Identification of the micro-organisms in a beer using Nanopore sequencing</td>
      <td>https://usegalaxy.eu</td>
      <td>0590554db9794d13</td>
      <td>https://usegalaxy.eu/published/workflow?id=0590554db9794d13</td>
      <td>Bérénice Batut, Teresa Müller, Polina Polunina</td>
      <td>metagenomics, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>fastqc, fastp, taxonomy_krona_chart, Filter1, krakentools_kreport2krona, kraken2, porechop</td>
      <td>Statistical calculation, Sequencing quality control, Aggregation, Taxonomic classification, Visualisation, Sequence contamination filtering, Sequence composition calculation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>workflow-generate-dataset-for-assembly-tutorial</td>
      <td>https://usegalaxy.eu</td>
      <td>ce5c16ac23c4fbb1</td>
      <td>https://usegalaxy.eu/published/workflow?id=ce5c16ac23c4fbb1</td>
      <td></td>
      <td>name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>tp_cat, seqtk_subseq, fastqc, bamtools, bowtie2, filter_tabular, bg_uniq, megahit, random_lines1, cutadapt, ngsutils_bam_filter</td>
      <td>Sequencing quality control, Read pre-processing, Read mapping, Sequence trimming, Formatting, Sequence alignment analysis, Sequence contamination filtering, Genome assembly, Primer removal, Sequence file editing, Statistical calculation, Variant calling, Sequence composition calculation, Data handling</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Metagenomics assembly tutorial workflow</td>
      <td>https://usegalaxy.eu</td>
      <td>62c47e3c9d4f3aa3</td>
      <td>https://usegalaxy.eu/published/workflow?id=62c47e3c9d4f3aa3</td>
      <td></td>
      <td>name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>bandage_info, megahit_contig2fastg, bowtie2, quast, collection_column_join, bandage_image, metaspades, megahit</td>
      <td>Read mapping, Visualisation, Genome assembly, Sequence assembly validation, Sequence assembly visualisation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>WGS Part In "Analyses Of Metagenomics Data - The Global Picture"</td>
      <td>https://usegalaxy.eu</td>
      <td>c2b7ec5aacfc6107</td>
      <td>https://usegalaxy.eu/published/workflow?id=c2b7ec5aacfc6107</td>
      <td></td>
      <td>metagenomics, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>humann2_regroup_table, metaphlan2, humann2_renorm_table, taxonomy_krona_chart, metaphlan2krona, humann2</td>
      <td>Visualisation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Taxonomic Profiling and Visualization of Metagenomic Data</td>
      <td>https://usegalaxy.eu</td>
      <td>4f0daf8e9dd54127</td>
      <td>https://usegalaxy.eu/published/workflow?id=4f0daf8e9dd54127</td>
      <td>Bérénice Batut</td>
      <td>metagenomics, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>metaphlan, __UNZIP_COLLECTION__, taxonomy_krona_chart, kraken_biom, est_abundance, krakentools_kreport2krona, kraken2, interactive_tool_pavian, interactive_tool_phinch</td>
      <td>Statistical calculation, Aggregation, Phylogenetic tree analysis, Visualisation, Taxonomic classification, Nucleic acid sequence analysis</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Amplicon Tutorial</td>
      <td>https://usegalaxy.eu</td>
      <td>565c989730f68362</td>
      <td>https://usegalaxy.eu/published/workflow?id=565c989730f68362</td>
      <td></td>
      <td>metagenomics, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>mothur_classify_seqs, mothur_merge_files, mothur_filter_seqs, mothur_make_biom, mothur_make_group, mothur_screen_seqs, mothur_align_seqs, mothur_cluster_split, mothur_pre_cluster, mothur_classify_otu, mothur_make_shared, mothur_count_seqs, mothur_unique_seqs, krona-text, mothur_summary_seqs</td>
      <td>Sequence clustering, Sequencing quality control, Sequence read processing, DNA barcoding, Taxonomic classification, Visualisation, Phylogenetic analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Microbial Variant Calling</td>
      <td>https://usegalaxy.eu</td>
      <td>515ef1601bc24a6c</td>
      <td>https://usegalaxy.eu/published/workflow?id=515ef1601bc24a6c</td>
      <td></td>
      <td>variant-analysis, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>jbrowse, snippy</td>
      <td>Phylogenetic tree visualisation, Phylogenetic tree generation, Genome visualisation, Variant calling</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Calling variants in non-diploid systems</td>
      <td>https://usegalaxy.eu</td>
      <td>291c36e1b18e9deb</td>
      <td>https://usegalaxy.eu/published/workflow?id=291c36e1b18e9deb</td>
      <td></td>
      <td>variant-analysis, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>fastqc, bamFilter, vcffilter2, picard_MergeSamFiles, vcf2tsv, freebayes, bwa_mem, bamleftalign, picard_MarkDuplicates, Cut1</td>
      <td>Sequencing quality control, Genome indexing, Formatting, Read mapping, Sequence alignment analysis, Sequence alignment, Statistical calculation, Variant calling, Sequence composition calculation, Data handling, Generation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Training: 16S rRNA Sequencing With Mothur: Main Tutorial</td>
      <td>https://usegalaxy.eu</td>
      <td>c67d2e75eeb4fd19</td>
      <td>https://usegalaxy.eu/published/workflow?id=c67d2e75eeb4fd19</td>
      <td></td>
      <td>metagenomics, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>mothur_seq_error, newick_display, mothur_dist_seqs, mothur_make_shared, mothur_remove_groups, mothur_count_groups, mothur_tree_shared, mothur_dist_shared, mothur_filter_seqs, mothur_cluster_split, mothur_pre_cluster, XY_Plot_1, mothur_unique_seqs, mothur_venn, mothur_summary_seqs, mothur_make_biom, mothur_sub_sample, mothur_heatmap_sim, mothur_remove_seqs, mothur_screen_seqs, mothur_classify_otu, mothur_chimera_vsearch, mothur_classify_seqs, mothur_summary_single, mothur_taxonomy_to_krona, mothur_align_seqs, mothur_rarefaction_single, taxonomy_krona_chart, mothur_remove_lineage, mothur_cluster, mothur_count_seqs, mothur_get_groups, mothur_make_contigs</td>
      <td>Sequence clustering, Phylogenetic tree generation, Sequencing quality control, Sequence read processing, DNA barcoding, Taxonomic classification, Visualisation, Phylogenetic tree analysis, Phylogenetic analysis, Phylogenetic tree reconstruction</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Comparative gene analysis</td>
      <td>https://usegalaxy.eu</td>
      <td>86900be4282d4b1d</td>
      <td>https://usegalaxy.eu/published/workflow?id=86900be4282d4b1d</td>
      <td>Anton Nekrutenko</td>
      <td>vgp, genome-annotation, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>tab2fasta, rapidnj, regexColumn1, Add_a_column1, cat1, Filter1, collapse_dataset, tp_split_on_column, bg_diamond, bg_diamond_view, gops_intersect_1, Cut1, join1, rbc_mafft</td>
      <td>Sequence alignment analysis, Phylogenetic tree generation, Multiple sequence alignment</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Unicycler training</td>
      <td>https://usegalaxy.eu</td>
      <td>8f3ce55d44d9f369</td>
      <td>https://usegalaxy.eu/published/workflow?id=8f3ce55d44d9f369</td>
      <td></td>
      <td>assembly, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>unicycler, fastqc, prokka, quast, multiqc</td>
      <td>Sequencing quality control, Aggregation, Genome annotation, Genome assembly, Sequence assembly validation, Validation, Statistical calculation, Visualisation, Gene prediction, Sequence composition calculation, Coding region prediction</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Apollo Load Test</td>
      <td>https://usegalaxy.eu</td>
      <td>3c233cc56d4df8b0</td>
      <td>https://usegalaxy.eu/published/workflow?id=3c233cc56d4df8b0</td>
      <td></td>
      <td>genome-annotation, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>create_account, create_or_update, jbrowse, list_organism, iframe</td>
      <td>Genome visualisation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation with Prokka [Feb 2020]</td>
      <td>https://usegalaxy.eu</td>
      <td>e1894c36f1ab4ab4</td>
      <td>https://usegalaxy.eu/published/workflow?id=e1894c36f1ab4ab4</td>
      <td></td>
      <td>genome-annotation, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>prokka, jbrowse</td>
      <td>Gene prediction, Coding region prediction, Genome annotation, Genome visualisation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Flye-AMR workflow EMC/WMDI - v3.5  (imported from URL)</td>
      <td>https://usegalaxy.eu</td>
      <td>152223c8aed27357</td>
      <td>https://usegalaxy.eu/published/workflow?id=152223c8aed27357</td>
      <td></td>
      <td>wmdi, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>circos_interval_to_tile, tp_cut_tool, tab2fasta, circos, tp_text_file_with_recurring_lines, nanoplot, prokka, ncbi_blastn_wrapper, Add_a_column1, tp_split_on_column, gff2bed1, jbrowse, medaka_consensus_pipeline, fasta_compute_length, bandage_image, Grep1, circos_wiggle_to_scatter, flye, tp_cat, fasta2tab, staramr_search, join1, bg_column_arrange_by_header, regex1, circos_gc_skew, PlasFlow, Cut1, tp_head_tool, regex_replace, trimmomatic, Filter1, tbl2gff3, deeptools_bam_coverage, minimap2, tp_easyjoin_tool, tp_sort_header_tool</td>
      <td>Scatter plot plotting, Box-Whisker plot plotting, Sequence analysis, Cross-assembly, Base-calling, Genome annotation, Sequence visualisation, De-novo assembly, Genome assembly, Mapping assembly, Pairwise sequence alignment, Sequence assembly, Genome visualisation, Variant calling, Gene prediction, Coding region prediction, Sequence assembly visualisation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Mtb phylogeny</td>
      <td>https://usegalaxy.eu</td>
      <td>875b3ca170ac12a1</td>
      <td>https://usegalaxy.eu/published/workflow?id=875b3ca170ac12a1</td>
      <td></td>
      <td>evolution, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>interactive_tool_rstudio, raxml</td>
      <td>Sequence analysis, Phylogenetic tree analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>From VCFs to SNP distance matrix</td>
      <td>https://usegalaxy.eu</td>
      <td>6d333ed886aab471</td>
      <td>https://usegalaxy.eu/published/workflow?id=6d333ed886aab471</td>
      <td></td>
      <td>evolution, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>snp_dists, bcftools_consensus, tb_variant_filter, snp_sites, tp_cat</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>From Fastqs to VCFs and BAMs</td>
      <td>https://usegalaxy.eu</td>
      <td>3a8fde248a4d0383</td>
      <td>https://usegalaxy.eu/published/workflow?id=3a8fde248a4d0383</td>
      <td></td>
      <td>evolution, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>tb_variant_filter, trimmomatic, __MERGE_COLLECTION__, snippy</td>
      <td>Phylogenetic tree visualisation, Phylogenetic tree generation, Variant calling</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>From BAMs to drug resistance prediction with TB-profiler</td>
      <td>https://usegalaxy.eu</td>
      <td>d0d6dae0d1eecc21</td>
      <td>https://usegalaxy.eu/published/workflow?id=d0d6dae0d1eecc21</td>
      <td></td>
      <td>evolution, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>tp_cat, tp_grep_tool, tp_replace_in_line, samtools_view, tb_profiler_profile, tp_sed_tool, __MERGE_COLLECTION__, addName</td>
      <td>Antimicrobial resistance prediction</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>M. tuberculosis Variant Analysis tutorial</td>
      <td>https://usegalaxy.eu</td>
      <td>3ff4dbcc7a602e6b</td>
      <td>https://usegalaxy.eu/published/workflow?id=3ff4dbcc7a602e6b</td>
      <td>Peter van Heusden</td>
      <td>variant-analysis, prokaryote, name:microgalaxy</td>
      <td>2023-12-14</td>
      <td>2023-12-14</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td>snippy, fastqc, tb_variant_filter, trimmomatic, tb_profiler_profile, multiqc, jbrowse, kraken2, tbvcfreport</td>
      <td>Phylogenetic tree generation, Sequencing quality control, Antimicrobial resistance prediction, Variant calling, Validation, Statistical calculation, Genome visualisation, Taxonomic classification, Phylogenetic tree visualisation, Sequence composition calculation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Compute Allel Frequency Table from paired reads</td>
      <td>https://usegalaxy.eu</td>
      <td>21a39ac4c1becb31</td>
      <td>https://usegalaxy.eu/published/workflow?id=21a39ac4c1becb31</td>
      <td></td>
      <td>name:microgalaxy, hpylori</td>
      <td>2023-12-13</td>
      <td>2023-12-13</td>
      <td>2</td>
      <td>2</td>
      <td></td>
      <td>bowtie2, bcftools_view, vcf2tsv, freebayes, snpSift_filter, tp_awk_tool, bcftools_norm</td>
      <td>Variant calling, Statistical calculation, Read mapping</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Reference genome choice (indels, coverage, depth) for short reads</td>
      <td>https://usegalaxy.eu</td>
      <td>61fe08fb3d5e83f8</td>
      <td>https://usegalaxy.eu/published/workflow?id=61fe08fb3d5e83f8</td>
      <td></td>
      <td>name:microgalaxy, hpylori</td>
      <td>2023-10-27</td>
      <td>2023-12-13</td>
      <td>13</td>
      <td>13</td>
      <td></td>
      <td>bowtie2, samtools_coverage, collection_element_identifiers, vcf2tsv, freebayes, samtools_depth, tp_cat, collapse_dataset, bcftools_norm</td>
      <td>Variant calling, Statistical calculation, Read mapping</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>MGnify amplicon rRNA-prediction + ITS</td>
      <td>https://usegalaxy.eu</td>
      <td>437704898229dfb6</td>
      <td>https://usegalaxy.eu/published/workflow?id=437704898229dfb6</td>
      <td></td>
      <td>mgnify_amplicon_showcase, name:microgalaxy</td>
      <td>2023-12-12</td>
      <td>2023-12-13</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>infernal_cmsearch, mapseq, taxonomy_krona_chart, query_tabular, cmsearch_deoverlap, bedtools_maskfastabed, bedtools_getfastabed, tp_awk_tool, gops_concat_1, biom_convert</td>
      <td>k-mer counting, Alignment, Formatting, Visualisation, Mapping, Nucleic acid feature detection, Comparison</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>MGnify amplicon QC</td>
      <td>https://usegalaxy.eu</td>
      <td>8f415b0c409f56f7</td>
      <td>https://usegalaxy.eu/published/workflow?id=8f415b0c409f56f7</td>
      <td></td>
      <td>mgnify_amplicon_showcase, name:microgalaxy</td>
      <td>2023-12-13</td>
      <td>2023-12-13</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>fastq_to_fasta_python, fastqc, fastq_dump, __UNZIP_COLLECTION__, trimmomatic, iuc_pear, tp_find_and_replace, prinseq, multiqc, cshl_fasta_formatter, fastp</td>
      <td>Sequencing quality control, Read pre-processing, Sequence trimming, Sequence contamination filtering, Sequence merging, Validation, Statistical calculation, Sequence composition calculation, Data handling</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Cloud-Aerosole MT-MG Functional Profiling</td>
      <td>https://usegalaxy.eu</td>
      <td>4566262dd0acc8ab</td>
      <td>https://usegalaxy.eu/published/workflow?id=4566262dd0acc8ab</td>
      <td></td>
      <td>metagenomics, metatranscriptomics, name:microgalaxy</td>
      <td>2021-11-10</td>
      <td>2023-12-07</td>
      <td>2</td>
      <td>2</td>
      <td></td>
      <td>bowtie2, multiqc, featurecounts</td>
      <td>Validation, Sequencing quality control, Read mapping, RNA-Seq quantification, Read summarisation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Cloud-Aerosole MT-MG Reference Gene Catalog Building</td>
      <td>https://usegalaxy.eu</td>
      <td>9b9e1e28f3683f92</td>
      <td>https://usegalaxy.eu/published/workflow?id=9b9e1e28f3683f92</td>
      <td></td>
      <td>metagenomics, metatranscriptomics, name:microgalaxy</td>
      <td>2021-11-01</td>
      <td>2023-12-07</td>
      <td>72</td>
      <td>72</td>
      <td></td>
      <td>translate_nucleotides, fastqc, bowtie2, quast, metaeuk_easy_predict, eggnog_mapper, cd_hit, multiqc, dbbuilder, megahit, bg_diamond, kofamscan, metagene_annotator, bedtools_getfastabed, cat1, bg_diamond_makedb</td>
      <td>Sequencing quality control, Read mapping, Mapping, Homology-based gene prediction, Genome assembly, Fold recognition, Gene functional annotation, Sequence assembly validation, Sequence clustering, Sequence analysis, Query and retrieval, Sequence alignment analysis, Genome annotation, Information extraction, Validation, Statistical calculation, Visualisation, Sequence annotation, Sequence composition calculation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Cloud-Aerosole MT-MG Pre-Processing</td>
      <td>https://usegalaxy.eu</td>
      <td>bb604ff5e010e7ab</td>
      <td>https://usegalaxy.eu/published/workflow?id=bb604ff5e010e7ab</td>
      <td></td>
      <td>metagenomics, metatranscriptomics, name:microgalaxy</td>
      <td>2021-11-01</td>
      <td>2023-12-07</td>
      <td>51</td>
      <td>51</td>
      <td></td>
      <td>fastqc, trimmomatic, multiqc</td>
      <td>Validation, Statistical calculation, Sequencing quality control, Sequence composition calculation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Cloud Aerosol MT-MG Contamination Filtering</td>
      <td>https://usegalaxy.eu</td>
      <td>bdc7cfaac0db2eab</td>
      <td>https://usegalaxy.eu/published/workflow?id=bdc7cfaac0db2eab</td>
      <td></td>
      <td>metagenomics, metatranscriptomics, name:microgalaxy</td>
      <td>2022-06-29</td>
      <td>2023-12-07</td>
      <td>28</td>
      <td>28</td>
      <td></td>
      <td>seq_filter_by_id, bowtie2, filter_tabular, recentrifuge, kraken2, fastq_to_tabular</td>
      <td>Cross-assembly, Read mapping, Taxonomic classification, Expression analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Cloud-Aerosole MT-MG Taxonomic Profiling</td>
      <td>https://usegalaxy.eu</td>
      <td>e69b555faa6bb962</td>
      <td>https://usegalaxy.eu/published/workflow?id=e69b555faa6bb962</td>
      <td></td>
      <td>metagenomics, metatranscriptomics, name:microgalaxy</td>
      <td>2022-09-03</td>
      <td>2023-12-07</td>
      <td>18</td>
      <td>18</td>
      <td></td>
      <td>kraken_biom, recentrifuge, kraken2, maaslin2, interactive_tool_phinch</td>
      <td>Statistical calculation, Cross-assembly, Taxonomic classification, Visualisation, Filtering, Standardisation and normalisation, Expression analysis</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>pAllori ASiam Functional Information MT</td>
      <td>https://usegalaxy.eu</td>
      <td>a7a6d8ecc2795f58</td>
      <td>https://usegalaxy.eu/published/workflow?id=a7a6d8ecc2795f58</td>
      <td>Bérénice Batut, Pratik Jagtap, Subina Mehta, Ray Sajulga, Emma Leith, Praveen Kumar, Saskia Hiltemann, Paul Zierep, Engy Nasr</td>
      <td>name:microgalaxy, name:metatranscriptomics, name:adaptedasiam, name:functionalprofiling</td>
      <td>2023-07-07</td>
      <td>2023-07-07</td>
      <td>3</td>
      <td>3</td>
      <td></td>
      <td>humann_unpack_pathways, humann_regroup_table, humann, tp_find_and_replace, combine_metaphlan2_humann2, Grep1, humann_split_stratified_table, humann_rename_table, humann_renorm_table, Cut1</td>
      <td>Taxonomic classification, Species frequency estimation, Phylogenetic analysis</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>pAllori ASiam Community Profile MT</td>
      <td>https://usegalaxy.eu</td>
      <td>05f5f040337786bf</td>
      <td>https://usegalaxy.eu/published/workflow?id=05f5f040337786bf</td>
      <td>Engy Nasr, Bérénice Batut</td>
      <td>name:microgalaxy, name:metatranscriptomics, name:adaptedasiam, name:communityprofiling</td>
      <td>2023-06-30</td>
      <td>2023-07-06</td>
      <td>3</td>
      <td>3</td>
      <td></td>
      <td>graphlan, metaphlan, export2graphlan, taxonomy_krona_chart, graphlan_annotate, Cut1</td>
      <td>Conversion, Phylogenetic inference, Taxonomic classification, Phylogenetic tree analysis, Visualisation, Phylogenetic tree visualisation, Nucleic acid sequence analysis, Phylogenetic tree editing</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>pAllori ASiam Preprocessing MT</td>
      <td>https://usegalaxy.eu</td>
      <td>72e3e5fdc766e24a</td>
      <td>https://usegalaxy.eu/published/workflow?id=72e3e5fdc766e24a</td>
      <td>Bérénice Batut, Pratik Jagtap, Subina Mehta, Ray Sajulga, Emma Leith, Praveen Kumar, Saskia Hiltemann, Paul Zierep, Engy Nasr</td>
      <td>name:microgalaxy, name:adaptedasiam, name:metatranscriptomics, name:preprocessing</td>
      <td>2023-06-30</td>
      <td>2023-07-06</td>
      <td>2</td>
      <td>2</td>
      <td></td>
      <td>fastq_paired_end_interlacer, fastqc, bg_sortmerna, __FLATTEN__, multiqc, cutadapt</td>
      <td>Sequencing quality control, Read pre-processing, Sequence trimming, Sequence alignment analysis, Sequence similarity search, Primer removal, Validation, Statistical calculation, Sequence composition calculation, Sequence comparison</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>MetaP</td>
      <td>https://usegalaxy.eu</td>
      <td>96d198ed953449fd</td>
      <td>https://usegalaxy.eu/published/workflow?id=96d198ed953449fd</td>
      <td></td>
      <td>name:microgalaxy</td>
      <td>2020-01-30</td>
      <td>2023-03-16</td>
      <td>2</td>
      <td>2</td>
      <td></td>
      <td>maxquant</td>
      <td>Clustering, Heat map generation, Principal component plotting, Statistical calculation, Visualisation, Protein quantification, Imputation, Standardisation and normalisation</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>MetaG_extended</td>
      <td>https://usegalaxy.eu</td>
      <td>6158c3b5fc12044e</td>
      <td>https://usegalaxy.eu/published/workflow?id=6158c3b5fc12044e</td>
      <td></td>
      <td>name:microgalaxy</td>
      <td>2023-02-10</td>
      <td>2023-03-16</td>
      <td>22</td>
      <td>22</td>
      <td></td>
      <td>coverm_contig, fraggenescan, coverm_genome, fastqc, trim_galore, tp_grep_tool, quast, sort1, kofamscan, checkm_lineage_wf, tp_awk_tool, tp_cat, hmmer_hmmscan, maxbin2, interproscan, cat_bins, fasplit</td>
      <td>Sequencing quality control, Sequence generation, Sequence database search, Primer removal, Conversion, Sequence profile generation, Sequence motif recognition, Gene functional annotation, Protein feature detection, Sequence assembly validation, Sequence analysis, Multiple sequence alignment, Read pre-processing, Sequence trimming, Formatting, Probabilistic sequence generation, Data retrieval, Format validation, Validation, Local alignment, Statistical calculation, Sequence assembly, Taxonomic classification, Visualisation, Database search, Gene prediction, Sequence composition calculation, Coding region prediction</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>MetaT</td>
      <td>https://usegalaxy.eu</td>
      <td>385cc7df70d7916b</td>
      <td>https://usegalaxy.eu/published/workflow?id=385cc7df70d7916b</td>
      <td></td>
      <td>name:microgalaxy</td>
      <td>2020-01-27</td>
      <td>2023-02-27</td>
      <td>16</td>
      <td>16</td>
      <td></td>
      <td>kallisto_quant, fastqc, trim_galore, bg_sortmerna, __UNZIP_COLLECTION__, collection_column_join</td>
      <td>Sequencing quality control, Read pre-processing, Sequence trimming, Sequence alignment analysis, Sequence similarity search, Gene expression profiling, Primer removal, Statistical calculation, Sequence composition calculation, Sequence comparison</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Allele-based Pathogen Identification (release v0.1.1)</td>
      <td>https://usegalaxy.org</td>
      <td>29ba77708cdafe0f</td>
      <td>https://usegalaxy.org/published/workflow?id=29ba77708cdafe0f</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-11-28</td>
      <td>2024-11-28</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>table_compute, tp_cut_tool, regexColumn1, bcftools_consensus, samtools_coverage, Paste1, clair3, CONVERTER_gz_to_uncompressed, Remove beginning1, Count1, collapse_dataset, snpSift_extractFields, samtools_depth, snpSift_filter, minimap2, Cut1, tp_head_tool, bcftools_norm</td>
      <td>Variant calling, Pairwise sequence alignment</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Allele-based Pathogen Identification (release v0.1.2)</td>
      <td>https://usegalaxy.org</td>
      <td>6d778eeb176d563c</td>
      <td>https://usegalaxy.org/published/workflow?id=6d778eeb176d563c</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-10-31</td>
      <td>2024-10-31</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>table_compute, tp_cut_tool, regexColumn1, bcftools_consensus, samtools_coverage, Paste1, clair3, CONVERTER_gz_to_uncompressed, Remove beginning1, Count1, collapse_dataset, snpSift_extractFields, samtools_depth, snpSift_filter, minimap2, Cut1, tp_head_tool, bcftools_norm</td>
      <td>Variant calling, Pairwise sequence alignment</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Taxonomy Profiling and Visualization with Kron (release v0.1)</td>
      <td>https://usegalaxy.org</td>
      <td>4b5f6b5d4f36e38b</td>
      <td>https://usegalaxy.org/published/workflow?id=4b5f6b5d4f36e38b</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-09-19</td>
      <td>2024-09-19</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>krakentools_kreport2krona, taxonomy_krona_chart, kraken2</td>
      <td>Aggregation, Taxonomic classification, Visualisation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Pathogen Detection PathoGFAIR Samples Aggregation and Visualisation (release v0.1)</td>
      <td>https://usegalaxy.org</td>
      <td>3ef08553982f6ffc</td>
      <td>https://usegalaxy.org/published/workflow?id=3ef08553982f6ffc</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-09-19</td>
      <td>2024-09-19</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>tab2fasta, newick_display, __FILTER_EMPTY_DATASETS__, fasta_merge_files_and_filter_unique_sequences, tp_split_on_column, bedtools_getfastabed, Count1, __FILTER_FAILED_DATASETS__, regexColumn1, fasta2tab, collapse_dataset, tp_replace_in_column, clustalw, regex1, Remove beginning1, Cut1, ggplot2_heatmap, tp_multijoin_tool, fasttree, Grouping1, collection_column_join, tp_sorted_uniq</td>
      <td>Phylogenetic tree generation (maximum likelihood and Bayesian methods), Phylogenetic tree generation, Multiple sequence alignment, Phylogenetic tree analysis, Visualisation, Phylogenetic tree generation (from molecular sequences), Mapping, Phylogenetic tree reconstruction</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Gene-based Pathogen Identification (release v0.1)</td>
      <td>https://usegalaxy.org</td>
      <td>3f252e077e0bcce5</td>
      <td>https://usegalaxy.org/published/workflow?id=3f252e077e0bcce5</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:pathogfair, name:iwc, name:microgalaxy</td>
      <td>2024-09-19</td>
      <td>2024-09-19</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>compose_text_param, tab2fasta, medaka_consensus_pipeline, __BUILD_LIST__, collection_element_identifiers, bandage_image, tp_find_and_replace, flye, fasta2tab, split_file_to_collection, abricate, param_value_from_file</td>
      <td>Cross-assembly, Base-calling, De-novo assembly, Genome assembly, Antimicrobial resistance prediction, Mapping assembly, Sequence assembly, Variant calling, Sequence assembly visualisation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Pathogen Detection PathoGFAIR Samples Aggregation and Visualisation</td>
      <td>https://usegalaxy.org</td>
      <td>2d3063882d8239ff</td>
      <td>https://usegalaxy.org/published/workflow?id=2d3063882d8239ff</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-07-03</td>
      <td>2024-07-03</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>tab2fasta, newick_display, __FILTER_EMPTY_DATASETS__, fasta_merge_files_and_filter_unique_sequences, tp_split_on_column, bedtools_getfastabed, Count1, __FILTER_FAILED_DATASETS__, regexColumn1, fasta2tab, collapse_dataset, tp_replace_in_column, clustalw, regex1, Remove beginning1, Cut1, ggplot2_heatmap, tp_multijoin_tool, fasttree, Grouping1, collection_column_join, tp_sorted_uniq</td>
      <td>Phylogenetic tree generation (maximum likelihood and Bayesian methods), Phylogenetic tree generation, Multiple sequence alignment, Phylogenetic tree analysis, Visualisation, Phylogenetic tree generation (from molecular sequences), Mapping, Phylogenetic tree reconstruction</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>PathoGFAIR (imported from uploaded file)</td>
      <td>https://usegalaxy.org</td>
      <td>e55593af91337a05</td>
      <td>https://usegalaxy.org/published/workflow?id=e55593af91337a05</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:microgalaxy, name:iwc, name:pathogfair, name:nanopore, name:collection</td>
      <td>2024-07-03</td>
      <td>2024-07-03</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Taxonomy Profiling and Visualization with Krona</td>
      <td>https://usegalaxy.org</td>
      <td>8f5904693b5f74f4</td>
      <td>https://usegalaxy.org/published/workflow?id=8f5904693b5f74f4</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-06-25</td>
      <td>2024-06-25</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>krakentools_kreport2krona, taxonomy_krona_chart, kraken2</td>
      <td>Aggregation, Taxonomic classification, Visualisation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Gene-based Pathogen Identification</td>
      <td>https://usegalaxy.org</td>
      <td>cce88bc57b180d09</td>
      <td>https://usegalaxy.org/published/workflow?id=cce88bc57b180d09</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:pathogfair, name:iwc, name:microgalaxy</td>
      <td>2024-06-25</td>
      <td>2024-06-25</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>compose_text_param, tab2fasta, medaka_consensus_pipeline, __BUILD_LIST__, collection_element_identifiers, bandage_image, tp_find_and_replace, flye, fasta2tab, split_file_to_collection, abricate, param_value_from_file</td>
      <td>Cross-assembly, Base-calling, De-novo assembly, Genome assembly, Antimicrobial resistance prediction, Mapping assembly, Sequence assembly, Variant calling, Sequence assembly visualisation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Allele-based Pathogen Identification</td>
      <td>https://usegalaxy.org</td>
      <td>38911ba6f66d80f6</td>
      <td>https://usegalaxy.org/published/workflow?id=38911ba6f66d80f6</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-06-25</td>
      <td>2024-06-25</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>table_compute, tp_cut_tool, regexColumn1, bcftools_consensus, samtools_coverage, Paste1, clair3, CONVERTER_gz_to_uncompressed, Remove beginning1, Count1, collapse_dataset, snpSift_extractFields, samtools_depth, snpSift_filter, minimap2, Cut1, tp_head_tool, bcftools_norm</td>
      <td>Variant calling, Pairwise sequence alignment</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Nanopore Preprocessing</td>
      <td>https://usegalaxy.org</td>
      <td>574e42683dc3961b</td>
      <td>https://usegalaxy.org/published/workflow?id=574e42683dc3961b</td>
      <td>Bérénice Batut, Engy Nasr, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:nanopore, name:iwc</td>
      <td>2024-06-25</td>
      <td>2024-06-25</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>samtools_fastx, __FILTER_FAILED_DATASETS__, bamtools_split_mapped, krakentools_extract_kraken_reads, nanoplot, fastqc, Add_a_column1, fastp, regexColumn1, collection_column_join, Grep1, multiqc, kraken2, minimap2, porechop, Cut1, collapse_dataset</td>
      <td>Scatter plot plotting, Box-Whisker plot plotting, Sequencing quality control, Aggregation, Sequence contamination filtering, Sequence alignment analysis, Pairwise sequence alignment, Validation, Statistical calculation, Taxonomic classification, Visualisation, Sequence composition calculation, Data handling</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Allele-based Pathogen Identification (release v0.1.2)</td>
      <td>https://usegalaxy.org.au</td>
      <td>857e482acfcb2dba</td>
      <td>https://usegalaxy.org.au/published/workflow?id=857e482acfcb2dba</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-10-31</td>
      <td>2024-10-31</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>table_compute, tp_cut_tool, regexColumn1, bcftools_consensus, samtools_coverage, Paste1, clair3, CONVERTER_gz_to_uncompressed, Remove beginning1, Count1, collapse_dataset, snpSift_extractFields, samtools_depth, snpSift_filter, minimap2, Cut1, tp_head_tool, bcftools_norm</td>
      <td>Variant calling, Pairwise sequence alignment</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Taxonomy Profiling and Visualization with Kron (release v0.1)</td>
      <td>https://usegalaxy.org.au</td>
      <td>131636a795bac485</td>
      <td>https://usegalaxy.org.au/published/workflow?id=131636a795bac485</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-09-19</td>
      <td>2024-09-19</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>krakentools_kreport2krona, taxonomy_krona_chart, kraken2</td>
      <td>Aggregation, Taxonomic classification, Visualisation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Pathogen Detection PathoGFAIR Samples Aggregation and Visualisation (release v0.1)</td>
      <td>https://usegalaxy.org.au</td>
      <td>b60922a253df6654</td>
      <td>https://usegalaxy.org.au/published/workflow?id=b60922a253df6654</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-09-19</td>
      <td>2024-09-19</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>tab2fasta, newick_display, __FILTER_EMPTY_DATASETS__, fasta_merge_files_and_filter_unique_sequences, tp_split_on_column, bedtools_getfastabed, Count1, __FILTER_FAILED_DATASETS__, regexColumn1, fasta2tab, collapse_dataset, tp_replace_in_column, clustalw, regex1, Remove beginning1, Cut1, ggplot2_heatmap, tp_multijoin_tool, fasttree, Grouping1, collection_column_join, tp_sorted_uniq</td>
      <td>Phylogenetic tree generation (maximum likelihood and Bayesian methods), Phylogenetic tree generation, Multiple sequence alignment, Phylogenetic tree analysis, Visualisation, Phylogenetic tree generation (from molecular sequences), Mapping, Phylogenetic tree reconstruction</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Nanopore Preprocessing (release v0.1)</td>
      <td>https://usegalaxy.org.au</td>
      <td>69d91340fc7effa2</td>
      <td>https://usegalaxy.org.au/published/workflow?id=69d91340fc7effa2</td>
      <td>Bérénice Batut, Engy Nasr, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:nanopore, name:iwc</td>
      <td>2024-09-19</td>
      <td>2024-09-19</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>samtools_fastx, __FILTER_FAILED_DATASETS__, bamtools_split_mapped, krakentools_extract_kraken_reads, nanoplot, fastqc, Add_a_column1, fastp, regexColumn1, collection_column_join, Grep1, multiqc, kraken2, minimap2, porechop, Cut1, collapse_dataset</td>
      <td>Scatter plot plotting, Box-Whisker plot plotting, Sequencing quality control, Aggregation, Sequence contamination filtering, Sequence alignment analysis, Pairwise sequence alignment, Validation, Statistical calculation, Taxonomic classification, Visualisation, Sequence composition calculation, Data handling</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Gene-based Pathogen Identification (release v0.1)</td>
      <td>https://usegalaxy.org.au</td>
      <td>084bb76cf47d7060</td>
      <td>https://usegalaxy.org.au/published/workflow?id=084bb76cf47d7060</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:pathogfair, name:iwc, name:microgalaxy</td>
      <td>2024-09-19</td>
      <td>2024-09-19</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>compose_text_param, tab2fasta, medaka_consensus_pipeline, __BUILD_LIST__, collection_element_identifiers, bandage_image, tp_find_and_replace, flye, fasta2tab, split_file_to_collection, abricate, param_value_from_file</td>
      <td>Cross-assembly, Base-calling, De-novo assembly, Genome assembly, Antimicrobial resistance prediction, Mapping assembly, Sequence assembly, Variant calling, Sequence assembly visualisation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>PathoGFAIR (imported from uploaded file)</td>
      <td>https://usegalaxy.org.au</td>
      <td>f5f9808fb50b6f2c</td>
      <td>https://usegalaxy.org.au/published/workflow?id=f5f9808fb50b6f2c</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:microgalaxy, name:iwc, name:pathogfair, name:nanopore, name:collection</td>
      <td>2024-07-03</td>
      <td>2024-07-03</td>
      <td>0</td>
      <td>0</td>
      <td></td>
      <td></td>
      <td></td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Pathogen Detection PathoGFAIR Samples Aggregation and Visualisation</td>
      <td>https://usegalaxy.org.au</td>
      <td>eda40b58616a0fe4</td>
      <td>https://usegalaxy.org.au/published/workflow?id=eda40b58616a0fe4</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-07-03</td>
      <td>2024-07-03</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>tab2fasta, newick_display, __FILTER_EMPTY_DATASETS__, fasta_merge_files_and_filter_unique_sequences, tp_split_on_column, bedtools_getfastabed, Count1, __FILTER_FAILED_DATASETS__, regexColumn1, fasta2tab, collapse_dataset, tp_replace_in_column, clustalw, regex1, Remove beginning1, Cut1, ggplot2_heatmap, tp_multijoin_tool, fasttree, Grouping1, collection_column_join, tp_sorted_uniq</td>
      <td>Phylogenetic tree generation (maximum likelihood and Bayesian methods), Phylogenetic tree generation, Multiple sequence alignment, Phylogenetic tree analysis, Visualisation, Phylogenetic tree generation (from molecular sequences), Mapping, Phylogenetic tree reconstruction</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Allele-based Pathogen Identification</td>
      <td>https://usegalaxy.org.au</td>
      <td>244ea5e94237ebad</td>
      <td>https://usegalaxy.org.au/published/workflow?id=244ea5e94237ebad</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-06-25</td>
      <td>2024-06-25</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>table_compute, tp_cut_tool, regexColumn1, bcftools_consensus, samtools_coverage, Paste1, clair3, CONVERTER_gz_to_uncompressed, Remove beginning1, Count1, collapse_dataset, snpSift_extractFields, samtools_depth, snpSift_filter, minimap2, Cut1, tp_head_tool, bcftools_norm</td>
      <td>Variant calling, Pairwise sequence alignment</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Gene-based Pathogen Identification</td>
      <td>https://usegalaxy.org.au</td>
      <td>ef8c22c2525063a2</td>
      <td>https://usegalaxy.org.au/published/workflow?id=ef8c22c2525063a2</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:pathogfair, name:iwc, name:microgalaxy</td>
      <td>2024-06-25</td>
      <td>2024-06-25</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>compose_text_param, tab2fasta, medaka_consensus_pipeline, __BUILD_LIST__, collection_element_identifiers, bandage_image, tp_find_and_replace, flye, fasta2tab, split_file_to_collection, abricate, param_value_from_file</td>
      <td>Cross-assembly, Base-calling, De-novo assembly, Genome assembly, Antimicrobial resistance prediction, Mapping assembly, Sequence assembly, Variant calling, Sequence assembly visualisation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Taxonomy Profiling and Visualization with Krona</td>
      <td>https://usegalaxy.org.au</td>
      <td>d9ba165e6ae55417</td>
      <td>https://usegalaxy.org.au/published/workflow?id=d9ba165e6ae55417</td>
      <td>Engy Nasr, Bérénice Batut, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:iwc</td>
      <td>2024-06-25</td>
      <td>2024-06-25</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>krakentools_kreport2krona, taxonomy_krona_chart, kraken2</td>
      <td>Aggregation, Taxonomic classification, Visualisation</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
    <tr>
      <td></td>
      <td>Nanopore Preprocessing</td>
      <td>https://usegalaxy.org.au</td>
      <td>25d52afddaa3451b</td>
      <td>https://usegalaxy.org.au/published/workflow?id=25d52afddaa3451b</td>
      <td>Bérénice Batut, Engy Nasr, Paul Zierep</td>
      <td>name:collection, name:microgalaxy, name:pathogfair, name:nanopore, name:iwc</td>
      <td>2024-06-25</td>
      <td>2024-06-25</td>
      <td>1</td>
      <td>1</td>
      <td></td>
      <td>samtools_fastx, __FILTER_FAILED_DATASETS__, bamtools_split_mapped, krakentools_extract_kraken_reads, nanoplot, fastqc, Add_a_column1, fastp, regexColumn1, collection_column_join, Grep1, multiqc, kraken2, minimap2, porechop, Cut1, collapse_dataset</td>
      <td>Scatter plot plotting, Box-Whisker plot plotting, Sequencing quality control, Aggregation, Sequence contamination filtering, Sequence alignment analysis, Pairwise sequence alignment, Validation, Statistical calculation, Taxonomic classification, Visualisation, Sequence composition calculation, Data handling</td>
      <td></td>
      <td>MIT</td>
      <td></td>
      <td></td>
    </tr>
  </tbody>
</table>

</div>