<script src="https://code.jquery.com/jquery-3.7.0.js"></script>
<script src="https://cdn.datatables.net/1.13.6/js/jquery.dataTables.min.js"></script>
<script src="https://cdn.datatables.net/searchbuilder/1.6.0/js/dataTables.searchBuilder.min.js"></script>
<script src="https://cdn.datatables.net/datetime/1.5.1/js/dataTables.dateTime.min.js"></script>

<script src="https://cdn.datatables.net/responsive/2.5.0/js/dataTables.responsive.min.js"></script>

<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/responsive/2.2.3/css/responsive.dataTables.min.css">

<link rel="stylesheet" href="https://cdn.datatables.net/1.13.6/css/jquery.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/searchbuilder/1.6.0/css/searchBuilder.dataTables.min.css">
<link rel="stylesheet" href="https://cdn.datatables.net/datetime/1.5.1/css/dataTables.dateTime.min.css">

<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" />
<script type="text/javascript" src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>

<style type="text/css">
  .tooltip {
    font-family: Georgia;
    font-size: 18px;
  }
  .tooltip .tooltip-inner {
    background-color: #00FFFF;
    color: #000000;
    min-width: 400px;
  }
  </style>

<script>
    $(document).ready(function() {
    $('#dataframe').DataTable( {
        "sScrollX": "100%",
        "dom": "Qlfrtip",

        responsive: {
            details: {
                type: 'column'
            }
        },
        "bAutoWidth": false,
        columnDefs: [ {
            className: 'control',
            orderable: false,
            targets:   [ 0 ],
            width: 100,
        },
        { type: 'natural', targets: 1 } // allow normal sorting
        ],

    });
});
</script>

<div class="display nowrap" style="width:80%" width="100%">

<table class="dataframe display nowrap" id="dataframe">
  <thead>
    <tr style="text-align: right;">
      <th>Expand</th>
      <th>Topic</th>
      <th>Title</th>
      <th>Link</th>
      <th>EDAM topic</th>
      <th>EDAM operation</th>
      <th>Creation</th>
      <th>Last modification</th>
      <th>Version</th>
      <th>Tutorial</th>
      <th>Slides</th>
      <th>Video</th>
      <th>Workflows</th>
      <th>Tools</th>
      <th>Servers with precise tool versions</th>
      <th>Servers with tool but different versions</th>
      <th>Feedback number</th>
      <th>Feedback mean note</th>
      <th>Visitors</th>
      <th>Page views</th>
      <th>Visit duration</th>
      <th>Video views</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Decontamination of a genome assembly</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/assembly-decontamination/tutorial.html</td>
      <td></td>
      <td>Taxonomic classification, Data handling</td>
      <td>2024-09-04</td>
      <td>2025-05-15</td>
      <td>4</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>parse_mito_blast, ncbi_dustmasker_wrapper, ncbi_blastn_wrapper, tp_cat, tp_sed_tool, kraken2, Cut1, Filter1, gfastats</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>978</td>
      <td>1242</td>
      <td>2.700000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Unicycler Assembly</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/unicycler-assembly/tutorial.html</td>
      <td>Sequence assembly, Genomics, Microbiology</td>
      <td>Validation, Coding region prediction, Visualisation, Aggregation, Sequence composition calculation, Sequencing quality control, Sequence assembly validation, Genome assembly, Statistical calculation, Genome annotation, Gene prediction</td>
      <td>2017-10-11</td>
      <td>2025-05-15</td>
      <td>24</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>prokka, fastqc, multiqc, quast, unicycler</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>GalaxyTrakr, MISSISSIPPI, UseGalaxy.be, UseGalaxy.fr, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2547668</td>
      <td>6937019</td>
      <td>4.650000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Vertebrate genome assembly using HiFi, Bionano and Hi-C data - Step by Step</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/vgp_genome_assembly/tutorial.html</td>
      <td></td>
      <td>Visualisation, De-novo assembly, Sequence trimming, Data handling, Read pre-processing, Sequence alignment, Sequence assembly validation, Mapping, Primer removal, Read binning, Genome assembly, Phasing, k-mer counting, Pairwise sequence alignment, Transcriptome assembly, Scaffolding</td>
      <td>2021-06-04</td>
      <td>2025-05-15</td>
      <td>56</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>salsa, collapse_dataset, merqury, pretext_map, quast, tp_grep_tool, minimap2, busco, meryl, cutadapt, bwa_mem2, hifiasm, bedtools_bamtobed, gfa_to_fa, tp_cut_tool, gfastats, pretext_snapshot, collection_column_join, tp_find_and_replace, sort1, Add_a_column1, cat1, Convert characters1, bellerophon, genomescope, bionano_scaffold, purge_dups, yahs</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>14095</td>
      <td>21663</td>
      <td>2.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Unicycler assembly of SARS-CoV-2 genome with preprocessing to remove human genome reads</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/assembly-with-preprocessing/tutorial.html</td>
      <td></td>
      <td>Validation, Aggregation, Data handling, Sequence assembly visualisation, Read mapping, Sequencing quality control, Scatter plot plotting, Genome assembly, Pairwise sequence alignment, Sequence file editing, Sequence contamination filtering, Box-Whisker plot plotting</td>
      <td>2020-04-21</td>
      <td>2025-05-15</td>
      <td>14</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>collapse_dataset, multiqc, fasta_filter_by_length, fasterq_dump, fastp, bandage_image, samtools_stats, bowtie2, minimap2, samtools_fastx, __ZIP_COLLECTION__, samtools_view, seqtk_sample, unicycler, nanoplot, bandage_info</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td>MISSISSIPPI, UseGalaxy.be, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>2540535</td>
      <td>6924549</td>
      <td>5.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Genome assembly using PacBio data</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/hifi-assembly/tutorial.html</td>
      <td></td>
      <td>Visualisation, De-novo assembly, Mapping assembly, Sequence assembly validation, Cross-assembly, Genome assembly, Transcriptome assembly, Scaffolding</td>
      <td>2021-11-29</td>
      <td>2025-05-16</td>
      <td>1</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>fasta-stats, quast, busco, flye, hifiasm, gfa_to_fa</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>GalaxyTrakr</td>
      <td>0</td>
      <td></td>
      <td>77</td>
      <td>89</td>
      <td>0.666667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>ERGA post-assembly QC</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/ERGA-post-assembly-QC/tutorial.html</td>
      <td></td>
      <td>Sequence alignment analysis, De-novo assembly, Data handling, Sequence trimming, Sequence assembly visualisation, Genotyping, Sequence alignment, Sequence assembly validation, Genome assembly, Phasing, k-mer counting, Pairwise sequence alignment, Transcriptome assembly, Scaffolding</td>
      <td>2023-06-23</td>
      <td>2025-05-15</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>collapse_dataset, merqury, pretext_map, CONVERTER_gz_to_uncompressed, minimap2, blobtoolkit, busco, meryl, interactive_tool_blobtoolkit, bwa_mem2, gfastats, pretext_snapshot, bandage_image, bg_diamond, __EXTRACT_DATASET__, tp_find_and_replace, smudgeplot, samtools_stats, bellerophon, genomescope</td>
      <td>UseGalaxy.eu</td>
      <td>UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>1552</td>
      <td>2007</td>
      <td>1.983333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Using the VGP workflows to assemble a vertebrate genome with HiFi and Hi-C data</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/vgp_workflow_training/tutorial.html</td>
      <td></td>
      <td>Visualisation, Sequence assembly validation</td>
      <td>2022-04-06</td>
      <td>2025-05-15</td>
      <td>24</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>False</td>
      <td>quast</td>
      <td>Galaxy@AuBi, GalaxyTrakr, MISSISSIPPI, UseGalaxy.be, UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>2907</td>
      <td>4107</td>
      <td>2.833333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Genome Assembly Quality Control</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/assembly-quality-control/tutorial.html</td>
      <td></td>
      <td>Visualisation, De-novo assembly, Sequence assembly validation, Genome assembly, Phasing, k-mer counting, Transcriptome assembly, Scaffolding</td>
      <td>2022-08-26</td>
      <td>2025-05-15</td>
      <td>5</td>
      <td>True</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>merqury, quast, chromeister, busco, meryl</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1632280</td>
      <td>3960819</td>
      <td>4.666667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Assembly of the mitochondrial genome from PacBio HiFi reads</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/mitochondrion-assembly/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-09-03</td>
      <td>2025-05-15</td>
      <td>3</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>compress_file, mitohifi</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>583</td>
      <td>729</td>
      <td>2.100000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Large genome assembly and polishing</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/largegenome/tutorial.html</td>
      <td></td>
      <td>Visualisation, Variant calling, Sequencing quality control, Sequence assembly validation, Scatter plot plotting, Cross-assembly, Base-calling, Genome assembly, Transcriptome assembly, Box-Whisker plot plotting, Validation, De-novo assembly, Sequence composition calculation, Sequence assembly visualisation, Mapping assembly, Sequence assembly, Pairwise sequence alignment, Statistical calculation, Sequence contamination filtering, Scaffolding</td>
      <td>2023-03-06</td>
      <td>2025-05-15</td>
      <td>13</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>fasta-stats, fastqc, multiqc, quast, meryl, fastp, bandage_image, minimap2, busco, barchart_gnuplot, flye, racon, genomescope, nanoplot, medaka_consensus_pipeline</td>
      <td>UseGalaxy.eu, UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>6027</td>
      <td>7886</td>
      <td>2.066667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Assembly</td>
      <td>Chloroplast genome assembly</td>
      <td>https://training.galaxyproject.org//topics/assembly/tutorials/chloroplast-assembly/tutorial.html</td>
      <td></td>
      <td>Generation, Scatter plot plotting, Cross-assembly, Analysis, Genome assembly, Genome annotation, Box-Whisker plot plotting, Coding region prediction, Gene prediction, Read alignment, Genome visualisation, De-novo assembly, Sequence assembly visualisation, Read mapping, Sequence alignment, Mapping assembly, Sequence assembly, Genome indexing</td>
      <td>2020-12-04</td>
      <td>2025-05-15</td>
      <td>21</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>fasta-stats, prokka, pilon, bandage_image, jbrowse, bwa_mem, flye, upload1, nanoplot, bandage_info</td>
      <td>UseGalaxy.eu</td>
      <td>GalaxyTrakr, UseGalaxy.be, UseGalaxy.fr, UseGalaxy.no, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>7367</td>
      <td>11493</td>
      <td>2.800000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Climate</td>
      <td>Getting your hands-on earth data</td>
      <td>https://training.galaxyproject.org//topics/climate/tutorials/earth_system/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2024-10-04</td>
      <td>2025-05-10</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>argo_getdata, c3s, spocc_occ, obis_data</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>59</td>
      <td>74</td>
      <td>1.650000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Ecology</td>
      <td>Obis marine indicators</td>
      <td>https://training.galaxyproject.org//topics/ecology/tutorials/obisindicators/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2023-04-11</td>
      <td>2024-06-27</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>obisindicators, obis_data, tp_cut_tool</td>
      <td>UseGalaxy.eu</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>267</td>
      <td>322</td>
      <td>1.816667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Genome annotation with Helixer</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/helixer/tutorial.html</td>
      <td></td>
      <td>Differential protein expression profiling, Genome visualisation, Sequence assembly validation, Genome assembly, Sequence annotation, Transcriptome assembly, Genome annotation, Scaffolding, Gene prediction</td>
      <td>2024-08-21</td>
      <td>2025-05-15</td>
      <td>7</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>helixer, jcvi_gff_stats, jbrowse, busco, omark, gffread</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>738</td>
      <td>883</td>
      <td>1.966667</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Genome annotation with Braker3</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/braker3/tutorial.html</td>
      <td>Sequence assembly, Genomics</td>
      <td>Differential protein expression profiling, Genome visualisation, Sequence alignment, Sequence assembly validation, Genome assembly, Sequence annotation, Transcriptome assembly, Genome annotation, Scaffolding, Gene prediction</td>
      <td>2025-03-17</td>
      <td>2025-05-15</td>
      <td>2</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>fasta-stats, rna_star, jbrowse, busco, omark, braker3, gffread</td>
      <td></td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>411</td>
      <td>483</td>
      <td>1.283333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Masking repeats with RepeatMasker</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/repeatmasker/tutorial.html</td>
      <td></td>
      <td>RNA-Seq analysis, Editing, Repeat sequence detection, Mapping, Genome annotation</td>
      <td>2021-11-29</td>
      <td>2025-05-15</td>
      <td>21</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>bedtools_maskfastabed, repeatmodeler, red, repeatmasker_wrapper</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>8992</td>
      <td>11533</td>
      <td>1.383333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Refining Genome Annotations with Apollo (prokaryotes)</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/apollo/tutorial.html</td>
      <td>Genomics, Microbiology, Gene and protein families, Sequence analysis</td>
      <td>Genome visualisation</td>
      <td>2021-06-04</td>
      <td>2025-05-15</td>
      <td>22</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>True</td>
      <td>create_or_update, jbrowse, list_organism, iframe, create_account</td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>2309956</td>
      <td>6138571</td>
      <td>5.300000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Identification of AMR genes in an assembled bacterial genome</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/amr-gene-detection/tutorial.html</td>
      <td>Whole genome sequencing, Public health and epidemiology, Genomics, Microbiology, Sequence analysis, Infectious disease, Antimicrobial resistance</td>
      <td>Antimicrobial resistance prediction, Read mapping, Genome visualisation, Genome annotation</td>
      <td>2024-01-23</td>
      <td>2025-05-15</td>
      <td>8</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>Grep1, jbrowse, bowtie2, bakta, staramr_search, tbl2gff3, upload1</td>
      <td>UseGalaxy.eu, UseGalaxy.fr</td>
      <td>Galaxy@AuBi, UseGalaxy.org (Main)</td>
      <td>0</td>
      <td></td>
      <td>2638</td>
      <td>3599</td>
      <td>2.883333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Genome Annotation</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/genome-annotation/tutorial.html</td>
      <td></td>
      <td></td>
      <td>2015-06-29</td>
      <td>2025-05-15</td>
      <td>18</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td></td>
      <td></td>
      <td>UseGalaxy.eu, UseGalaxy.org, UseGalaxy.org.au, UseGalaxy.fr</td>
      <td>0</td>
      <td></td>
      <td>34542</td>
      <td>63252</td>
      <td>1.450000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Comparative gene analysis in unannotated genomes</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/gene-centric/tutorial.html</td>
      <td>Genomics, Gene and protein families, Sequence analysis, Phylogeny</td>
      <td>Sequence alignment analysis, De-novo assembly, Database search, Phylogenetic tree generation, Multiple sequence alignment, Transcriptome assembly, Coding region prediction</td>
      <td>2022-09-08</td>
      <td>2025-05-15</td>
      <td>9</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>bg_diamond_makedb, collapse_dataset, bg_diamond_view, Add_a_column1, gops_intersect_1, join1, cat1, regexColumn1, rbc_mafft, orfipy, bg_diamond, rapidnj, tp_split_on_column, tab2fasta, Cut1, Filter1</td>
      <td>UseGalaxy.eu, UseGalaxy.org (Main)</td>
      <td></td>
      <td>0</td>
      <td></td>
      <td>1626</td>
      <td>2343</td>
      <td>3.133333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Genome annotation with Funannotate</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/funannotate/tutorial.html</td>
      <td></td>
      <td>Information extraction, Genome visualisation, Sequence motif recognition, Sequence alignment, Sequence assembly validation, Protein feature detection, Homology-based gene prediction, Fold recognition, Genome assembly, Query and retrieval, Formatting, Sequence annotation, Transcriptome assembly, Genome annotation, Scaffolding</td>
      <td>2021-11-29</td>
      <td>2025-05-15</td>
      <td>25</td>
      <td>True</td>
      <td>False</td>
      <td>True</td>
      <td>True</td>
      <td>eggnog_mapper, funannotate_annotate, rna_star, picard_MergeSamFiles, aegean_parseval, funannotate_compare, jbrowse, busco, interproscan, funannotate_predict</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main)</td>
      <td>UseGalaxy.be, UseGalaxy.org.au</td>
      <td>0</td>
      <td></td>
      <td>9315</td>
      <td>13458</td>
      <td>2.000000</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Functional annotation of protein sequences</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/functional/tutorial.html</td>
      <td></td>
      <td>Sequence alignment analysis, Fold recognition, Query and retrieval, Homology-based gene prediction, Information extraction, Sequence motif recognition, Genome annotation, Protein feature detection</td>
      <td>2022-07-20</td>
      <td>2025-05-15</td>
      <td>11</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>True</td>
      <td>eggnog_mapper, bg_diamond, interproscan, ncbi_blastp_wrapper</td>
      <td>UseGalaxy.eu, UseGalaxy.fr, UseGalaxy.org (Main), UseGalaxy.org.au</td>
      <td>UseGalaxy.no</td>
      <td>0</td>
      <td></td>
      <td>3453</td>
      <td>4757</td>
      <td>1.683333</td>
      <td>0</td>
    </tr>
    <tr>
      <td></td>
      <td>Genome Annotation</td>
      <td>Refining Genome Annotations with Apollo (eukaryotes)</td>
      <td>https://training.galaxyproject.org//topics/genome-annotation/tutorials/apollo-euk/tutorial.html</td>
      <td></td>
      <td>Genome visualisation</td>
      <td>2022-08-22</td>
      <td>2025-05-15</td>
      <td>15</td>
      <td>True</td>
      <td>False</td>
      <td>False</td>
      <td>False</td>
      <td>create_or_update, iframe, jbrowse</td>
      <td>UseGalaxy.no</td>
      <td>UseGalaxy.eu</td>
      <td>0</td>
      <td></td>
      <td>1205</td>
      <td>1922</td>
      <td>2.550000</td>
      <td>0</td>
    </tr>
  </tbody>
</table>

</div>